sql/tests: TestRandomSyntaxFunctions failed

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/6329087?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/6329087?buildTab=artifacts#/) on master @ [c080f7ac2f5be11aedd787dceba77cca7df18a16](https://github.com/cockroachdb/cockroach/commits/c080f7ac2f5be11aedd787dceba77cca7df18a16):


```
    rsg_test.go:820: 45s of 5m0s: 156746 executions, 96673 successful
    rsg_test.go:820: 50s of 5m0s: 174703 executions, 107762 successful
    rsg_test.go:820: 55s of 5m0s: 192760 executions, 118873 successful
    rsg_test.go:820: 1m0s of 5m0s: 210021 executions, 129546 successful
    rsg_test.go:820: 1m5s of 5m0s: 228154 executions, 140737 successful
    rsg_test.go:820: 1m10s of 5m0s: 246198 executions, 151877 successful
    rsg_test.go:820: 1m15s of 5m0s: 264090 executions, 163033 successful
    rsg_test.go:820: 1m20s of 5m0s: 282194 executions, 174087 successful
    rsg_test.go:820: 1m25s of 5m0s: 300059 executions, 185025 successful
    rsg_test.go:820: 1m30s of 5m0s: 317408 executions, 195747 successful
    rsg_test.go:820: 1m35s of 5m0s: 335667 executions, 207013 successful
    rsg_test.go:820: 1m40s of 5m0s: 353489 executions, 218030 successful
    rsg_test.go:820: 1m45s of 5m0s: 371337 executions, 229052 successful
    rsg_test.go:820: 1m50s of 5m0s: 389520 executions, 240274 successful
    rsg_test.go:820: 1m55s of 5m0s: 407339 executions, 251292 successful
    rsg_test.go:820: 2m0s of 5m0s: 424933 executions, 262068 successful
    rsg_test.go:820: 2m5s of 5m0s: 443046 executions, 273205 successful
    rsg_test.go:820: 2m10s of 5m0s: 461153 executions, 284461 successful
    rsg_test.go:820: 2m15s of 5m0s: 479289 executions, 295589 successful
    rsg_test.go:820: 2m20s of 5m0s: 497522 executions, 306814 successful
    rsg_test.go:820: 2m25s of 5m0s: 515149 executions, 317713 successful
    rsg_test.go:820: 2m30s of 5m0s: 533213 executions, 328893 successful
    rsg_test.go:820: 2m35s of 5m0s: 551090 executions, 339877 successful
    rsg_test.go:820: 2m40s of 5m0s: 569107 executions, 351013 successful
    rsg_test.go:820: 2m45s of 5m0s: 586718 executions, 361835 successful
    rsg_test.go:820: 2m50s of 5m0s: 604914 executions, 373081 successful
    rsg_test.go:820: 2m55s of 5m0s: 622200 executions, 383725 successful
    rsg_test.go:820: 3m0s of 5m0s: 640554 executions, 395036 successful
    rsg_test.go:820: 3m5s of 5m0s: 658946 executions, 406404 successful
    rsg_test.go:820: 3m10s of 5m0s: 677338 executions, 417797 successful
    rsg_test.go:820: 3m15s of 5m0s: 695571 executions, 428937 successful
    rsg_test.go:820: 3m20s of 5m0s: 713660 executions, 440100 successful
    rsg_test.go:820: 3m25s of 5m0s: 731214 executions, 450938 successful
    rsg_test.go:820: 3m30s of 5m0s: 749689 executions, 462326 successful
    rsg_test.go:820: 3m35s of 5m0s: 768087 executions, 473765 successful
    rsg_test.go:820: 3m40s of 5m0s: 786034 executions, 484960 successful
    rsg_test.go:820: 3m45s of 5m0s: 804230 executions, 496174 successful
    rsg_test.go:820: 3m50s of 5m0s: 822656 executions, 507511 successful
    rsg_test.go:820: 3m55s of 5m0s: 840188 executions, 518318 successful
    rsg_test.go:820: 4m0s of 5m0s: 858312 executions, 529615 successful
    rsg_test.go:820: 4m5s of 5m0s: 876306 executions, 540582 successful
    rsg_test.go:820: 4m10s of 5m0s: 894440 executions, 551799 successful
    rsg_test.go:820: 4m15s of 5m0s: 912671 executions, 563066 successful
    rsg_test.go:820: 4m20s of 5m0s: 930734 executions, 574302 successful
    rsg_test.go:820: 4m25s of 5m0s: 948784 executions, 585327 successful
    rsg_test.go:820: 4m30s of 5m0s: 966004 executions, 595997 successful
    rsg_test.go:820: 4m35s of 5m0s: 984193 executions, 607130 successful
    rsg_test.go:820: 4m40s of 5m0s: 1002572 executions, 618428 successful
    rsg_test.go:820: 4m45s of 5m0s: 1020659 executions, 629599 successful
    rsg_test.go:820: 4m50s of 5m0s: 1038666 executions, 640825 successful
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #81570 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-21.2]
- #78070 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.1]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-19268

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7342311?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7342311?buildTab=artifacts#/) on master @ [c9094fa1a1ef8d44a2d5299d1d7380b34a06241b](https://github.com/cockroachdb/cockroach/commits/c9094fa1a1ef8d44a2d5299d1d7380b34a06241b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 2684371 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 17613 executions, 10794 successful
    rsg_test.go:823: 10s of 5m0s: 35960 executions, 22204 successful
    rsg_test.go:823: 15s of 5m0s: 54373 executions, 33563 successful
    rsg_test.go:823: 20s of 5m0s: 73129 executions, 45144 successful
    rsg_test.go:823: 25s of 5m0s: 91771 executions, 56769 successful
    rsg_test.go:823: 30s of 5m0s: 109734 executions, 67926 successful
    rsg_test.go:823: 35s of 5m0s: 128637 executions, 79752 successful
    rsg_test.go:823: 40s of 5m0s: 147670 executions, 91453 successful
    rsg_test.go:823: 45s of 5m0s: 166646 executions, 103148 successful
    rsg_test.go:823: 50s of 5m0s: 185434 executions, 114768 successful
    rsg_test.go:823: 55s of 5m0s: 203878 executions, 126126 successful
    rsg_test.go:823: 1m0s of 5m0s: 222559 executions, 137764 successful
    rsg_test.go:823: 1m5s of 5m0s: 240871 executions, 149078 successful
    rsg_test.go:823: 1m10s of 5m0s: 259861 executions, 160871 successful
    rsg_test.go:823: 1m15s of 5m0s: 278879 executions, 172642 successful
    rsg_test.go:823: 1m20s of 5m0s: 297970 executions, 184436 successful
    rsg_test.go:823: 1m25s of 5m0s: 316948 executions, 196103 successful
    rsg_test.go:823: 1m30s of 5m0s: 334886 executions, 207264 successful
    rsg_test.go:823: 1m35s of 5m0s: 353630 executions, 218850 successful
    rsg_test.go:823: 1m40s of 5m0s: 372631 executions, 230443 successful
    rsg_test.go:823: 1m45s of 5m0s: 391628 executions, 242225 successful
    rsg_test.go:823: 1m50s of 5m0s: 411049 executions, 254320 successful
    rsg_test.go:823: 1m55s of 5m0s: 430277 executions, 266086 successful
    rsg_test.go:823: 2m0s of 5m0s: 449454 executions, 277989 successful
    rsg_test.go:823: 2m5s of 5m0s: 467871 executions, 289469 successful
    rsg_test.go:823: 2m10s of 5m0s: 486801 executions, 301121 successful
    rsg_test.go:823: 2m15s of 5m0s: 505967 executions, 312985 successful
    rsg_test.go:823: 2m20s of 5m0s: 525316 executions, 324950 successful
    rsg_test.go:823: 2m25s of 5m0s: 544435 executions, 336891 successful
    rsg_test.go:823: 2m30s of 5m0s: 563575 executions, 348789 successful
    rsg_test.go:823: 2m35s of 5m0s: 581912 executions, 360210 successful
    rsg_test.go:823: 2m40s of 5m0s: 600591 executions, 371773 successful
    rsg_test.go:823: 2m45s of 5m0s: 619769 executions, 383479 successful
    rsg_test.go:823: 2m50s of 5m0s: 638968 executions, 395403 successful
    rsg_test.go:823: 2m55s of 5m0s: 658152 executions, 407339 successful
    rsg_test.go:823: 3m0s of 5m0s: 677368 executions, 419160 successful
    rsg_test.go:823: 3m5s of 5m0s: 695592 executions, 430543 successful
    rsg_test.go:823: 3m10s of 5m0s: 714882 executions, 442418 successful
    rsg_test.go:823: 3m15s of 5m0s: 733942 executions, 454197 successful
    rsg_test.go:823: 3m20s of 5m0s: 752910 executions, 466030 successful
    rsg_test.go:823: 3m25s of 5m0s: 771756 executions, 477657 successful
    rsg_test.go:823: 3m30s of 5m0s: 790806 executions, 489348 successful
    rsg_test.go:823: 3m35s of 5m0s: 808996 executions, 500635 successful
    rsg_test.go:823: 3m40s of 5m0s: 828271 executions, 512493 successful
    rsg_test.go:823: 3m45s of 5m0s: 847186 executions, 524174 successful
    rsg_test.go:823: 3m50s of 5m0s: 866163 executions, 536027 successful
    rsg_test.go:823: 3m55s of 5m0s: 885102 executions, 547788 successful
    rsg_test.go:823: 4m0s of 5m0s: 904224 executions, 559565 successful
    rsg_test.go:823: 4m5s of 5m0s: 923040 executions, 571205 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7354421?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7354421?buildTab=artifacts#/) on master @ [4c828ca5df8ea70b92ba58c6cb4868303cee1262](https://github.com/cockroachdb/cockroach/commits/4c828ca5df8ea70b92ba58c6cb4868303cee1262):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 3500333 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions551076624
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 17663 executions, 10856 successful
    rsg_test.go:823: 10s of 5m0s: 36021 executions, 22276 successful
    rsg_test.go:823: 15s of 5m0s: 54938 executions, 33962 successful
    rsg_test.go:823: 20s of 5m0s: 73551 executions, 45496 successful
    rsg_test.go:823: 25s of 5m0s: 92491 executions, 57237 successful
    rsg_test.go:823: 30s of 5m0s: 110315 executions, 68258 successful
    rsg_test.go:823: 35s of 5m0s: 128871 executions, 79707 successful
    rsg_test.go:823: 40s of 5m0s: 148009 executions, 91524 successful
    rsg_test.go:823: 45s of 5m0s: 166913 executions, 103264 successful
    rsg_test.go:823: 50s of 5m0s: 186208 executions, 115256 successful
    rsg_test.go:823: 55s of 5m0s: 205155 executions, 126955 successful
    rsg_test.go:823: 1m0s of 5m0s: 223125 executions, 138051 successful
    rsg_test.go:823: 1m5s of 5m0s: 241555 executions, 149445 successful
    rsg_test.go:823: 1m10s of 5m0s: 260511 executions, 161237 successful
    rsg_test.go:823: 1m15s of 5m0s: 279378 executions, 172965 successful
    rsg_test.go:823: 1m20s of 5m0s: 298534 executions, 184795 successful
    rsg_test.go:823: 1m25s of 5m0s: 317654 executions, 196600 successful
    rsg_test.go:823: 1m30s of 5m0s: 336111 executions, 208019 successful
    rsg_test.go:823: 1m35s of 5m0s: 353950 executions, 218995 successful
    rsg_test.go:823: 1m40s of 5m0s: 372827 executions, 230630 successful
    rsg_test.go:823: 1m45s of 5m0s: 391754 executions, 242274 successful
    rsg_test.go:823: 1m50s of 5m0s: 410911 executions, 254172 successful
    rsg_test.go:823: 1m55s of 5m0s: 430208 executions, 265992 successful
    rsg_test.go:823: 2m0s of 5m0s: 448590 executions, 277415 successful
    rsg_test.go:823: 2m5s of 5m0s: 467703 executions, 289150 successful
    rsg_test.go:823: 2m10s of 5m0s: 486481 executions, 300798 successful
    rsg_test.go:823: 2m15s of 5m0s: 505264 executions, 312393 successful
    rsg_test.go:823: 2m20s of 5m0s: 524431 executions, 324299 successful
    rsg_test.go:823: 2m25s of 5m0s: 543617 executions, 336100 successful
    rsg_test.go:823: 2m30s of 5m0s: 562016 executions, 347536 successful
    rsg_test.go:823: 2m35s of 5m0s: 581181 executions, 359376 successful
    rsg_test.go:823: 2m40s of 5m0s: 600356 executions, 371321 successful
    rsg_test.go:823: 2m45s of 5m0s: 619997 executions, 383404 successful
    rsg_test.go:823: 2m50s of 5m0s: 638966 executions, 395139 successful
    rsg_test.go:823: 2m55s of 5m0s: 658414 executions, 407232 successful
    rsg_test.go:823: 3m0s of 5m0s: 676818 executions, 418551 successful
    rsg_test.go:823: 3m5s of 5m0s: 696281 executions, 430533 successful
    rsg_test.go:823: 3m10s of 5m0s: 715232 executions, 442237 successful
    rsg_test.go:823: 3m15s of 5m0s: 734573 executions, 454237 successful
    rsg_test.go:823: 3m20s of 5m0s: 753951 executions, 466167 successful
    rsg_test.go:823: 3m25s of 5m0s: 773081 executions, 477998 successful
    rsg_test.go:823: 3m30s of 5m0s: 792260 executions, 489859 successful
    rsg_test.go:823: 3m35s of 5m0s: 810651 executions, 501212 successful
    rsg_test.go:823: 3m40s of 5m0s: 829872 executions, 512997 successful
    rsg_test.go:823: 3m45s of 5m0s: 849314 executions, 525125 successful
    rsg_test.go:823: 3m50s of 5m0s: 868649 executions, 536997 successful
    rsg_test.go:823: 3m55s of 5m0s: 888272 executions, 549162 successful
    rsg_test.go:823: 4m0s of 5m0s: 907363 executions, 561017 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7377860?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7377860?buildTab=artifacts#/) on master @ [ff8f7eda167887e1f11bbcd9029aa4cc2d28085e](https://github.com/cockroachdb/cockroach/commits/ff8f7eda167887e1f11bbcd9029aa4cc2d28085e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4466935 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 20s of 5m0s: 76260 executions, 47181 successful
    rsg_test.go:823: 25s of 5m0s: 95410 executions, 59063 successful
    rsg_test.go:823: 30s of 5m0s: 114907 executions, 71024 successful
    rsg_test.go:823: 35s of 5m0s: 133585 executions, 82507 successful
    rsg_test.go:823: 40s of 5m0s: 153327 executions, 94708 successful
    rsg_test.go:823: 45s of 5m0s: 172462 executions, 106580 successful
    rsg_test.go:823: 50s of 5m0s: 191461 executions, 118274 successful
    rsg_test.go:823: 55s of 5m0s: 210807 executions, 130184 successful
    rsg_test.go:823: 1m0s of 5m0s: 229332 executions, 141668 successful
    rsg_test.go:823: 1m5s of 5m0s: 248614 executions, 153632 successful
    rsg_test.go:823: 1m10s of 5m0s: 268088 executions, 165755 successful
    rsg_test.go:823: 1m15s of 5m0s: 287385 executions, 177622 successful
    rsg_test.go:823: 1m20s of 5m0s: 306663 executions, 189585 successful
    rsg_test.go:823: 1m25s of 5m0s: 326264 executions, 201687 successful
    rsg_test.go:823: 1m30s of 5m0s: 345290 executions, 213385 successful
    rsg_test.go:823: 1m35s of 5m0s: 364846 executions, 225564 successful
    rsg_test.go:823: 1m40s of 5m0s: 384656 executions, 237717 successful
    rsg_test.go:823: 1m45s of 5m0s: 403978 executions, 249718 successful
    rsg_test.go:823: 1m50s of 5m0s: 423512 executions, 261856 successful
    rsg_test.go:823: 1m55s of 5m0s: 443033 executions, 273912 successful
    rsg_test.go:823: 2m0s of 5m0s: 461882 executions, 285533 successful
    rsg_test.go:823: 2m5s of 5m0s: 481419 executions, 297798 successful
    rsg_test.go:823: 2m10s of 5m0s: 500933 executions, 309733 successful
    rsg_test.go:823: 2m15s of 5m0s: 520420 executions, 321872 successful
    rsg_test.go:823: 2m20s of 5m0s: 539609 executions, 333760 successful
    rsg_test.go:823: 2m25s of 5m0s: 558018 executions, 345178 successful
    rsg_test.go:823: 2m30s of 5m0s: 577392 executions, 357112 successful
    rsg_test.go:823: 2m35s of 5m0s: 596877 executions, 369235 successful
    rsg_test.go:823: 2m40s of 5m0s: 615003 executions, 380393 successful
    rsg_test.go:823: 2m45s of 5m0s: 633326 executions, 391764 successful
    rsg_test.go:823: 2m50s of 5m0s: 652663 executions, 403797 successful
    rsg_test.go:823: 2m55s of 5m0s: 671211 executions, 415335 successful
    rsg_test.go:823: 3m0s of 5m0s: 690540 executions, 427214 successful
    rsg_test.go:823: 3m5s of 5m0s: 709957 executions, 439252 successful
    rsg_test.go:823: 3m10s of 5m0s: 729256 executions, 451161 successful
    rsg_test.go:823: 3m15s of 5m0s: 748544 executions, 463105 successful
    rsg_test.go:823: 3m20s of 5m0s: 767672 executions, 475030 successful
    rsg_test.go:823: 3m25s of 5m0s: 787253 executions, 487084 successful
    rsg_test.go:823: 3m30s of 5m0s: 805821 executions, 498555 successful
    rsg_test.go:823: 3m35s of 5m0s: 825006 executions, 510507 successful
    rsg_test.go:823: 3m40s of 5m0s: 844437 executions, 522511 successful
    rsg_test.go:823: 3m45s of 5m0s: 863570 executions, 534446 successful
    rsg_test.go:823: 3m50s of 5m0s: 882784 executions, 546406 successful
    rsg_test.go:823: 3m55s of 5m0s: 901713 executions, 558150 successful
    rsg_test.go:823: 4m0s of 5m0s: 920277 executions, 569642 successful
    rsg_test.go:823: 4m5s of 5m0s: 939729 executions, 581678 successful
    rsg_test.go:823: 4m10s of 5m0s: 958946 executions, 593513 successful
    rsg_test.go:823: 4m15s of 5m0s: 977999 executions, 605225 successful
    rsg_test.go:823: 4m20s of 5m0s: 997302 executions, 617119 successful
    rsg_test.go:823: 4m25s of 5m0s: 1016567 executions, 629100 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7402571?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7402571?buildTab=artifacts#/) on master @ [ee4fdb7fff683a7916ee585ac0c1e22d5d9a4412](https://github.com/cockroachdb/cockroach/commits/ee4fdb7fff683a7916ee585ac0c1e22d5d9a4412):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4385700 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions4193413703
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 16884 executions, 10483 successful
    rsg_test.go:823: 10s of 5m0s: 34299 executions, 21346 successful
    rsg_test.go:823: 15s of 5m0s: 51601 executions, 32052 successful
    rsg_test.go:823: 20s of 5m0s: 68797 executions, 42677 successful
    rsg_test.go:823: 25s of 5m0s: 86689 executions, 53764 successful
    rsg_test.go:823: 30s of 5m0s: 104365 executions, 64770 successful
    rsg_test.go:823: 35s of 5m0s: 121113 executions, 75164 successful
    rsg_test.go:823: 40s of 5m0s: 139082 executions, 86248 successful
    rsg_test.go:823: 45s of 5m0s: 157315 executions, 97522 successful
    rsg_test.go:823: 50s of 5m0s: 174982 executions, 108463 successful
    rsg_test.go:823: 55s of 5m0s: 193049 executions, 119611 successful
    rsg_test.go:823: 1m0s of 5m0s: 210899 executions, 130709 successful
    rsg_test.go:823: 1m5s of 5m0s: 228557 executions, 141637 successful
    rsg_test.go:823: 1m10s of 5m0s: 245656 executions, 152284 successful
    rsg_test.go:823: 1m15s of 5m0s: 263868 executions, 163564 successful
    rsg_test.go:823: 1m20s of 5m0s: 282406 executions, 175052 successful
    rsg_test.go:823: 1m25s of 5m0s: 300674 executions, 186374 successful
    rsg_test.go:823: 1m30s of 5m0s: 318984 executions, 197741 successful
    rsg_test.go:823: 1m35s of 5m0s: 337435 executions, 209222 successful
    rsg_test.go:823: 1m40s of 5m0s: 354691 executions, 219793 successful
    rsg_test.go:823: 1m45s of 5m0s: 373117 executions, 231241 successful
    rsg_test.go:823: 1m50s of 5m0s: 391510 executions, 242756 successful
    rsg_test.go:823: 1m55s of 5m0s: 409717 executions, 254061 successful
    rsg_test.go:823: 2m0s of 5m0s: 427957 executions, 265383 successful
    rsg_test.go:823: 2m5s of 5m0s: 446641 executions, 277045 successful
    rsg_test.go:823: 2m10s of 5m0s: 464017 executions, 287792 successful
    rsg_test.go:823: 2m15s of 5m0s: 482265 executions, 299080 successful
    rsg_test.go:823: 2m20s of 5m0s: 500572 executions, 310355 successful
    rsg_test.go:823: 2m25s of 5m0s: 518900 executions, 321653 successful
    rsg_test.go:823: 2m30s of 5m0s: 536742 executions, 332713 successful
    rsg_test.go:823: 2m35s of 5m0s: 554312 executions, 343566 successful
    rsg_test.go:823: 2m40s of 5m0s: 572759 executions, 355000 successful
    rsg_test.go:823: 2m45s of 5m0s: 590883 executions, 366244 successful
    rsg_test.go:823: 2m50s of 5m0s: 609200 executions, 377645 successful
    rsg_test.go:823: 2m55s of 5m0s: 627560 executions, 389035 successful
    rsg_test.go:823: 3m0s of 5m0s: 645692 executions, 400231 successful
    rsg_test.go:823: 3m5s of 5m0s: 663148 executions, 411092 successful
    rsg_test.go:823: 3m10s of 5m0s: 681009 executions, 422068 successful
    rsg_test.go:823: 3m15s of 5m0s: 699272 executions, 433428 successful
    rsg_test.go:823: 3m20s of 5m0s: 717571 executions, 444755 successful
    rsg_test.go:823: 3m25s of 5m0s: 736117 executions, 456185 successful
    rsg_test.go:823: 3m30s of 5m0s: 753717 executions, 467120 successful
    rsg_test.go:823: 3m35s of 5m0s: 772020 executions, 478501 successful
    rsg_test.go:823: 3m40s of 5m0s: 790794 executions, 490132 successful
    rsg_test.go:823: 3m45s of 5m0s: 809514 executions, 501703 successful
    rsg_test.go:823: 3m50s of 5m0s: 827855 executions, 512983 successful
    rsg_test.go:823: 3m55s of 5m0s: 846006 executions, 524175 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7423831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7423831?buildTab=artifacts#/) on master @ [d6ae9653b67071cca09ed296711cecc821fc284e](https://github.com/cockroachdb/cockroach/commits/d6ae9653b67071cca09ed296711cecc821fc284e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5485737 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 45s of 5m0s: 161879 executions, 100284 successful
    rsg_test.go:823: 50s of 5m0s: 180232 executions, 111583 successful
    rsg_test.go:823: 55s of 5m0s: 198716 executions, 123038 successful
    rsg_test.go:823: 1m0s of 5m0s: 216174 executions, 133867 successful
    rsg_test.go:823: 1m5s of 5m0s: 234208 executions, 145015 successful
    rsg_test.go:823: 1m10s of 5m0s: 252339 executions, 156259 successful
    rsg_test.go:823: 1m15s of 5m0s: 270413 executions, 167550 successful
    rsg_test.go:823: 1m20s of 5m0s: 289144 executions, 179093 successful
    rsg_test.go:823: 1m25s of 5m0s: 307655 executions, 190612 successful
    rsg_test.go:823: 1m30s of 5m0s: 325942 executions, 201956 successful
    rsg_test.go:823: 1m35s of 5m0s: 343994 executions, 213069 successful
    rsg_test.go:823: 1m40s of 5m0s: 362242 executions, 224401 successful
    rsg_test.go:823: 1m45s of 5m0s: 381086 executions, 236102 successful
    rsg_test.go:823: 1m50s of 5m0s: 399718 executions, 247653 successful
    rsg_test.go:823: 1m55s of 5m0s: 418434 executions, 259273 successful
    rsg_test.go:823: 2m0s of 5m0s: 437028 executions, 270825 successful
    rsg_test.go:823: 2m5s of 5m0s: 454629 executions, 281742 successful
    rsg_test.go:823: 2m10s of 5m0s: 473285 executions, 293317 successful
    rsg_test.go:823: 2m15s of 5m0s: 491874 executions, 304827 successful
    rsg_test.go:823: 2m20s of 5m0s: 510334 executions, 316347 successful
    rsg_test.go:823: 2m25s of 5m0s: 529181 executions, 328009 successful
    rsg_test.go:823: 2m30s of 5m0s: 547692 executions, 339527 successful
    rsg_test.go:823: 2m35s of 5m0s: 566430 executions, 351180 successful
    rsg_test.go:823: 2m40s of 5m0s: 584233 executions, 362182 successful
    rsg_test.go:823: 2m45s of 5m0s: 602868 executions, 373683 successful
    rsg_test.go:823: 2m50s of 5m0s: 621747 executions, 385384 successful
    rsg_test.go:823: 2m55s of 5m0s: 640252 executions, 396768 successful
    rsg_test.go:823: 3m0s of 5m0s: 658970 executions, 408342 successful
    rsg_test.go:823: 3m5s of 5m0s: 677864 executions, 420053 successful
    rsg_test.go:823: 3m10s of 5m0s: 695816 executions, 431139 successful
    rsg_test.go:823: 3m15s of 5m0s: 714338 executions, 442532 successful
    rsg_test.go:823: 3m20s of 5m0s: 733501 executions, 454573 successful
    rsg_test.go:823: 3m25s of 5m0s: 752106 executions, 466111 successful
    rsg_test.go:823: 3m30s of 5m0s: 770822 executions, 477650 successful
    rsg_test.go:823: 3m35s of 5m0s: 789417 executions, 489187 successful
    rsg_test.go:823: 3m40s of 5m0s: 807278 executions, 500183 successful
    rsg_test.go:823: 3m45s of 5m0s: 825879 executions, 511668 successful
    rsg_test.go:823: 3m50s of 5m0s: 844582 executions, 523282 successful
    rsg_test.go:823: 3m55s of 5m0s: 863086 executions, 534805 successful
    rsg_test.go:823: 4m0s of 5m0s: 881786 executions, 546283 successful
    rsg_test.go:823: 4m5s of 5m0s: 900497 executions, 557866 successful
    rsg_test.go:823: 4m10s of 5m0s: 919155 executions, 569453 successful
    rsg_test.go:823: 4m15s of 5m0s: 937107 executions, 580600 successful
    rsg_test.go:823: 4m20s of 5m0s: 955778 executions, 592139 successful
    rsg_test.go:823: 4m25s of 5m0s: 974837 executions, 603997 successful
    rsg_test.go:823: 4m30s of 5m0s: 993578 executions, 615667 successful
    rsg_test.go:823: 4m35s of 5m0s: 1012391 executions, 627339 successful
    rsg_test.go:823: 4m40s of 5m0s: 1030544 executions, 638597 successful
    rsg_test.go:823: 4m45s of 5m0s: 1049126 executions, 650090 successful
    rsg_test.go:823: 4m50s of 5m0s: 1067973 executions, 661723 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7448356?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7448356?buildTab=artifacts#/) on master @ [d136c1fa50fa507c2232d7b215d98b88a83649c5](https://github.com/cockroachdb/cockroach/commits/d136c1fa50fa507c2232d7b215d98b88a83649c5):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5433003 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 45s of 5m0s: 158691 executions, 98168 successful
    rsg_test.go:823: 50s of 5m0s: 177176 executions, 109662 successful
    rsg_test.go:823: 55s of 5m0s: 195210 executions, 120754 successful
    rsg_test.go:823: 1m0s of 5m0s: 213305 executions, 131939 successful
    rsg_test.go:823: 1m5s of 5m0s: 230238 executions, 142383 successful
    rsg_test.go:823: 1m10s of 5m0s: 248203 executions, 153563 successful
    rsg_test.go:823: 1m15s of 5m0s: 266535 executions, 164897 successful
    rsg_test.go:823: 1m20s of 5m0s: 284720 executions, 176075 successful
    rsg_test.go:823: 1m25s of 5m0s: 302941 executions, 187375 successful
    rsg_test.go:823: 1m30s of 5m0s: 321031 executions, 198493 successful
    rsg_test.go:823: 1m35s of 5m0s: 339566 executions, 210038 successful
    rsg_test.go:823: 1m40s of 5m0s: 356929 executions, 220732 successful
    rsg_test.go:823: 1m45s of 5m0s: 375501 executions, 232224 successful
    rsg_test.go:823: 1m50s of 5m0s: 394098 executions, 243814 successful
    rsg_test.go:823: 1m55s of 5m0s: 412361 executions, 255037 successful
    rsg_test.go:823: 2m0s of 5m0s: 430522 executions, 266333 successful
    rsg_test.go:823: 2m5s of 5m0s: 449113 executions, 277892 successful
    rsg_test.go:823: 2m10s of 5m0s: 467715 executions, 289419 successful
    rsg_test.go:823: 2m15s of 5m0s: 485270 executions, 300263 successful
    rsg_test.go:823: 2m20s of 5m0s: 503793 executions, 311730 successful
    rsg_test.go:823: 2m25s of 5m0s: 522645 executions, 323326 successful
    rsg_test.go:823: 2m30s of 5m0s: 541674 executions, 335146 successful
    rsg_test.go:823: 2m35s of 5m0s: 560757 executions, 346974 successful
    rsg_test.go:823: 2m40s of 5m0s: 578521 executions, 357922 successful
    rsg_test.go:823: 2m45s of 5m0s: 596771 executions, 369191 successful
    rsg_test.go:823: 2m50s of 5m0s: 615216 executions, 380621 successful
    rsg_test.go:823: 2m55s of 5m0s: 634150 executions, 392420 successful
    rsg_test.go:823: 3m0s of 5m0s: 652662 executions, 403904 successful
    rsg_test.go:823: 3m5s of 5m0s: 671127 executions, 415292 successful
    rsg_test.go:823: 3m10s of 5m0s: 688993 executions, 426578 successful
    rsg_test.go:823: 3m15s of 5m0s: 707963 executions, 438243 successful
    rsg_test.go:823: 3m20s of 5m0s: 726754 executions, 449877 successful
    rsg_test.go:823: 3m25s of 5m0s: 745651 executions, 461643 successful
    rsg_test.go:823: 3m30s of 5m0s: 763906 executions, 472857 successful
    rsg_test.go:823: 3m35s of 5m0s: 782344 executions, 484355 successful
    rsg_test.go:823: 3m40s of 5m0s: 800575 executions, 495727 successful
    rsg_test.go:823: 3m45s of 5m0s: 818699 executions, 506804 successful
    rsg_test.go:823: 3m50s of 5m0s: 837140 executions, 518264 successful
    rsg_test.go:823: 3m55s of 5m0s: 855597 executions, 529721 successful
    rsg_test.go:823: 4m0s of 5m0s: 873902 executions, 541004 successful
    rsg_test.go:823: 4m5s of 5m0s: 892787 executions, 552682 successful
    rsg_test.go:823: 4m10s of 5m0s: 911440 executions, 564315 successful
    rsg_test.go:823: 4m15s of 5m0s: 929383 executions, 575428 successful
    rsg_test.go:823: 4m20s of 5m0s: 948197 executions, 586958 successful
    rsg_test.go:823: 4m25s of 5m0s: 966742 executions, 598395 successful
    rsg_test.go:823: 4m30s of 5m0s: 985508 executions, 610043 successful
    rsg_test.go:823: 4m35s of 5m0s: 1004331 executions, 621746 successful
    rsg_test.go:823: 4m40s of 5m0s: 1022817 executions, 633159 successful
    rsg_test.go:823: 4m45s of 5m0s: 1041431 executions, 644687 successful
    rsg_test.go:823: 4m50s of 5m0s: 1060411 executions, 656498 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7466971?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7466971?buildTab=artifacts#/) on master @ [fb4014a31b9b8d8235dc48de52196e64b185f490](https://github.com/cockroachdb/cockroach/commits/fb4014a31b9b8d8235dc48de52196e64b185f490):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 3290179 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions2139726638
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 17453 executions, 10857 successful
    rsg_test.go:823: 10s of 5m0s: 35831 executions, 22249 successful
    rsg_test.go:823: 15s of 5m0s: 54235 executions, 33646 successful
    rsg_test.go:823: 20s of 5m0s: 72974 executions, 45275 successful
    rsg_test.go:823: 25s of 5m0s: 91807 executions, 56920 successful
    rsg_test.go:823: 30s of 5m0s: 109940 executions, 68105 successful
    rsg_test.go:823: 35s of 5m0s: 128814 executions, 79761 successful
    rsg_test.go:823: 40s of 5m0s: 147577 executions, 91322 successful
    rsg_test.go:823: 45s of 5m0s: 166680 executions, 102999 successful
    rsg_test.go:823: 50s of 5m0s: 185606 executions, 114756 successful
    rsg_test.go:823: 55s of 5m0s: 204415 executions, 126338 successful
    rsg_test.go:823: 1m0s of 5m0s: 223058 executions, 137806 successful
    rsg_test.go:823: 1m5s of 5m0s: 240455 executions, 148629 successful
    rsg_test.go:823: 1m10s of 5m0s: 259378 executions, 160361 successful
    rsg_test.go:823: 1m15s of 5m0s: 278287 executions, 171991 successful
    rsg_test.go:823: 1m20s of 5m0s: 297293 executions, 183823 successful
    rsg_test.go:823: 1m25s of 5m0s: 316125 executions, 195516 successful
    rsg_test.go:823: 1m30s of 5m0s: 335064 executions, 207245 successful
    rsg_test.go:823: 1m35s of 5m0s: 353074 executions, 218413 successful
    rsg_test.go:823: 1m40s of 5m0s: 372038 executions, 230157 successful
    rsg_test.go:823: 1m45s of 5m0s: 391399 executions, 242203 successful
    rsg_test.go:823: 1m50s of 5m0s: 410503 executions, 253983 successful
    rsg_test.go:823: 1m55s of 5m0s: 429586 executions, 265812 successful
    rsg_test.go:823: 2m0s of 5m0s: 448593 executions, 277559 successful
    rsg_test.go:823: 2m5s of 5m0s: 467805 executions, 289407 successful
    rsg_test.go:823: 2m10s of 5m0s: 486192 executions, 300871 successful
    rsg_test.go:823: 2m15s of 5m0s: 505595 executions, 312866 successful
    rsg_test.go:823: 2m20s of 5m0s: 524594 executions, 324661 successful
    rsg_test.go:823: 2m25s of 5m0s: 544003 executions, 336792 successful
    rsg_test.go:823: 2m30s of 5m0s: 563167 executions, 348583 successful
    rsg_test.go:823: 2m35s of 5m0s: 581664 executions, 359984 successful
    rsg_test.go:823: 2m40s of 5m0s: 600649 executions, 371773 successful
    rsg_test.go:823: 2m45s of 5m0s: 619709 executions, 383572 successful
    rsg_test.go:823: 2m50s of 5m0s: 638908 executions, 395501 successful
    rsg_test.go:823: 2m55s of 5m0s: 658325 executions, 407443 successful
    rsg_test.go:823: 3m0s of 5m0s: 677684 executions, 419594 successful
    rsg_test.go:823: 3m5s of 5m0s: 697178 executions, 431643 successful
    rsg_test.go:823: 3m10s of 5m0s: 715309 executions, 442845 successful
    rsg_test.go:823: 3m15s of 5m0s: 734666 executions, 454774 successful
    rsg_test.go:823: 3m20s of 5m0s: 753532 executions, 466445 successful
    rsg_test.go:823: 3m25s of 5m0s: 772259 executions, 478080 successful
    rsg_test.go:823: 3m30s of 5m0s: 791342 executions, 489912 successful
    rsg_test.go:823: 3m35s of 5m0s: 809635 executions, 501039 successful
    rsg_test.go:823: 3m40s of 5m0s: 828956 executions, 513087 successful
    rsg_test.go:823: 3m45s of 5m0s: 848247 executions, 525070 successful
    rsg_test.go:823: 3m50s of 5m0s: 866981 executions, 536641 successful
    rsg_test.go:823: 3m55s of 5m0s: 886006 executions, 548419 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7478523?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7478523?buildTab=artifacts#/) on master @ [af18b5d2f8ec09d6f3c3092be9ef5fe3f460724d](https://github.com/cockroachdb/cockroach/commits/af18b5d2f8ec09d6f3c3092be9ef5fe3f460724d):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 3709138 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1359155665
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 17473 executions, 10876 successful
    rsg_test.go:823: 10s of 5m0s: 35962 executions, 22420 successful
    rsg_test.go:823: 15s of 5m0s: 54546 executions, 33874 successful
    rsg_test.go:823: 20s of 5m0s: 73411 executions, 45537 successful
    rsg_test.go:823: 25s of 5m0s: 92502 executions, 57375 successful
    rsg_test.go:823: 30s of 5m0s: 110832 executions, 68758 successful
    rsg_test.go:823: 35s of 5m0s: 130152 executions, 80749 successful
    rsg_test.go:823: 40s of 5m0s: 149469 executions, 92794 successful
    rsg_test.go:823: 45s of 5m0s: 168797 executions, 104610 successful
    rsg_test.go:823: 50s of 5m0s: 187985 executions, 116523 successful
    rsg_test.go:823: 55s of 5m0s: 207180 executions, 128420 successful
    rsg_test.go:823: 1m0s of 5m0s: 225576 executions, 139828 successful
    rsg_test.go:823: 1m5s of 5m0s: 244542 executions, 151569 successful
    rsg_test.go:823: 1m10s of 5m0s: 263939 executions, 163477 successful
    rsg_test.go:823: 1m15s of 5m0s: 283330 executions, 175550 successful
    rsg_test.go:823: 1m20s of 5m0s: 302723 executions, 187527 successful
    rsg_test.go:823: 1m25s of 5m0s: 321647 executions, 199179 successful
    rsg_test.go:823: 1m30s of 5m0s: 339941 executions, 210456 successful
    rsg_test.go:823: 1m35s of 5m0s: 358977 executions, 222200 successful
    rsg_test.go:823: 1m40s of 5m0s: 378696 executions, 234421 successful
    rsg_test.go:823: 1m45s of 5m0s: 397879 executions, 246324 successful
    rsg_test.go:823: 1m50s of 5m0s: 417202 executions, 258326 successful
    rsg_test.go:823: 1m55s of 5m0s: 436972 executions, 270472 successful
    rsg_test.go:823: 2m0s of 5m0s: 455565 executions, 281955 successful
    rsg_test.go:823: 2m5s of 5m0s: 474710 executions, 293812 successful
    rsg_test.go:823: 2m10s of 5m0s: 494197 executions, 305877 successful
    rsg_test.go:823: 2m15s of 5m0s: 513529 executions, 317904 successful
    rsg_test.go:823: 2m20s of 5m0s: 532835 executions, 329954 successful
    rsg_test.go:823: 2m25s of 5m0s: 551601 executions, 341613 successful
    rsg_test.go:823: 2m30s of 5m0s: 571080 executions, 353633 successful
    rsg_test.go:823: 2m35s of 5m0s: 590867 executions, 365877 successful
    rsg_test.go:823: 2m40s of 5m0s: 610629 executions, 378196 successful
    rsg_test.go:823: 2m45s of 5m0s: 630415 executions, 390427 successful
    rsg_test.go:823: 2m50s of 5m0s: 650001 executions, 402675 successful
    rsg_test.go:823: 2m55s of 5m0s: 669496 executions, 414711 successful
    rsg_test.go:823: 3m0s of 5m0s: 688219 executions, 426293 successful
    rsg_test.go:823: 3m5s of 5m0s: 707540 executions, 438411 successful
    rsg_test.go:823: 3m10s of 5m0s: 726935 executions, 450338 successful
    rsg_test.go:823: 3m15s of 5m0s: 746602 executions, 462632 successful
    rsg_test.go:823: 3m20s of 5m0s: 766134 executions, 474667 successful
    rsg_test.go:823: 3m25s of 5m0s: 785809 executions, 486982 successful
    rsg_test.go:823: 3m30s of 5m0s: 805521 executions, 499125 successful
    rsg_test.go:823: 3m35s of 5m0s: 824071 executions, 510607 successful
    rsg_test.go:823: 3m40s of 5m0s: 843370 executions, 522600 successful
    rsg_test.go:823: 3m45s of 5m0s: 862809 executions, 534534 successful
    rsg_test.go:823: 3m50s of 5m0s: 882266 executions, 546624 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7490755?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7490755?buildTab=artifacts#/) on master @ [7f77f57a768a1bc488629234126e9e456bb6ddbb](https://github.com/cockroachdb/cockroach/commits/7f77f57a768a1bc488629234126e9e456bb6ddbb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 2589711 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions383803089
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 16799 executions, 10396 successful
    rsg_test.go:823: 10s of 5m0s: 34436 executions, 21338 successful
    rsg_test.go:823: 15s of 5m0s: 52183 executions, 32351 successful
    rsg_test.go:823: 20s of 5m0s: 70272 executions, 43577 successful
    rsg_test.go:823: 25s of 5m0s: 88315 executions, 54743 successful
    rsg_test.go:823: 30s of 5m0s: 105817 executions, 65590 successful
    rsg_test.go:823: 35s of 5m0s: 123859 executions, 76724 successful
    rsg_test.go:823: 40s of 5m0s: 142087 executions, 88062 successful
    rsg_test.go:823: 45s of 5m0s: 160440 executions, 99484 successful
    rsg_test.go:823: 50s of 5m0s: 178551 executions, 110631 successful
    rsg_test.go:823: 55s of 5m0s: 196807 executions, 121924 successful
    rsg_test.go:823: 1m0s of 5m0s: 214262 executions, 132838 successful
    rsg_test.go:823: 1m5s of 5m0s: 232276 executions, 143835 successful
    rsg_test.go:823: 1m10s of 5m0s: 250568 executions, 155139 successful
    rsg_test.go:823: 1m15s of 5m0s: 268805 executions, 166373 successful
    rsg_test.go:823: 1m20s of 5m0s: 287053 executions, 177688 successful
    rsg_test.go:823: 1m25s of 5m0s: 305297 executions, 188971 successful
    rsg_test.go:823: 1m30s of 5m0s: 322897 executions, 199807 successful
    rsg_test.go:823: 1m35s of 5m0s: 341241 executions, 211242 successful
    rsg_test.go:823: 1m40s of 5m0s: 359636 executions, 222665 successful
    rsg_test.go:823: 1m45s of 5m0s: 377650 executions, 233698 successful
    rsg_test.go:823: 1m50s of 5m0s: 396003 executions, 245086 successful
    rsg_test.go:823: 1m55s of 5m0s: 413726 executions, 256087 successful
    rsg_test.go:823: 2m0s of 5m0s: 431955 executions, 267321 successful
    rsg_test.go:823: 2m5s of 5m0s: 450370 executions, 278662 successful
    rsg_test.go:823: 2m10s of 5m0s: 468888 executions, 290126 successful
    rsg_test.go:823: 2m15s of 5m0s: 487151 executions, 301446 successful
    rsg_test.go:823: 2m20s of 5m0s: 505489 executions, 312778 successful
    rsg_test.go:823: 2m25s of 5m0s: 522754 executions, 323417 successful
    rsg_test.go:823: 2m30s of 5m0s: 541107 executions, 334823 successful
    rsg_test.go:823: 2m35s of 5m0s: 559500 executions, 346252 successful
    rsg_test.go:823: 2m40s of 5m0s: 578034 executions, 357686 successful
    rsg_test.go:823: 2m45s of 5m0s: 596822 executions, 369337 successful
    rsg_test.go:823: 2m50s of 5m0s: 615476 executions, 380918 successful
    rsg_test.go:823: 2m55s of 5m0s: 633041 executions, 391903 successful
    rsg_test.go:823: 3m0s of 5m0s: 651351 executions, 403173 successful
    rsg_test.go:823: 3m5s of 5m0s: 669668 executions, 414574 successful
    rsg_test.go:823: 3m10s of 5m0s: 688165 executions, 426020 successful
    rsg_test.go:823: 3m15s of 5m0s: 706704 executions, 437480 successful
    rsg_test.go:823: 3m20s of 5m0s: 725092 executions, 448830 successful
    rsg_test.go:823: 3m25s of 5m0s: 742866 executions, 459909 successful
    rsg_test.go:823: 3m30s of 5m0s: 761522 executions, 471442 successful
    rsg_test.go:823: 3m35s of 5m0s: 780103 executions, 482968 successful
    rsg_test.go:823: 3m40s of 5m0s: 798265 executions, 494124 successful
    rsg_test.go:823: 3m45s of 5m0s: 816628 executions, 505504 successful
    rsg_test.go:823: 3m50s of 5m0s: 834286 executions, 516462 successful
    rsg_test.go:823: 3m55s of 5m0s: 853035 executions, 528130 successful
    rsg_test.go:823: 4m0s of 5m0s: 871399 executions, 539539 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7512764?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7512764?buildTab=artifacts#/) on master @ [47c7b3a1bc047fc3e481cf12166885b39519c022](https://github.com/cockroachdb/cockroach/commits/47c7b3a1bc047fc3e481cf12166885b39519c022):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4552445 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 30s of 5m0s: 113769 executions, 70571 successful
    rsg_test.go:823: 35s of 5m0s: 131829 executions, 81632 successful
    rsg_test.go:823: 40s of 5m0s: 151220 executions, 93738 successful
    rsg_test.go:823: 45s of 5m0s: 170669 executions, 105670 successful
    rsg_test.go:823: 50s of 5m0s: 189913 executions, 117540 successful
    rsg_test.go:823: 55s of 5m0s: 209067 executions, 129403 successful
    rsg_test.go:823: 1m0s of 5m0s: 228121 executions, 141163 successful
    rsg_test.go:823: 1m5s of 5m0s: 247302 executions, 153053 successful
    rsg_test.go:823: 1m10s of 5m0s: 266265 executions, 164840 successful
    rsg_test.go:823: 1m15s of 5m0s: 285393 executions, 176685 successful
    rsg_test.go:823: 1m20s of 5m0s: 304737 executions, 188581 successful
    rsg_test.go:823: 1m25s of 5m0s: 323885 executions, 200365 successful
    rsg_test.go:823: 1m30s of 5m0s: 342672 executions, 212088 successful
    rsg_test.go:823: 1m35s of 5m0s: 362022 executions, 224045 successful
    rsg_test.go:823: 1m40s of 5m0s: 381405 executions, 235998 successful
    rsg_test.go:823: 1m45s of 5m0s: 400885 executions, 248131 successful
    rsg_test.go:823: 1m50s of 5m0s: 420239 executions, 260096 successful
    rsg_test.go:823: 1m55s of 5m0s: 438805 executions, 271606 successful
    rsg_test.go:823: 2m0s of 5m0s: 458414 executions, 283819 successful
    rsg_test.go:823: 2m5s of 5m0s: 477862 executions, 295780 successful
    rsg_test.go:823: 2m10s of 5m0s: 497459 executions, 307986 successful
    rsg_test.go:823: 2m15s of 5m0s: 517214 executions, 320250 successful
    rsg_test.go:823: 2m20s of 5m0s: 536411 executions, 332221 successful
    rsg_test.go:823: 2m25s of 5m0s: 555935 executions, 344259 successful
    rsg_test.go:823: 2m30s of 5m0s: 575752 executions, 356521 successful
    rsg_test.go:823: 2m35s of 5m0s: 595120 executions, 368453 successful
    rsg_test.go:823: 2m40s of 5m0s: 614799 executions, 380663 successful
    rsg_test.go:823: 2m45s of 5m0s: 634416 executions, 392681 successful
    rsg_test.go:823: 2m50s of 5m0s: 654015 executions, 404922 successful
    rsg_test.go:823: 2m55s of 5m0s: 673143 executions, 416791 successful
    rsg_test.go:823: 3m0s of 5m0s: 692894 executions, 429096 successful
    rsg_test.go:823: 3m5s of 5m0s: 712813 executions, 441386 successful
    rsg_test.go:823: 3m10s of 5m0s: 732753 executions, 453757 successful
    rsg_test.go:823: 3m15s of 5m0s: 752386 executions, 465924 successful
    rsg_test.go:823: 3m20s of 5m0s: 771479 executions, 477627 successful
    rsg_test.go:823: 3m25s of 5m0s: 791388 executions, 490100 successful
    rsg_test.go:823: 3m30s of 5m0s: 811226 executions, 502301 successful
    rsg_test.go:823: 3m35s of 5m0s: 830697 executions, 514410 successful
    rsg_test.go:823: 3m40s of 5m0s: 850279 executions, 526425 successful
    rsg_test.go:823: 3m45s of 5m0s: 869702 executions, 538513 successful
    rsg_test.go:823: 3m50s of 5m0s: 889443 executions, 550680 successful
    rsg_test.go:823: 3m55s of 5m0s: 908581 executions, 562669 successful
    rsg_test.go:823: 4m0s of 5m0s: 927873 executions, 574486 successful
    rsg_test.go:823: 4m5s of 5m0s: 947574 executions, 586792 successful
    rsg_test.go:823: 4m10s of 5m0s: 967122 executions, 598795 successful
    rsg_test.go:823: 4m15s of 5m0s: 986713 executions, 610960 successful
    rsg_test.go:823: 4m20s of 5m0s: 1005531 executions, 622580 successful
    rsg_test.go:823: 4m25s of 5m0s: 1025042 executions, 634564 successful
    rsg_test.go:823: 4m30s of 5m0s: 1044658 executions, 646754 successful
    rsg_test.go:823: 4m35s of 5m0s: 1064028 executions, 658748 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7534910?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7534910?buildTab=artifacts#/) on master @ [d98d195dfa7a3480ce3e07657aa870692fe71cea](https://github.com/cockroachdb/cockroach/commits/d98d195dfa7a3480ce3e07657aa870692fe71cea):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5316707 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 45s of 5m0s: 159818 executions, 99013 successful
    rsg_test.go:823: 50s of 5m0s: 178139 executions, 110378 successful
    rsg_test.go:823: 55s of 5m0s: 196354 executions, 121741 successful
    rsg_test.go:823: 1m0s of 5m0s: 214605 executions, 133112 successful
    rsg_test.go:823: 1m5s of 5m0s: 231559 executions, 143547 successful
    rsg_test.go:823: 1m10s of 5m0s: 249263 executions, 154402 successful
    rsg_test.go:823: 1m15s of 5m0s: 267567 executions, 165745 successful
    rsg_test.go:823: 1m20s of 5m0s: 285322 executions, 176873 successful
    rsg_test.go:823: 1m25s of 5m0s: 303177 executions, 187859 successful
    rsg_test.go:823: 1m30s of 5m0s: 321363 executions, 199138 successful
    rsg_test.go:823: 1m35s of 5m0s: 339455 executions, 210439 successful
    rsg_test.go:823: 1m40s of 5m0s: 356726 executions, 220989 successful
    rsg_test.go:823: 1m45s of 5m0s: 374490 executions, 232116 successful
    rsg_test.go:823: 1m50s of 5m0s: 392362 executions, 243156 successful
    rsg_test.go:823: 1m55s of 5m0s: 410046 executions, 254241 successful
    rsg_test.go:823: 2m0s of 5m0s: 427933 executions, 265252 successful
    rsg_test.go:823: 2m5s of 5m0s: 446245 executions, 276564 successful
    rsg_test.go:823: 2m10s of 5m0s: 464387 executions, 287834 successful
    rsg_test.go:823: 2m15s of 5m0s: 481329 executions, 298217 successful
    rsg_test.go:823: 2m20s of 5m0s: 499566 executions, 309618 successful
    rsg_test.go:823: 2m25s of 5m0s: 518051 executions, 321141 successful
    rsg_test.go:823: 2m30s of 5m0s: 536128 executions, 332273 successful
    rsg_test.go:823: 2m35s of 5m0s: 554536 executions, 343671 successful
    rsg_test.go:823: 2m40s of 5m0s: 572252 executions, 354767 successful
    rsg_test.go:823: 2m45s of 5m0s: 590228 executions, 365928 successful
    rsg_test.go:823: 2m50s of 5m0s: 608329 executions, 377176 successful
    rsg_test.go:823: 2m55s of 5m0s: 626523 executions, 388421 successful
    rsg_test.go:823: 3m0s of 5m0s: 644596 executions, 399714 successful
    rsg_test.go:823: 3m5s of 5m0s: 662507 executions, 410726 successful
    rsg_test.go:823: 3m10s of 5m0s: 679633 executions, 421360 successful
    rsg_test.go:823: 3m15s of 5m0s: 697649 executions, 432608 successful
    rsg_test.go:823: 3m20s of 5m0s: 716049 executions, 444039 successful
    rsg_test.go:823: 3m25s of 5m0s: 734270 executions, 455399 successful
    rsg_test.go:823: 3m30s of 5m0s: 752657 executions, 466796 successful
    rsg_test.go:823: 3m35s of 5m0s: 770935 executions, 478041 successful
    rsg_test.go:823: 3m40s of 5m0s: 788323 executions, 488821 successful
    rsg_test.go:823: 3m45s of 5m0s: 806328 executions, 499872 successful
    rsg_test.go:823: 3m50s of 5m0s: 824336 executions, 511109 successful
    rsg_test.go:823: 3m55s of 5m0s: 842286 executions, 522260 successful
    rsg_test.go:823: 4m0s of 5m0s: 860048 executions, 533191 successful
    rsg_test.go:823: 4m5s of 5m0s: 878412 executions, 544622 successful
    rsg_test.go:823: 4m10s of 5m0s: 896186 executions, 555672 successful
    rsg_test.go:823: 4m15s of 5m0s: 913447 executions, 566276 successful
    rsg_test.go:823: 4m20s of 5m0s: 931438 executions, 577535 successful
    rsg_test.go:823: 4m25s of 5m0s: 949498 executions, 588628 successful
    rsg_test.go:823: 4m30s of 5m0s: 967605 executions, 599842 successful
    rsg_test.go:823: 4m35s of 5m0s: 985703 executions, 611091 successful
    rsg_test.go:823: 4m40s of 5m0s: 1003719 executions, 622284 successful
    rsg_test.go:823: 4m45s of 5m0s: 1021479 executions, 633245 successful
    rsg_test.go:823: 4m50s of 5m0s: 1039594 executions, 644543 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7555555?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7555555?buildTab=artifacts#/) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 3225338 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 17889 executions, 10981 successful
    rsg_test.go:823: 10s of 5m0s: 35869 executions, 22187 successful
    rsg_test.go:823: 15s of 5m0s: 54129 executions, 33466 successful
    rsg_test.go:823: 20s of 5m0s: 72122 executions, 44570 successful
    rsg_test.go:823: 25s of 5m0s: 90059 executions, 55658 successful
    rsg_test.go:823: 30s of 5m0s: 107672 executions, 66435 successful
    rsg_test.go:823: 35s of 5m0s: 126253 executions, 77994 successful
    rsg_test.go:823: 40s of 5m0s: 144788 executions, 89479 successful
    rsg_test.go:823: 45s of 5m0s: 162703 executions, 100615 successful
    rsg_test.go:823: 50s of 5m0s: 181242 executions, 112050 successful
    rsg_test.go:823: 55s of 5m0s: 199821 executions, 123529 successful
    rsg_test.go:823: 1m0s of 5m0s: 217441 executions, 134382 successful
    rsg_test.go:823: 1m5s of 5m0s: 235692 executions, 145636 successful
    rsg_test.go:823: 1m10s of 5m0s: 253984 executions, 157058 successful
    rsg_test.go:823: 1m15s of 5m0s: 272486 executions, 168552 successful
    rsg_test.go:823: 1m20s of 5m0s: 291006 executions, 180015 successful
    rsg_test.go:823: 1m25s of 5m0s: 309312 executions, 191263 successful
    rsg_test.go:823: 1m30s of 5m0s: 327297 executions, 202387 successful
    rsg_test.go:823: 1m35s of 5m0s: 345801 executions, 213793 successful
    rsg_test.go:823: 1m40s of 5m0s: 363867 executions, 225036 successful
    rsg_test.go:823: 1m45s of 5m0s: 382442 executions, 236578 successful
    rsg_test.go:823: 1m50s of 5m0s: 401364 executions, 248210 successful
    rsg_test.go:823: 1m55s of 5m0s: 420291 executions, 259893 successful
    rsg_test.go:823: 2m0s of 5m0s: 438650 executions, 271215 successful
    rsg_test.go:823: 2m5s of 5m0s: 456989 executions, 282675 successful
    rsg_test.go:823: 2m10s of 5m0s: 475623 executions, 294229 successful
    rsg_test.go:823: 2m15s of 5m0s: 494488 executions, 305881 successful
    rsg_test.go:823: 2m20s of 5m0s: 513378 executions, 317603 successful
    rsg_test.go:823: 2m25s of 5m0s: 532432 executions, 329419 successful
    rsg_test.go:823: 2m30s of 5m0s: 551855 executions, 341527 successful
    rsg_test.go:823: 2m35s of 5m0s: 569759 executions, 352543 successful
    rsg_test.go:823: 2m40s of 5m0s: 588199 executions, 363903 successful
    rsg_test.go:823: 2m45s of 5m0s: 607051 executions, 375658 successful
    rsg_test.go:823: 2m50s of 5m0s: 625726 executions, 387275 successful
    rsg_test.go:823: 2m55s of 5m0s: 644711 executions, 399083 successful
    rsg_test.go:823: 3m0s of 5m0s: 663119 executions, 410507 successful
    rsg_test.go:823: 3m5s of 5m0s: 681865 executions, 422063 successful
    rsg_test.go:823: 3m10s of 5m0s: 699508 executions, 432897 successful
    rsg_test.go:823: 3m15s of 5m0s: 718177 executions, 444375 successful
    rsg_test.go:823: 3m20s of 5m0s: 737099 executions, 456095 successful
    rsg_test.go:823: 3m25s of 5m0s: 756049 executions, 467922 successful
    rsg_test.go:823: 3m30s of 5m0s: 774791 executions, 479533 successful
    rsg_test.go:823: 3m35s of 5m0s: 792518 executions, 490422 successful
    rsg_test.go:823: 3m40s of 5m0s: 811303 executions, 502078 successful
    rsg_test.go:823: 3m45s of 5m0s: 830132 executions, 513689 successful
    rsg_test.go:823: 3m50s of 5m0s: 848975 executions, 525353 successful
    rsg_test.go:823: 3m55s of 5m0s: 867370 executions, 536736 successful
    rsg_test.go:823: 4m0s of 5m0s: 886115 executions, 548323 successful
    rsg_test.go:823: 4m5s of 5m0s: 904149 executions, 559518 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7593510?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7593510?buildTab=artifacts#/) on master @ [8357abb668a5adaff781343b394b162fb1b66c6e](https://github.com/cockroachdb/cockroach/commits/8357abb668a5adaff781343b394b162fb1b66c6e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4949224 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions970215784
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:823: 5s of 5m0s: 17978 executions, 11124 successful
    rsg_test.go:823: 10s of 5m0s: 36742 executions, 22842 successful
    rsg_test.go:823: 15s of 5m0s: 55326 executions, 34293 successful
    rsg_test.go:823: 20s of 5m0s: 73828 executions, 45746 successful
    rsg_test.go:823: 25s of 5m0s: 92745 executions, 57565 successful
    rsg_test.go:823: 30s of 5m0s: 111537 executions, 69181 successful
    rsg_test.go:823: 35s of 5m0s: 129896 executions, 80575 successful
    rsg_test.go:823: 40s of 5m0s: 148679 executions, 92158 successful
    rsg_test.go:823: 45s of 5m0s: 167612 executions, 103970 successful
    rsg_test.go:823: 50s of 5m0s: 186752 executions, 115905 successful
    rsg_test.go:823: 55s of 5m0s: 205482 executions, 127502 successful
    rsg_test.go:823: 1m0s of 5m0s: 224303 executions, 139045 successful
    rsg_test.go:823: 1m5s of 5m0s: 243402 executions, 150947 successful
    rsg_test.go:823: 1m10s of 5m0s: 261050 executions, 161942 successful
    rsg_test.go:823: 1m15s of 5m0s: 279894 executions, 173556 successful
    rsg_test.go:823: 1m20s of 5m0s: 298909 executions, 185262 successful
    rsg_test.go:823: 1m25s of 5m0s: 317874 executions, 197020 successful
    rsg_test.go:823: 1m30s of 5m0s: 337094 executions, 208903 successful
    rsg_test.go:823: 1m35s of 5m0s: 355532 executions, 220362 successful
    rsg_test.go:823: 1m40s of 5m0s: 374635 executions, 232246 successful
    rsg_test.go:823: 1m45s of 5m0s: 393959 executions, 244130 successful
    rsg_test.go:823: 1m50s of 5m0s: 413107 executions, 256052 successful
    rsg_test.go:823: 1m55s of 5m0s: 432509 executions, 268111 successful
    rsg_test.go:823: 2m0s of 5m0s: 451497 executions, 279739 successful
    rsg_test.go:823: 2m5s of 5m0s: 470308 executions, 291377 successful
    rsg_test.go:823: 2m10s of 5m0s: 489509 executions, 303258 successful
    rsg_test.go:823: 2m15s of 5m0s: 508787 executions, 315230 successful
    rsg_test.go:823: 2m20s of 5m0s: 528341 executions, 327239 successful
    rsg_test.go:823: 2m25s of 5m0s: 547452 executions, 339115 successful
    rsg_test.go:823: 2m30s of 5m0s: 566609 executions, 350986 successful
    rsg_test.go:823: 2m35s of 5m0s: 585923 executions, 362951 successful
    rsg_test.go:823: 2m40s of 5m0s: 604081 executions, 374224 successful
    rsg_test.go:823: 2m45s of 5m0s: 623575 executions, 386362 successful
    rsg_test.go:823: 2m50s of 5m0s: 642664 executions, 398268 successful
    rsg_test.go:823: 2m55s of 5m0s: 661929 executions, 410035 successful
    rsg_test.go:823: 3m0s of 5m0s: 681216 executions, 422008 successful
    rsg_test.go:823: 3m5s of 5m0s: 699698 executions, 433467 successful
    rsg_test.go:823: 3m10s of 5m0s: 719340 executions, 445703 successful
    rsg_test.go:823: 3m15s of 5m0s: 738693 executions, 457616 successful
    rsg_test.go:823: 3m20s of 5m0s: 758254 executions, 469761 successful
    rsg_test.go:823: 3m25s of 5m0s: 777901 executions, 481900 successful
    rsg_test.go:823: 3m30s of 5m0s: 797277 executions, 493939 successful
    rsg_test.go:823: 3m35s of 5m0s: 816701 executions, 505846 successful
    rsg_test.go:823: 3m40s of 5m0s: 834891 executions, 517197 successful
    rsg_test.go:823: 3m45s of 5m0s: 854542 executions, 529347 successful
    rsg_test.go:823: 3m50s of 5m0s: 874224 executions, 541477 successful
    rsg_test.go:823: 3m55s of 5m0s: 893549 executions, 553471 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7605135?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7605135?buildTab=artifacts#/) on master @ [cfb5ae9a96e1770daa4aef1615a46e212b561a84](https://github.com/cockroachdb/cockroach/commits/cfb5ae9a96e1770daa4aef1615a46e212b561a84):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4354982 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 5s of 5m0s: 17044 executions, 10612 successful
    rsg_test.go:823: 10s of 5m0s: 34603 executions, 21388 successful
    rsg_test.go:823: 15s of 5m0s: 52286 executions, 32373 successful
    rsg_test.go:823: 20s of 5m0s: 70315 executions, 43537 successful
    rsg_test.go:823: 25s of 5m0s: 88777 executions, 54985 successful
    rsg_test.go:823: 30s of 5m0s: 106679 executions, 66053 successful
    rsg_test.go:823: 35s of 5m0s: 123781 executions, 76577 successful
    rsg_test.go:823: 40s of 5m0s: 142035 executions, 87955 successful
    rsg_test.go:823: 45s of 5m0s: 159927 executions, 98971 successful
    rsg_test.go:823: 50s of 5m0s: 178360 executions, 110403 successful
    rsg_test.go:823: 55s of 5m0s: 196661 executions, 121727 successful
    rsg_test.go:823: 1m0s of 5m0s: 214375 executions, 132732 successful
    rsg_test.go:823: 1m5s of 5m0s: 232275 executions, 143690 successful
    rsg_test.go:823: 1m10s of 5m0s: 250871 executions, 155175 successful
    rsg_test.go:823: 1m15s of 5m0s: 269065 executions, 166416 successful
    rsg_test.go:823: 1m20s of 5m0s: 287224 executions, 177734 successful
    rsg_test.go:823: 1m25s of 5m0s: 305426 executions, 189008 successful
    rsg_test.go:823: 1m30s of 5m0s: 323408 executions, 200060 successful
    rsg_test.go:823: 1m35s of 5m0s: 342000 executions, 211556 successful
    rsg_test.go:823: 1m40s of 5m0s: 360444 executions, 222982 successful
    rsg_test.go:823: 1m45s of 5m0s: 378412 executions, 234186 successful
    rsg_test.go:823: 1m50s of 5m0s: 396738 executions, 245556 successful
    rsg_test.go:823: 1m55s of 5m0s: 415060 executions, 256796 successful
    rsg_test.go:823: 2m0s of 5m0s: 432882 executions, 267776 successful
    rsg_test.go:823: 2m5s of 5m0s: 451511 executions, 279384 successful
    rsg_test.go:823: 2m10s of 5m0s: 469719 executions, 290684 successful
    rsg_test.go:823: 2m15s of 5m0s: 488180 executions, 302159 successful
    rsg_test.go:823: 2m20s of 5m0s: 506814 executions, 313732 successful
    rsg_test.go:823: 2m25s of 5m0s: 525134 executions, 325117 successful
    rsg_test.go:823: 2m30s of 5m0s: 543763 executions, 336654 successful
    rsg_test.go:823: 2m35s of 5m0s: 561412 executions, 347558 successful
    rsg_test.go:823: 2m40s of 5m0s: 579961 executions, 359101 successful
    rsg_test.go:823: 2m45s of 5m0s: 598540 executions, 370593 successful
    rsg_test.go:823: 2m50s of 5m0s: 616947 executions, 382039 successful
    rsg_test.go:823: 2m55s of 5m0s: 635690 executions, 393634 successful
    rsg_test.go:823: 3m0s of 5m0s: 653361 executions, 404639 successful
    rsg_test.go:823: 3m5s of 5m0s: 671968 executions, 416194 successful
    rsg_test.go:823: 3m10s of 5m0s: 690441 executions, 427594 successful
    rsg_test.go:823: 3m15s of 5m0s: 708858 executions, 438987 successful
    rsg_test.go:823: 3m20s of 5m0s: 727346 executions, 450425 successful
    rsg_test.go:823: 3m25s of 5m0s: 746006 executions, 461989 successful
    rsg_test.go:823: 3m30s of 5m0s: 764232 executions, 473257 successful
    rsg_test.go:823: 3m35s of 5m0s: 782875 executions, 484734 successful
    rsg_test.go:823: 3m40s of 5m0s: 801311 executions, 496116 successful
    rsg_test.go:823: 3m45s of 5m0s: 819681 executions, 507532 successful
    rsg_test.go:823: 3m50s of 5m0s: 838423 executions, 519146 successful
    rsg_test.go:823: 3m55s of 5m0s: 855990 executions, 530045 successful
    rsg_test.go:823: 4m0s of 5m0s: 874461 executions, 541502 successful
    rsg_test.go:823: 4m5s of 5m0s: 893048 executions, 552997 successful
    rsg_test.go:823: 4m10s of 5m0s: 911894 executions, 564719 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7617005?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7617005?buildTab=artifacts#/) on master @ [f694132d6ed73c707576a28f6e784fda8ea13c63](https://github.com/cockroachdb/cockroach/commits/f694132d6ed73c707576a28f6e784fda8ea13c63):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5685230 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:823: 45s of 5m0s: 168142 executions, 104144 successful
    rsg_test.go:823: 50s of 5m0s: 187593 executions, 116157 successful
    rsg_test.go:823: 55s of 5m0s: 206075 executions, 127595 successful
    rsg_test.go:823: 1m0s of 5m0s: 225285 executions, 139566 successful
    rsg_test.go:823: 1m5s of 5m0s: 244332 executions, 151374 successful
    rsg_test.go:823: 1m10s of 5m0s: 263897 executions, 163526 successful
    rsg_test.go:823: 1m15s of 5m0s: 283314 executions, 175553 successful
    rsg_test.go:823: 1m20s of 5m0s: 302697 executions, 187439 successful
    rsg_test.go:823: 1m25s of 5m0s: 321883 executions, 199294 successful
    rsg_test.go:823: 1m30s of 5m0s: 340265 executions, 210717 successful
    rsg_test.go:823: 1m35s of 5m0s: 359300 executions, 222532 successful
    rsg_test.go:823: 1m40s of 5m0s: 378425 executions, 234378 successful
    rsg_test.go:823: 1m45s of 5m0s: 397475 executions, 245937 successful
    rsg_test.go:823: 1m50s of 5m0s: 416606 executions, 257858 successful
    rsg_test.go:823: 1m55s of 5m0s: 436018 executions, 269947 successful
    rsg_test.go:823: 2m0s of 5m0s: 455363 executions, 281879 successful
    rsg_test.go:823: 2m5s of 5m0s: 473450 executions, 293118 successful
    rsg_test.go:823: 2m10s of 5m0s: 492903 executions, 305249 successful
    rsg_test.go:823: 2m15s of 5m0s: 512199 executions, 317234 successful
    rsg_test.go:823: 2m20s of 5m0s: 531388 executions, 329002 successful
    rsg_test.go:823: 2m25s of 5m0s: 550613 executions, 340989 successful
    rsg_test.go:823: 2m30s of 5m0s: 570114 executions, 353139 successful
    rsg_test.go:823: 2m35s of 5m0s: 589543 executions, 365208 successful
    rsg_test.go:823: 2m40s of 5m0s: 607903 executions, 376565 successful
    rsg_test.go:823: 2m45s of 5m0s: 627242 executions, 388600 successful
    rsg_test.go:823: 2m50s of 5m0s: 646376 executions, 400494 successful
    rsg_test.go:823: 2m55s of 5m0s: 665790 executions, 412498 successful
    rsg_test.go:823: 3m0s of 5m0s: 684832 executions, 424278 successful
    rsg_test.go:823: 3m5s of 5m0s: 703293 executions, 435743 successful
    rsg_test.go:823: 3m10s of 5m0s: 721771 executions, 447173 successful
    rsg_test.go:823: 3m15s of 5m0s: 741200 executions, 459297 successful
    rsg_test.go:823: 3m20s of 5m0s: 760382 executions, 471043 successful
    rsg_test.go:823: 3m25s of 5m0s: 779653 executions, 483028 successful
    rsg_test.go:823: 3m30s of 5m0s: 798874 executions, 495006 successful
    rsg_test.go:823: 3m35s of 5m0s: 818014 executions, 506818 successful
    rsg_test.go:823: 3m40s of 5m0s: 836577 executions, 518320 successful
    rsg_test.go:823: 3m45s of 5m0s: 856019 executions, 530355 successful
    rsg_test.go:823: 3m50s of 5m0s: 875745 executions, 542538 successful
    rsg_test.go:823: 3m55s of 5m0s: 895413 executions, 554868 successful
    rsg_test.go:823: 4m0s of 5m0s: 914868 executions, 566804 successful
    rsg_test.go:823: 4m5s of 5m0s: 934385 executions, 578976 successful
    rsg_test.go:823: 4m10s of 5m0s: 953175 executions, 590583 successful
    rsg_test.go:823: 4m15s of 5m0s: 972773 executions, 602705 successful
    rsg_test.go:823: 4m20s of 5m0s: 992270 executions, 614802 successful
    rsg_test.go:823: 4m25s of 5m0s: 1011577 executions, 626645 successful
    rsg_test.go:823: 4m30s of 5m0s: 1031088 executions, 638772 successful
    rsg_test.go:823: 4m35s of 5m0s: 1050565 executions, 650806 successful
    rsg_test.go:823: 4m40s of 5m0s: 1069369 executions, 662399 successful
    rsg_test.go:823: 4m45s of 5m0s: 1089098 executions, 674754 successful
    rsg_test.go:823: 4m50s of 5m0s: 1108845 executions, 686876 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7655675?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7655675?buildTab=artifacts#/) on master @ [0d9669a8669cf5abe22b61e1f1faf9a520f2f372](https://github.com/cockroachdb/cockroach/commits/0d9669a8669cf5abe22b61e1f1faf9a520f2f372):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5770673 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 45s of 5m0s: 174901 executions, 108356 successful
    rsg_test.go:822: 50s of 5m0s: 194438 executions, 120438 successful
    rsg_test.go:822: 55s of 5m0s: 213909 executions, 132534 successful
    rsg_test.go:822: 1m0s of 5m0s: 232431 executions, 144022 successful
    rsg_test.go:822: 1m5s of 5m0s: 251698 executions, 156085 successful
    rsg_test.go:822: 1m10s of 5m0s: 270924 executions, 167863 successful
    rsg_test.go:822: 1m15s of 5m0s: 290190 executions, 179819 successful
    rsg_test.go:822: 1m20s of 5m0s: 310085 executions, 192153 successful
    rsg_test.go:822: 1m25s of 5m0s: 329475 executions, 204238 successful
    rsg_test.go:822: 1m30s of 5m0s: 347901 executions, 215623 successful
    rsg_test.go:822: 1m35s of 5m0s: 368073 executions, 228136 successful
    rsg_test.go:822: 1m40s of 5m0s: 388118 executions, 240493 successful
    rsg_test.go:822: 1m45s of 5m0s: 408108 executions, 252977 successful
    rsg_test.go:822: 1m50s of 5m0s: 427613 executions, 265097 successful
    rsg_test.go:822: 1m55s of 5m0s: 447531 executions, 277364 successful
    rsg_test.go:822: 2m0s of 5m0s: 466157 executions, 288900 successful
    rsg_test.go:822: 2m5s of 5m0s: 486147 executions, 301360 successful
    rsg_test.go:822: 2m10s of 5m0s: 505584 executions, 313274 successful
    rsg_test.go:822: 2m15s of 5m0s: 525279 executions, 325539 successful
    rsg_test.go:822: 2m20s of 5m0s: 545028 executions, 337726 successful
    rsg_test.go:822: 2m25s of 5m0s: 563984 executions, 349461 successful
    rsg_test.go:822: 2m30s of 5m0s: 583744 executions, 361696 successful
    rsg_test.go:822: 2m35s of 5m0s: 603681 executions, 374201 successful
    rsg_test.go:822: 2m40s of 5m0s: 623865 executions, 386703 successful
    rsg_test.go:822: 2m45s of 5m0s: 644129 executions, 399303 successful
    rsg_test.go:822: 2m50s of 5m0s: 663970 executions, 411577 successful
    rsg_test.go:822: 2m55s of 5m0s: 682994 executions, 423334 successful
    rsg_test.go:822: 3m0s of 5m0s: 702633 executions, 435472 successful
    rsg_test.go:822: 3m5s of 5m0s: 722601 executions, 447927 successful
    rsg_test.go:822: 3m10s of 5m0s: 742609 executions, 460278 successful
    rsg_test.go:822: 3m15s of 5m0s: 761886 executions, 472205 successful
    rsg_test.go:822: 3m20s of 5m0s: 782040 executions, 484753 successful
    rsg_test.go:822: 3m25s of 5m0s: 801674 executions, 496781 successful
    rsg_test.go:822: 3m30s of 5m0s: 820685 executions, 508497 successful
    rsg_test.go:822: 3m35s of 5m0s: 840744 executions, 520935 successful
    rsg_test.go:822: 3m40s of 5m0s: 860532 executions, 533179 successful
    rsg_test.go:822: 3m45s of 5m0s: 880628 executions, 545585 successful
    rsg_test.go:822: 3m50s of 5m0s: 900560 executions, 557899 successful
    rsg_test.go:822: 3m55s of 5m0s: 920093 executions, 570055 successful
    rsg_test.go:822: 4m0s of 5m0s: 938831 executions, 581564 successful
    rsg_test.go:822: 4m5s of 5m0s: 958498 executions, 593822 successful
    rsg_test.go:822: 4m10s of 5m0s: 978523 executions, 606207 successful
    rsg_test.go:822: 4m15s of 5m0s: 998186 executions, 618469 successful
    rsg_test.go:822: 4m20s of 5m0s: 1017864 executions, 630594 successful
    rsg_test.go:822: 4m25s of 5m0s: 1037599 executions, 642768 successful
    rsg_test.go:822: 4m30s of 5m0s: 1057123 executions, 654906 successful
    rsg_test.go:822: 4m35s of 5m0s: 1076170 executions, 666739 successful
    rsg_test.go:822: 4m40s of 5m0s: 1095738 executions, 678901 successful
    rsg_test.go:822: 4m45s of 5m0s: 1115448 executions, 691203 successful
    rsg_test.go:822: 4m50s of 5m0s: 1135186 executions, 703393 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7672301?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7672301?buildTab=artifacts#/) on master @ [39250f643837156c8fb75b0ced07ef8a99c14185](https://github.com/cockroachdb/cockroach/commits/39250f643837156c8fb75b0ced07ef8a99c14185):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5448192 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 45s of 5m0s: 163751 executions, 101305 successful
    rsg_test.go:822: 50s of 5m0s: 182218 executions, 112791 successful
    rsg_test.go:822: 55s of 5m0s: 200616 executions, 124220 successful
    rsg_test.go:822: 1m0s of 5m0s: 218066 executions, 135100 successful
    rsg_test.go:822: 1m5s of 5m0s: 236243 executions, 146271 successful
    rsg_test.go:822: 1m10s of 5m0s: 254658 executions, 157685 successful
    rsg_test.go:822: 1m15s of 5m0s: 272770 executions, 168805 successful
    rsg_test.go:822: 1m20s of 5m0s: 290904 executions, 180095 successful
    rsg_test.go:822: 1m25s of 5m0s: 309017 executions, 191345 successful
    rsg_test.go:822: 1m30s of 5m0s: 326840 executions, 202332 successful
    rsg_test.go:822: 1m35s of 5m0s: 345300 executions, 213779 successful
    rsg_test.go:822: 1m40s of 5m0s: 363513 executions, 225153 successful
    rsg_test.go:822: 1m45s of 5m0s: 381864 executions, 236549 successful
    rsg_test.go:822: 1m50s of 5m0s: 400453 executions, 248085 successful
    rsg_test.go:822: 1m55s of 5m0s: 418820 executions, 259383 successful
    rsg_test.go:822: 2m0s of 5m0s: 437482 executions, 270978 successful
    rsg_test.go:822: 2m5s of 5m0s: 455226 executions, 281920 successful
    rsg_test.go:822: 2m10s of 5m0s: 473877 executions, 293581 successful
    rsg_test.go:822: 2m15s of 5m0s: 492391 executions, 305033 successful
    rsg_test.go:822: 2m20s of 5m0s: 510178 executions, 316064 successful
    rsg_test.go:822: 2m25s of 5m0s: 528342 executions, 327256 successful
    rsg_test.go:822: 2m30s of 5m0s: 546748 executions, 338638 successful
    rsg_test.go:822: 2m35s of 5m0s: 565280 executions, 350025 successful
    rsg_test.go:822: 2m40s of 5m0s: 583025 executions, 361118 successful
    rsg_test.go:822: 2m45s of 5m0s: 601805 executions, 372784 successful
    rsg_test.go:822: 2m50s of 5m0s: 620200 executions, 384249 successful
    rsg_test.go:822: 2m55s of 5m0s: 638830 executions, 395807 successful
    rsg_test.go:822: 3m0s of 5m0s: 657921 executions, 407607 successful
    rsg_test.go:822: 3m5s of 5m0s: 676320 executions, 419003 successful
    rsg_test.go:822: 3m10s of 5m0s: 693797 executions, 429791 successful
    rsg_test.go:822: 3m15s of 5m0s: 712461 executions, 441318 successful
    rsg_test.go:822: 3m20s of 5m0s: 731174 executions, 452824 successful
    rsg_test.go:822: 3m25s of 5m0s: 749770 executions, 464337 successful
    rsg_test.go:822: 3m30s of 5m0s: 768437 executions, 475893 successful
    rsg_test.go:822: 3m35s of 5m0s: 787042 executions, 487411 successful
    rsg_test.go:822: 3m40s of 5m0s: 805584 executions, 498978 successful
    rsg_test.go:822: 3m45s of 5m0s: 823704 executions, 510204 successful
    rsg_test.go:822: 3m50s of 5m0s: 842564 executions, 521877 successful
    rsg_test.go:822: 3m55s of 5m0s: 861330 executions, 533467 successful
    rsg_test.go:822: 4m0s of 5m0s: 880150 executions, 545033 successful
    rsg_test.go:822: 4m5s of 5m0s: 899092 executions, 556784 successful
    rsg_test.go:822: 4m10s of 5m0s: 917017 executions, 567809 successful
    rsg_test.go:822: 4m15s of 5m0s: 935429 executions, 579232 successful
    rsg_test.go:822: 4m20s of 5m0s: 954388 executions, 590968 successful
    rsg_test.go:822: 4m25s of 5m0s: 973308 executions, 602691 successful
    rsg_test.go:822: 4m30s of 5m0s: 992192 executions, 614445 successful
    rsg_test.go:822: 4m35s of 5m0s: 1011159 executions, 626233 successful
    rsg_test.go:822: 4m40s of 5m0s: 1029494 executions, 637600 successful
    rsg_test.go:822: 4m45s of 5m0s: 1048131 executions, 649097 successful
    rsg_test.go:822: 4m50s of 5m0s: 1067250 executions, 660860 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7685369?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7685369?buildTab=artifacts#/) on master @ [b5be006bedd7d3cedc3fb3d2248df168e3d64be2](https://github.com/cockroachdb/cockroach/commits/b5be006bedd7d3cedc3fb3d2248df168e3d64be2):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6005273 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 45s of 5m0s: 181758 executions, 112458 successful
    rsg_test.go:822: 50s of 5m0s: 201926 executions, 124943 successful
    rsg_test.go:822: 55s of 5m0s: 222610 executions, 137754 successful
    rsg_test.go:822: 1m0s of 5m0s: 242946 executions, 150398 successful
    rsg_test.go:822: 1m5s of 5m0s: 261822 executions, 162028 successful
    rsg_test.go:822: 1m10s of 5m0s: 281855 executions, 174521 successful
    rsg_test.go:822: 1m15s of 5m0s: 302277 executions, 187142 successful
    rsg_test.go:822: 1m20s of 5m0s: 323031 executions, 199969 successful
    rsg_test.go:822: 1m25s of 5m0s: 343863 executions, 212928 successful
    rsg_test.go:822: 1m30s of 5m0s: 364087 executions, 225401 successful
    rsg_test.go:822: 1m35s of 5m0s: 384464 executions, 237940 successful
    rsg_test.go:822: 1m40s of 5m0s: 403842 executions, 249890 successful
    rsg_test.go:822: 1m45s of 5m0s: 424179 executions, 262502 successful
    rsg_test.go:822: 1m50s of 5m0s: 444681 executions, 275231 successful
    rsg_test.go:822: 1m55s of 5m0s: 465385 executions, 288050 successful
    rsg_test.go:822: 2m0s of 5m0s: 486299 executions, 301064 successful
    rsg_test.go:822: 2m5s of 5m0s: 506388 executions, 313430 successful
    rsg_test.go:822: 2m10s of 5m0s: 526955 executions, 326179 successful
    rsg_test.go:822: 2m15s of 5m0s: 547510 executions, 338864 successful
    rsg_test.go:822: 2m20s of 5m0s: 568283 executions, 351788 successful
    rsg_test.go:822: 2m25s of 5m0s: 588553 executions, 364288 successful
    rsg_test.go:822: 2m30s of 5m0s: 609149 executions, 377089 successful
    rsg_test.go:822: 2m35s of 5m0s: 630431 executions, 390186 successful
    rsg_test.go:822: 2m40s of 5m0s: 650079 executions, 402455 successful
    rsg_test.go:822: 2m45s of 5m0s: 670945 executions, 415402 successful
    rsg_test.go:822: 2m50s of 5m0s: 692128 executions, 428469 successful
    rsg_test.go:822: 2m55s of 5m0s: 713047 executions, 441466 successful
    rsg_test.go:822: 3m0s of 5m0s: 733527 executions, 454133 successful
    rsg_test.go:822: 3m5s of 5m0s: 753974 executions, 466834 successful
    rsg_test.go:822: 3m10s of 5m0s: 774898 executions, 479847 successful
    rsg_test.go:822: 3m15s of 5m0s: 795518 executions, 492644 successful
    rsg_test.go:822: 3m20s of 5m0s: 816349 executions, 505502 successful
    rsg_test.go:822: 3m25s of 5m0s: 837188 executions, 518291 successful
    rsg_test.go:822: 3m30s of 5m0s: 857639 executions, 530915 successful
    rsg_test.go:822: 3m35s of 5m0s: 877649 executions, 543456 successful
    rsg_test.go:822: 3m40s of 5m0s: 898804 executions, 556547 successful
    rsg_test.go:822: 3m45s of 5m0s: 919801 executions, 569581 successful
    rsg_test.go:822: 3m50s of 5m0s: 940303 executions, 582277 successful
    rsg_test.go:822: 3m55s of 5m0s: 961325 executions, 595342 successful
    rsg_test.go:822: 4m0s of 5m0s: 982168 executions, 608103 successful
    rsg_test.go:822: 4m5s of 5m0s: 1003549 executions, 621344 successful
    rsg_test.go:822: 4m10s of 5m0s: 1022886 executions, 633329 successful
    rsg_test.go:822: 4m15s of 5m0s: 1043720 executions, 646210 successful
    rsg_test.go:822: 4m20s of 5m0s: 1064533 executions, 659128 successful
    rsg_test.go:822: 4m25s of 5m0s: 1085080 executions, 671817 successful
    rsg_test.go:822: 4m30s of 5m0s: 1105844 executions, 684602 successful
    rsg_test.go:822: 4m35s of 5m0s: 1125900 executions, 697035 successful
    rsg_test.go:822: 4m40s of 5m0s: 1146733 executions, 709917 successful
    rsg_test.go:822: 4m45s of 5m0s: 1167789 executions, 723044 successful
    rsg_test.go:822: 4m50s of 5m0s: 1188417 executions, 735841 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7697782?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7697782?buildTab=artifacts#/) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4951974 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 10s of 5m0s: 36032 executions, 22352 successful
    rsg_test.go:822: 15s of 5m0s: 54850 executions, 33879 successful
    rsg_test.go:822: 20s of 5m0s: 73773 executions, 45571 successful
    rsg_test.go:822: 25s of 5m0s: 92722 executions, 57371 successful
    rsg_test.go:822: 30s of 5m0s: 110957 executions, 68646 successful
    rsg_test.go:822: 35s of 5m0s: 129824 executions, 80263 successful
    rsg_test.go:822: 40s of 5m0s: 148905 executions, 92026 successful
    rsg_test.go:822: 45s of 5m0s: 168112 executions, 103969 successful
    rsg_test.go:822: 50s of 5m0s: 186834 executions, 115495 successful
    rsg_test.go:822: 55s of 5m0s: 205226 executions, 126886 successful
    rsg_test.go:822: 1m0s of 5m0s: 224573 executions, 138714 successful
    rsg_test.go:822: 1m5s of 5m0s: 243541 executions, 150521 successful
    rsg_test.go:822: 1m10s of 5m0s: 262599 executions, 162297 successful
    rsg_test.go:822: 1m15s of 5m0s: 282206 executions, 174444 successful
    rsg_test.go:822: 1m20s of 5m0s: 301264 executions, 186335 successful
    rsg_test.go:822: 1m25s of 5m0s: 320499 executions, 198320 successful
    rsg_test.go:822: 1m30s of 5m0s: 338523 executions, 209392 successful
    rsg_test.go:822: 1m35s of 5m0s: 357665 executions, 221286 successful
    rsg_test.go:822: 1m40s of 5m0s: 376932 executions, 233250 successful
    rsg_test.go:822: 1m45s of 5m0s: 396364 executions, 245206 successful
    rsg_test.go:822: 1m50s of 5m0s: 415875 executions, 257379 successful
    rsg_test.go:822: 1m55s of 5m0s: 434296 executions, 268765 successful
    rsg_test.go:822: 2m0s of 5m0s: 453176 executions, 280401 successful
    rsg_test.go:822: 2m5s of 5m0s: 472494 executions, 292309 successful
    rsg_test.go:822: 2m10s of 5m0s: 491556 executions, 304079 successful
    rsg_test.go:822: 2m15s of 5m0s: 510748 executions, 316047 successful
    rsg_test.go:822: 2m20s of 5m0s: 529581 executions, 327639 successful
    rsg_test.go:822: 2m25s of 5m0s: 548764 executions, 339565 successful
    rsg_test.go:822: 2m30s of 5m0s: 567100 executions, 350903 successful
    rsg_test.go:822: 2m35s of 5m0s: 586176 executions, 362765 successful
    rsg_test.go:822: 2m40s of 5m0s: 605767 executions, 374911 successful
    rsg_test.go:822: 2m45s of 5m0s: 624979 executions, 386766 successful
    rsg_test.go:822: 2m50s of 5m0s: 644055 executions, 398565 successful
    rsg_test.go:822: 2m55s of 5m0s: 662817 executions, 410192 successful
    rsg_test.go:822: 3m0s of 5m0s: 682095 executions, 422173 successful
    rsg_test.go:822: 3m5s of 5m0s: 701198 executions, 433939 successful
    rsg_test.go:822: 3m10s of 5m0s: 720300 executions, 445840 successful
    rsg_test.go:822: 3m15s of 5m0s: 739030 executions, 457402 successful
    rsg_test.go:822: 3m20s of 5m0s: 757954 executions, 469153 successful
    rsg_test.go:822: 3m25s of 5m0s: 777067 executions, 481047 successful
    rsg_test.go:822: 3m30s of 5m0s: 795431 executions, 492487 successful
    rsg_test.go:822: 3m35s of 5m0s: 814388 executions, 504237 successful
    rsg_test.go:822: 3m40s of 5m0s: 833608 executions, 516006 successful
    rsg_test.go:822: 3m45s of 5m0s: 852766 executions, 527843 successful
    rsg_test.go:822: 3m50s of 5m0s: 871945 executions, 539826 successful
    rsg_test.go:822: 3m55s of 5m0s: 891257 executions, 551729 successful
    rsg_test.go:822: 4m0s of 5m0s: 910328 executions, 563495 successful
    rsg_test.go:822: 4m5s of 5m0s: 929007 executions, 575128 successful
    rsg_test.go:822: 4m10s of 5m0s: 948548 executions, 587283 successful
    rsg_test.go:822: 4m15s of 5m0s: 968100 executions, 599432 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7709097?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7709097?buildTab=artifacts#/) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 3554972 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 25s of 5m0s: 96808 executions, 59954 successful
    rsg_test.go:822: 30s of 5m0s: 116098 executions, 71934 successful
    rsg_test.go:822: 35s of 5m0s: 134609 executions, 83377 successful
    rsg_test.go:822: 40s of 5m0s: 153676 executions, 95066 successful
    rsg_test.go:822: 45s of 5m0s: 173579 executions, 107445 successful
    rsg_test.go:822: 50s of 5m0s: 192968 executions, 119336 successful
    rsg_test.go:822: 55s of 5m0s: 212949 executions, 131855 successful
    rsg_test.go:822: 1m0s of 5m0s: 232297 executions, 143805 successful
    rsg_test.go:822: 1m5s of 5m0s: 250730 executions, 155181 successful
    rsg_test.go:822: 1m10s of 5m0s: 270479 executions, 167518 successful
    rsg_test.go:822: 1m15s of 5m0s: 289893 executions, 179458 successful
    rsg_test.go:822: 1m20s of 5m0s: 309501 executions, 191687 successful
    rsg_test.go:822: 1m25s of 5m0s: 328794 executions, 203441 successful
    rsg_test.go:822: 1m30s of 5m0s: 348575 executions, 215815 successful
    rsg_test.go:822: 1m35s of 5m0s: 368281 executions, 228001 successful
    rsg_test.go:822: 1m40s of 5m0s: 387187 executions, 239688 successful
    rsg_test.go:822: 1m45s of 5m0s: 406541 executions, 251624 successful
    rsg_test.go:822: 1m50s of 5m0s: 426623 executions, 264098 successful
    rsg_test.go:822: 1m55s of 5m0s: 446178 executions, 276101 successful
    rsg_test.go:822: 2m0s of 5m0s: 466299 executions, 288594 successful
    rsg_test.go:822: 2m5s of 5m0s: 485236 executions, 300249 successful
    rsg_test.go:822: 2m10s of 5m0s: 504564 executions, 312246 successful
    rsg_test.go:822: 2m15s of 5m0s: 523883 executions, 324215 successful
    rsg_test.go:822: 2m20s of 5m0s: 543502 executions, 336386 successful
    rsg_test.go:822: 2m25s of 5m0s: 562986 executions, 348438 successful
    rsg_test.go:822: 2m30s of 5m0s: 582555 executions, 360473 successful
    rsg_test.go:822: 2m35s of 5m0s: 601284 executions, 372049 successful
    rsg_test.go:822: 2m40s of 5m0s: 621267 executions, 384448 successful
    rsg_test.go:822: 2m45s of 5m0s: 641097 executions, 396807 successful
    rsg_test.go:822: 2m50s of 5m0s: 660985 executions, 409079 successful
    rsg_test.go:822: 2m55s of 5m0s: 680886 executions, 421452 successful
    rsg_test.go:822: 3m0s of 5m0s: 700752 executions, 433812 successful
    rsg_test.go:822: 3m5s of 5m0s: 719886 executions, 445636 successful
    rsg_test.go:822: 3m10s of 5m0s: 739520 executions, 457863 successful
    rsg_test.go:822: 3m15s of 5m0s: 759286 executions, 470063 successful
    rsg_test.go:822: 3m20s of 5m0s: 778964 executions, 482290 successful
    rsg_test.go:822: 3m25s of 5m0s: 798762 executions, 494539 successful
    rsg_test.go:822: 3m30s of 5m0s: 818498 executions, 506741 successful
    rsg_test.go:822: 3m35s of 5m0s: 837487 executions, 518498 successful
    rsg_test.go:822: 3m40s of 5m0s: 856930 executions, 530601 successful
    rsg_test.go:822: 3m45s of 5m0s: 876394 executions, 542635 successful
    rsg_test.go:822: 3m50s of 5m0s: 895714 executions, 554634 successful
    rsg_test.go:822: 3m55s of 5m0s: 915903 executions, 567140 successful
    rsg_test.go:822: 4m0s of 5m0s: 935561 executions, 579303 successful
    rsg_test.go:822: 4m5s of 5m0s: 954286 executions, 590846 successful
    rsg_test.go:822: 4m10s of 5m0s: 973710 executions, 602906 successful
    rsg_test.go:822: 4m15s of 5m0s: 993180 executions, 614936 successful
    rsg_test.go:822: 4m20s of 5m0s: 1013226 executions, 627386 successful
    rsg_test.go:822: 4m25s of 5m0s: 1032682 executions, 639349 successful
    rsg_test.go:822: 4m30s of 5m0s: 1052284 executions, 651509 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7720732?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7720732?buildTab=artifacts#/) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4496677 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1054414811
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:822: 5s of 5m0s: 17202 executions, 10791 successful
    rsg_test.go:822: 10s of 5m0s: 34684 executions, 21537 successful
    rsg_test.go:822: 15s of 5m0s: 52597 executions, 32733 successful
    rsg_test.go:822: 20s of 5m0s: 70365 executions, 43660 successful
    rsg_test.go:822: 25s of 5m0s: 88244 executions, 54723 successful
    rsg_test.go:822: 30s of 5m0s: 105321 executions, 65227 successful
    rsg_test.go:822: 35s of 5m0s: 123521 executions, 76575 successful
    rsg_test.go:822: 40s of 5m0s: 141613 executions, 87728 successful
    rsg_test.go:822: 45s of 5m0s: 159762 executions, 98960 successful
    rsg_test.go:822: 50s of 5m0s: 177678 executions, 110067 successful
    rsg_test.go:822: 55s of 5m0s: 195508 executions, 121069 successful
    rsg_test.go:822: 1m0s of 5m0s: 212958 executions, 131904 successful
    rsg_test.go:822: 1m5s of 5m0s: 230639 executions, 142800 successful
    rsg_test.go:822: 1m10s of 5m0s: 249157 executions, 154279 successful
    rsg_test.go:822: 1m15s of 5m0s: 267323 executions, 165494 successful
    rsg_test.go:822: 1m20s of 5m0s: 285354 executions, 176763 successful
    rsg_test.go:822: 1m25s of 5m0s: 303312 executions, 187817 successful
    rsg_test.go:822: 1m30s of 5m0s: 321523 executions, 199120 successful
    rsg_test.go:822: 1m35s of 5m0s: 339656 executions, 210369 successful
    rsg_test.go:822: 1m40s of 5m0s: 357742 executions, 221595 successful
    rsg_test.go:822: 1m45s of 5m0s: 376068 executions, 232878 successful
    rsg_test.go:822: 1m50s of 5m0s: 394647 executions, 244359 successful
    rsg_test.go:822: 1m55s of 5m0s: 412254 executions, 255284 successful
    rsg_test.go:822: 2m0s of 5m0s: 430741 executions, 266683 successful
    rsg_test.go:822: 2m5s of 5m0s: 449145 executions, 278075 successful
    rsg_test.go:822: 2m10s of 5m0s: 467464 executions, 289365 successful
    rsg_test.go:822: 2m15s of 5m0s: 486205 executions, 301029 successful
    rsg_test.go:822: 2m20s of 5m0s: 504499 executions, 312295 successful
    rsg_test.go:822: 2m25s of 5m0s: 523035 executions, 323780 successful
    rsg_test.go:822: 2m30s of 5m0s: 540435 executions, 334566 successful
    rsg_test.go:822: 2m35s of 5m0s: 558943 executions, 346026 successful
    rsg_test.go:822: 2m40s of 5m0s: 577236 executions, 357282 successful
    rsg_test.go:822: 2m45s of 5m0s: 595813 executions, 368763 successful
    rsg_test.go:822: 2m50s of 5m0s: 614237 executions, 380123 successful
    rsg_test.go:822: 2m55s of 5m0s: 631782 executions, 391038 successful
    rsg_test.go:822: 3m0s of 5m0s: 649727 executions, 402075 successful
    rsg_test.go:822: 3m5s of 5m0s: 667683 executions, 413148 successful
    rsg_test.go:822: 3m10s of 5m0s: 685894 executions, 424507 successful
    rsg_test.go:822: 3m15s of 5m0s: 704479 executions, 436049 successful
    rsg_test.go:822: 3m20s of 5m0s: 722427 executions, 447052 successful
    rsg_test.go:822: 3m25s of 5m0s: 739784 executions, 457902 successful
    rsg_test.go:822: 3m30s of 5m0s: 757075 executions, 468508 successful
    rsg_test.go:822: 3m35s of 5m0s: 775282 executions, 479830 successful
    rsg_test.go:822: 3m40s of 5m0s: 793801 executions, 491342 successful
    rsg_test.go:822: 3m45s of 5m0s: 812321 executions, 502830 successful
    rsg_test.go:822: 3m50s of 5m0s: 830941 executions, 514397 successful
    rsg_test.go:822: 3m55s of 5m0s: 849580 executions, 525881 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7742771?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7742771?buildTab=artifacts#/) on master @ [2b595c5214aead3e7d4f47f9a1dedce57dba6e7f](https://github.com/cockroachdb/cockroach/commits/2b595c5214aead3e7d4f47f9a1dedce57dba6e7f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5022477 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1103475229
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:822: 5s of 5m0s: 17929 executions, 11047 successful
    rsg_test.go:822: 10s of 5m0s: 36257 executions, 22368 successful
    rsg_test.go:822: 15s of 5m0s: 55078 executions, 33977 successful
    rsg_test.go:822: 20s of 5m0s: 74067 executions, 45845 successful
    rsg_test.go:822: 25s of 5m0s: 93073 executions, 57559 successful
    rsg_test.go:822: 30s of 5m0s: 111396 executions, 69028 successful
    rsg_test.go:822: 35s of 5m0s: 130349 executions, 80726 successful
    rsg_test.go:822: 40s of 5m0s: 149180 executions, 92385 successful
    rsg_test.go:822: 45s of 5m0s: 168099 executions, 104119 successful
    rsg_test.go:822: 50s of 5m0s: 186818 executions, 115749 successful
    rsg_test.go:822: 55s of 5m0s: 206159 executions, 127640 successful
    rsg_test.go:822: 1m0s of 5m0s: 224946 executions, 139285 successful
    rsg_test.go:822: 1m5s of 5m0s: 242589 executions, 150266 successful
    rsg_test.go:822: 1m10s of 5m0s: 261301 executions, 161816 successful
    rsg_test.go:822: 1m15s of 5m0s: 280007 executions, 173399 successful
    rsg_test.go:822: 1m20s of 5m0s: 298998 executions, 185054 successful
    rsg_test.go:822: 1m25s of 5m0s: 318096 executions, 196912 successful
    rsg_test.go:822: 1m30s of 5m0s: 336996 executions, 208636 successful
    rsg_test.go:822: 1m35s of 5m0s: 355231 executions, 219922 successful
    rsg_test.go:822: 1m40s of 5m0s: 374081 executions, 231604 successful
    rsg_test.go:822: 1m45s of 5m0s: 393303 executions, 243640 successful
    rsg_test.go:822: 1m50s of 5m0s: 412646 executions, 255537 successful
    rsg_test.go:822: 1m55s of 5m0s: 431802 executions, 267372 successful
    rsg_test.go:822: 2m0s of 5m0s: 451379 executions, 279512 successful
    rsg_test.go:822: 2m5s of 5m0s: 470016 executions, 291043 successful
    rsg_test.go:822: 2m10s of 5m0s: 489192 executions, 302910 successful
    rsg_test.go:822: 2m15s of 5m0s: 508224 executions, 314666 successful
    rsg_test.go:822: 2m20s of 5m0s: 527866 executions, 326761 successful
    rsg_test.go:822: 2m25s of 5m0s: 547094 executions, 338744 successful
    rsg_test.go:822: 2m30s of 5m0s: 566253 executions, 350620 successful
    rsg_test.go:822: 2m35s of 5m0s: 585568 executions, 362424 successful
    rsg_test.go:822: 2m40s of 5m0s: 603864 executions, 373785 successful
    rsg_test.go:822: 2m45s of 5m0s: 622998 executions, 385559 successful
    rsg_test.go:822: 2m50s of 5m0s: 642396 executions, 397615 successful
    rsg_test.go:822: 2m55s of 5m0s: 661774 executions, 409674 successful
    rsg_test.go:822: 3m0s of 5m0s: 681444 executions, 421749 successful
    rsg_test.go:822: 3m5s of 5m0s: 700172 executions, 433288 successful
    rsg_test.go:822: 3m10s of 5m0s: 719430 executions, 445282 successful
    rsg_test.go:822: 3m15s of 5m0s: 738880 executions, 457333 successful
    rsg_test.go:822: 3m20s of 5m0s: 758245 executions, 469392 successful
    rsg_test.go:822: 3m25s of 5m0s: 777825 executions, 481470 successful
    rsg_test.go:822: 3m30s of 5m0s: 797384 executions, 493667 successful
    rsg_test.go:822: 3m35s of 5m0s: 815863 executions, 505019 successful
    rsg_test.go:822: 3m40s of 5m0s: 835165 executions, 516847 successful
    rsg_test.go:822: 3m45s of 5m0s: 854311 executions, 528757 successful
    rsg_test.go:822: 3m50s of 5m0s: 873497 executions, 540787 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7763940?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7763940?buildTab=artifacts#/) on master @ [7823e2f75474417a44f0d34f76c5f8914cb91d52](https://github.com/cockroachdb/cockroach/commits/7823e2f75474417a44f0d34f76c5f8914cb91d52):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 2643211 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1259065133
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:822: 5s of 5m0s: 16220 executions, 9993 successful
    rsg_test.go:822: 10s of 5m0s: 33100 executions, 20485 successful
    rsg_test.go:822: 15s of 5m0s: 49995 executions, 30882 successful
    rsg_test.go:822: 20s of 5m0s: 67568 executions, 41747 successful
    rsg_test.go:822: 25s of 5m0s: 85178 executions, 52575 successful
    rsg_test.go:822: 30s of 5m0s: 102127 executions, 63095 successful
    rsg_test.go:822: 35s of 5m0s: 119536 executions, 74007 successful
    rsg_test.go:822: 40s of 5m0s: 137315 executions, 85042 successful
    rsg_test.go:822: 45s of 5m0s: 154655 executions, 95829 successful
    rsg_test.go:822: 50s of 5m0s: 172466 executions, 106845 successful
    rsg_test.go:822: 55s of 5m0s: 190310 executions, 117968 successful
    rsg_test.go:822: 1m0s of 5m0s: 206944 executions, 128310 successful
    rsg_test.go:822: 1m5s of 5m0s: 224144 executions, 138940 successful
    rsg_test.go:822: 1m10s of 5m0s: 241813 executions, 149864 successful
    rsg_test.go:822: 1m15s of 5m0s: 259480 executions, 160892 successful
    rsg_test.go:822: 1m20s of 5m0s: 277245 executions, 171859 successful
    rsg_test.go:822: 1m25s of 5m0s: 294860 executions, 182791 successful
    rsg_test.go:822: 1m30s of 5m0s: 311986 executions, 193307 successful
    rsg_test.go:822: 1m35s of 5m0s: 329474 executions, 204107 successful
    rsg_test.go:822: 1m40s of 5m0s: 346982 executions, 214977 successful
    rsg_test.go:822: 1m45s of 5m0s: 364806 executions, 225935 successful
    rsg_test.go:822: 1m50s of 5m0s: 382597 executions, 236929 successful
    rsg_test.go:822: 1m55s of 5m0s: 399645 executions, 247476 successful
    rsg_test.go:822: 2m0s of 5m0s: 417359 executions, 258491 successful
    rsg_test.go:822: 2m5s of 5m0s: 435362 executions, 269607 successful
    rsg_test.go:822: 2m10s of 5m0s: 453210 executions, 280706 successful
    rsg_test.go:822: 2m15s of 5m0s: 471107 executions, 291721 successful
    rsg_test.go:822: 2m20s of 5m0s: 488879 executions, 302744 successful
    rsg_test.go:822: 2m25s of 5m0s: 505923 executions, 313311 successful
    rsg_test.go:822: 2m30s of 5m0s: 523356 executions, 324103 successful
    rsg_test.go:822: 2m35s of 5m0s: 541039 executions, 334976 successful
    rsg_test.go:822: 2m40s of 5m0s: 559081 executions, 346259 successful
    rsg_test.go:822: 2m45s of 5m0s: 576567 executions, 357012 successful
    rsg_test.go:822: 2m50s of 5m0s: 594535 executions, 368177 successful
    rsg_test.go:822: 2m55s of 5m0s: 611548 executions, 378634 successful
    rsg_test.go:822: 3m0s of 5m0s: 629455 executions, 389632 successful
    rsg_test.go:822: 3m5s of 5m0s: 647477 executions, 400737 successful
    rsg_test.go:822: 3m10s of 5m0s: 665341 executions, 411813 successful
    rsg_test.go:822: 3m15s of 5m0s: 683450 executions, 423008 successful
    rsg_test.go:822: 3m20s of 5m0s: 701400 executions, 434074 successful
    rsg_test.go:822: 3m25s of 5m0s: 718516 executions, 444619 successful
    rsg_test.go:822: 3m30s of 5m0s: 736408 executions, 455636 successful
    rsg_test.go:822: 3m35s of 5m0s: 754398 executions, 466881 successful
    rsg_test.go:822: 3m40s of 5m0s: 772469 executions, 478056 successful
    rsg_test.go:822: 3m45s of 5m0s: 790531 executions, 489198 successful
    rsg_test.go:822: 3m50s of 5m0s: 808199 executions, 500166 successful
    rsg_test.go:822: 3m55s of 5m0s: 825299 executions, 510842 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7782669?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7782669?buildTab=artifacts#/) on master @ [db71e045cc3f2937bf20170d254dd7e54a620ffb](https://github.com/cockroachdb/cockroach/commits/db71e045cc3f2937bf20170d254dd7e54a620ffb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7017780 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRandomSyntaxFunctions
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/d437d2c847dfedbc4972f231c3331c8e/logTestRandomSyntaxFunctions1454942563
    test_log_scope.go:79: use -show-logs to present logs inline
    rsg_test.go:822: 5s of 5m0s: 18183 executions, 11230 successful
    rsg_test.go:822: 10s of 5m0s: 36703 executions, 22664 successful
    rsg_test.go:822: 15s of 5m0s: 55838 executions, 34496 successful
    rsg_test.go:822: 20s of 5m0s: 75424 executions, 46579 successful
    rsg_test.go:822: 25s of 5m0s: 94984 executions, 58749 successful
    rsg_test.go:822: 30s of 5m0s: 114934 executions, 71182 successful
    rsg_test.go:822: 35s of 5m0s: 133709 executions, 82790 successful
    rsg_test.go:822: 40s of 5m0s: 153266 executions, 94895 successful
    rsg_test.go:822: 45s of 5m0s: 172850 executions, 106966 successful
    rsg_test.go:822: 50s of 5m0s: 192885 executions, 119328 successful
    rsg_test.go:822: 55s of 5m0s: 212839 executions, 131771 successful
    rsg_test.go:822: 1m0s of 5m0s: 232713 executions, 143988 successful
    rsg_test.go:822: 1m5s of 5m0s: 251567 executions, 155692 successful
    rsg_test.go:822: 1m10s of 5m0s: 271606 executions, 168055 successful
    rsg_test.go:822: 1m15s of 5m0s: 291176 executions, 180214 successful
    rsg_test.go:822: 1m20s of 5m0s: 310927 executions, 192378 successful
    rsg_test.go:822: 1m25s of 5m0s: 330992 executions, 204768 successful
    rsg_test.go:822: 1m30s of 5m0s: 350624 executions, 216986 successful
    rsg_test.go:822: 1m35s of 5m0s: 370054 executions, 228952 successful
    rsg_test.go:822: 1m40s of 5m0s: 390098 executions, 241334 successful
    rsg_test.go:822: 1m45s of 5m0s: 410110 executions, 253853 successful
    rsg_test.go:822: 1m50s of 5m0s: 430014 executions, 266109 successful
    rsg_test.go:822: 1m55s of 5m0s: 450072 executions, 278446 successful
    rsg_test.go:822: 2m0s of 5m0s: 469920 executions, 290805 successful
    rsg_test.go:822: 2m5s of 5m0s: 489065 executions, 302641 successful
    rsg_test.go:822: 2m10s of 5m0s: 508765 executions, 314897 successful
    rsg_test.go:822: 2m15s of 5m0s: 528555 executions, 327060 successful
    rsg_test.go:822: 2m20s of 5m0s: 548171 executions, 339222 successful
    rsg_test.go:822: 2m25s of 5m0s: 568268 executions, 351693 successful
    rsg_test.go:822: 2m30s of 5m0s: 587947 executions, 363796 successful
    rsg_test.go:822: 2m35s of 5m0s: 606756 executions, 375424 successful
    rsg_test.go:822: 2m40s of 5m0s: 626202 executions, 387526 successful
    rsg_test.go:822: 2m45s of 5m0s: 645749 executions, 399552 successful
    rsg_test.go:822: 2m50s of 5m0s: 665304 executions, 411657 successful
    rsg_test.go:822: 2m55s of 5m0s: 685060 executions, 423769 successful
    rsg_test.go:822: 3m0s of 5m0s: 704840 executions, 436004 successful
    rsg_test.go:822: 3m5s of 5m0s: 723491 executions, 447606 successful
    rsg_test.go:822: 3m10s of 5m0s: 743026 executions, 459719 successful
    rsg_test.go:822: 3m15s of 5m0s: 763095 executions, 472063 successful
    rsg_test.go:822: 3m20s of 5m0s: 783058 executions, 484400 successful
    rsg_test.go:822: 3m25s of 5m0s: 802866 executions, 496718 successful
    rsg_test.go:822: 3m30s of 5m0s: 822948 executions, 509122 successful
    rsg_test.go:822: 3m35s of 5m0s: 842163 executions, 521019 successful
    rsg_test.go:822: 3m40s of 5m0s: 861872 executions, 533108 successful
    rsg_test.go:822: 3m45s of 5m0s: 881809 executions, 545395 successful
    rsg_test.go:822: 3m50s of 5m0s: 901903 executions, 557904 successful
    rsg_test.go:822: 3m55s of 5m0s: 922001 executions, 570411 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7799013?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7799013?buildTab=artifacts#/) on master @ [e7b15ebaed9c14668ade0a7827a5525aedef1ab0](https://github.com/cockroachdb/cockroach/commits/e7b15ebaed9c14668ade0a7827a5525aedef1ab0):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7472721 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 45s of 5m0s: 162656 executions, 100820 successful
    rsg_test.go:822: 50s of 5m0s: 181248 executions, 112367 successful
    rsg_test.go:822: 55s of 5m0s: 199888 executions, 123877 successful
    rsg_test.go:822: 1m0s of 5m0s: 218498 executions, 135457 successful
    rsg_test.go:822: 1m5s of 5m0s: 235829 executions, 146179 successful
    rsg_test.go:822: 1m10s of 5m0s: 254440 executions, 157703 successful
    rsg_test.go:822: 1m15s of 5m0s: 272776 executions, 169060 successful
    rsg_test.go:822: 1m20s of 5m0s: 291102 executions, 180551 successful
    rsg_test.go:822: 1m25s of 5m0s: 309718 executions, 192132 successful
    rsg_test.go:822: 1m30s of 5m0s: 328177 executions, 203608 successful
    rsg_test.go:822: 1m35s of 5m0s: 346863 executions, 215139 successful
    rsg_test.go:822: 1m40s of 5m0s: 364614 executions, 226098 successful
    rsg_test.go:822: 1m45s of 5m0s: 383316 executions, 237636 successful
    rsg_test.go:822: 1m50s of 5m0s: 401661 executions, 249034 successful
    rsg_test.go:822: 1m55s of 5m0s: 420429 executions, 260692 successful
    rsg_test.go:822: 2m0s of 5m0s: 438846 executions, 272073 successful
    rsg_test.go:822: 2m5s of 5m0s: 457686 executions, 283707 successful
    rsg_test.go:822: 2m10s of 5m0s: 476364 executions, 295287 successful
    rsg_test.go:822: 2m15s of 5m0s: 493656 executions, 305991 successful
    rsg_test.go:822: 2m20s of 5m0s: 512123 executions, 317385 successful
    rsg_test.go:822: 2m25s of 5m0s: 531159 executions, 329216 successful
    rsg_test.go:822: 2m30s of 5m0s: 549825 executions, 340807 successful
    rsg_test.go:822: 2m35s of 5m0s: 567853 executions, 351861 successful
    rsg_test.go:822: 2m40s of 5m0s: 586789 executions, 363648 successful
    rsg_test.go:822: 2m45s of 5m0s: 604601 executions, 374605 successful
    rsg_test.go:822: 2m50s of 5m0s: 623259 executions, 386156 successful
    rsg_test.go:822: 2m55s of 5m0s: 642160 executions, 397933 successful
    rsg_test.go:822: 3m0s of 5m0s: 660816 executions, 409463 successful
    rsg_test.go:822: 3m5s of 5m0s: 679558 executions, 421081 successful
    rsg_test.go:822: 3m10s of 5m0s: 697875 executions, 432494 successful
    rsg_test.go:822: 3m15s of 5m0s: 716696 executions, 444141 successful
    rsg_test.go:822: 3m20s of 5m0s: 735453 executions, 455710 successful
    rsg_test.go:822: 3m25s of 5m0s: 754215 executions, 467398 successful
    rsg_test.go:822: 3m30s of 5m0s: 772625 executions, 478772 successful
    rsg_test.go:822: 3m35s of 5m0s: 791529 executions, 490520 successful
    rsg_test.go:822: 3m40s of 5m0s: 810315 executions, 502171 successful
    rsg_test.go:822: 3m45s of 5m0s: 828590 executions, 513371 successful
    rsg_test.go:822: 3m50s of 5m0s: 847214 executions, 524858 successful
    rsg_test.go:822: 3m55s of 5m0s: 865726 executions, 536258 successful
    rsg_test.go:822: 4m0s of 5m0s: 884688 executions, 548063 successful
    rsg_test.go:822: 4m5s of 5m0s: 903318 executions, 559630 successful
    rsg_test.go:822: 4m10s of 5m0s: 921407 executions, 570682 successful
    rsg_test.go:822: 4m15s of 5m0s: 940199 executions, 582336 successful
    rsg_test.go:822: 4m20s of 5m0s: 959484 executions, 594356 successful
    rsg_test.go:822: 4m25s of 5m0s: 978295 executions, 605989 successful
    rsg_test.go:822: 4m30s of 5m0s: 997135 executions, 617535 successful
    rsg_test.go:822: 4m35s of 5m0s: 1015957 executions, 629173 successful
    rsg_test.go:822: 4m40s of 5m0s: 1034036 executions, 640407 successful
    rsg_test.go:822: 4m45s of 5m0s: 1052635 executions, 651940 successful
    rsg_test.go:822: 4m50s of 5m0s: 1071355 executions, 663510 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7815203?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7815203?buildTab=artifacts#/) on master @ [9c21578450e395c83a1dc0df7090296fef06e006](https://github.com/cockroachdb/cockroach/commits/9c21578450e395c83a1dc0df7090296fef06e006):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7780693 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSVector for AsJSON
        SELECT st_asgeojson(('''PHXQq'' ''QYhDsRlyRr'':83C ''e'' ''iFVndoynW'':941A ''lWfki'':875C ''ooo'' ''tNaFpZhElF''':::TSVECTOR,)) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSQuery for AsJSON
        SELECT st_asgeojson(('''I''':::TSQUERY,)) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1129980 executions, 696834 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_json(): unexpected type *tree.DTSQuery for AsJSON
        SELECT to_json('''mYxwTmuvCv''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_jsonb(): unexpected type *tree.DTSQuery for AsJSON
        SELECT to_jsonb('''Jg''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: crdb_internal.to_json_as_changefeed_with_flags(): unexpected type *tree.DTSQuery for AsJSON
        SELECT crdb_internal.to_json_as_changefeed_with_flags(('''r''':::TSQUERY,), e'\x1b\\wY':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: row_to_json(): unexpected type *tree.DTSQuery for AsJSON
        SELECT row_to_json(('''F''':::TSQUERY,)) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT jsonb_build_array('''fqShICyl''':::TSQUERY, '\xed':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_jsonb(): unexpected type *tree.DTSVector for AsJSON
        SELECT to_jsonb('''LUK'':172B ''XgaRG'':342A,475A,872B,979A ''cRbhVU'' ''eIhRL'' ''iuYBFn'':247B,311B,474B,517C,841A ''nE'' ''pdBdu'' ''qTtYItBQJ'' ''r'' ''ro'':55C,295C,348C,366B':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_jsonb(): unexpected type *tree.DTSVector for AsJSON
        SELECT to_jsonb('''HBABQ'':140A,249B,578A':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1149924 executions, 709053 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSQuery for AsJSON
        SELECT st_asgeojson(('''IGo''':::TSQUERY,)) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: crdb_internal.to_json_as_changefeed_with_flags(): unexpected type *tree.DTSQuery for AsJSON
        SELECT crdb_internal.to_json_as_changefeed_with_flags(('''vKuOO''':::TSQUERY,)) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSQuery for AsJSON
        SELECT st_asgeojson(('''psuneJOee''':::TSQUERY,)) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7824987?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7824987?buildTab=artifacts#/) on master @ [859e1e1f26d79af43cce16519f9ba984baaa495c](https://github.com/cockroachdb/cockroach/commits/859e1e1f26d79af43cce16519f9ba984baaa495c):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7456639 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT jsonb_build_array((-5047):::INT8, 'BOX(0.5017968290657815 0.47738653743616755,1.491732734183289 0.7028677133198811)':::BOX2D, '''YjXC''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSVector for AsJSON
        SELECT json_build_array(B'01110100', '''NVXmlbFn'' ''cgVyOka'' ''cxcjPGkUiU''':::TSVECTOR, NULL::anyelement, 'L':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 908007 executions, 560411 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: unexpected type *tree.DTSVector for AsJSON
        SELECT jsonb_object_agg(e'\x1d04\fy':::STRING, '''KwbsSj'':168A,873C ''TzRlqa'':260B':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_json(): unexpected type *tree.DTSVector for AsJSON
        SELECT to_json('''AeeNSLlHh'' ''EsgKIp'':304B,690A ''FXYmgb'':171B,379C,814B ''LUXSncPD'' ''dKjpxQC'':246A,658A ''gvpYple'' ''li'':130B,510B,562A,728B ''p'':442B,480A ''xaIdc''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_jsonb(): unexpected type *tree.DTSVector for AsJSON
        SELECT to_jsonb((e'B\x12rSs':::STRING:::NAME, 'BOX(-0.40210872043487245 -0.2510739288629003,-0.36587427210528034 -0.03725215123696313)':::BOX2D, '''BQPMX'' ''DVvGPZWJ'' ''gyk'':54B,303B,829C ''hpI'' ''knWDxwsK'':700B ''vTy''':::TSVECTOR, (-1.9681888196859956):::FLOAT8)) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: unexpected type *tree.DTSVector for AsJSON
        SELECT json_agg('''TmbO'' ''jCwYCbHS'' ''msawZIQI''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: crdb_internal.to_json_as_changefeed_with_flags(): unexpected type *tree.DTSQuery for AsJSON
        SELECT crdb_internal.to_json_as_changefeed_with_flags(('''jYyuD''':::TSQUERY,), e'\x15cjBk':::STRING, 'X':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT json_build_array(ARRAY['00:17:01.403214':::TIME,'01:42:30.677396':::TIME,'10:28:45.462673':::TIME], '''sbsFiLR''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSVector for AsJSON
        SELECT st_asgeojson(('''DasLPV'':191A,744A,758B ''EqZGuMA'':343C,491A,618C ''PpzvvXjvK'':863B ''eZyDrxGO'':371A,591A,612C,964C ''ri'':306A,568B ''uyqr'':512A,515C,683B,873A ''v''':::TSVECTOR,), '':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT json_build_array(0.3038603365421295:::FLOAT8, '-62 years -10 mons -976 days -10:19:18.03719':::INTERVAL, NULL, '''Efkecgu''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSVector for AsJSON
        SELECT json_build_array('BOX(-1.2816589248245895 0.18426277863398843,0.0995032074484481 0.38521307601720867)':::BOX2D, (), '''CgGEvafE'' ''ERamZ'':59B,857B,945C ''XsUtoZBfEX'' ''hlFJoI'':231B,594A ''lPsVkckog'' ''nt'':999B':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 928093 executions, 572900 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT json_build_array(e'|i\x03-':::STRING, '''ISC''':::TSQUERY, (-1):::INT8) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7834216?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7834216?buildTab=artifacts#/) on master @ [8a5cb5109d6b7e122e996a44f940bfd61698cd81](https://github.com/cockroachdb/cockroach/commits/8a5cb5109d6b7e122e996a44f940bfd61698cd81):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7529448 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_jsonb(): unexpected type *tree.DTSVector for AsJSON
        SELECT to_jsonb('''CCHJxQEw'':356C,851C ''WiT'' ''YlOQaWDpAm'' ''Ytw'' ''bSr'' ''ehIbvM'' ''fyN'':86B,664A':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT json_build_array('''ENCTpxzDap''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSQuery for AsJSON
        SELECT st_asgeojson(('''ODpe''':::TSQUERY,), ']':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for AsJSON
        SELECT json_build_object(1.4919199868409119:::FLOAT8, '''Sfee''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSVector for AsJSON
        SELECT st_asgeojson(('''QqPrJBaqws''':::TSVECTOR,), '!O.;6':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSQuery for AsJSON
        SELECT st_asgeojson(('''QonQBmGIt''':::TSQUERY,), 'pLcy':::STRING, (-1058022966957733470):::INT8) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: unexpected type *tree.DTSQuery for AsJSON
        SELECT jsonb_object_agg(e'\x14 #,\x02lM':::STRING, '''cLjAwuO''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSQuery for AsJSON
        SELECT json_build_array('''NHCMAqP''':::TSQUERY, 'BOX(-0.2600131474860394 0.03316100431349778,-0.0008201646271180918 0.7112424780588276)':::BOX2D, '1987-01-05':::DATE, '\xed6a8d':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: st_asgeojson(): unexpected type *tree.DTSVector for AsJSON
        SELECT st_asgeojson(('''IRGdITZb'':344B,635A ''cLvuzAN'' ''dj'':51A ''tmIE''':::TSVECTOR,), e'\x128':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1057282 executions, 651352 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: to_json(): unexpected type *tree.DTSVector for AsJSON
        SELECT to_json('''LA'':137C,461C,545A,780B,908B ''MlICuqDUqK'' ''bEi'':64A,81B,759A ''lYQEWz'' ''oKFxGk'':378B,747A':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_array(): unexpected type *tree.DTSVector for AsJSON
        SELECT json_build_array('''HSY'':42A,926A ''LicCqf'':606A ''UOgWfUOw'' ''nBZ'':165A,167A,562A':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: row_to_json(): unexpected type *tree.DTSQuery for AsJSON
        SELECT row_to_json(('''iWZra''':::TSQUERY,)) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7852334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7852334?buildTab=artifacts#/) on master @ [146556e19f5e4fdc8c3e6a623b280cc33aee4d18](https://github.com/cockroachdb/cockroach/commits/146556e19f5e4fdc8c3e6a623b280cc33aee4d18):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 4408963 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 2m40s of 5m0s: 593683 executions, 366127 successful
    rsg_test.go:822: 2m45s of 5m0s: 612371 executions, 377639 successful
    rsg_test.go:822: 2m50s of 5m0s: 630429 executions, 388854 successful
    rsg_test.go:822: 2m55s of 5m0s: 649144 executions, 400392 successful
    rsg_test.go:822: 3m0s of 5m0s: 667717 executions, 411795 successful
    rsg_test.go:822: 3m5s of 5m0s: 686630 executions, 423540 successful
    rsg_test.go:822: 3m10s of 5m0s: 705053 executions, 434863 successful
    rsg_test.go:822: 3m15s of 5m0s: 724052 executions, 446675 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''AMhHqBp'':241B,534B,839A ''EuTGwkT'' ''GuJNaTLEck'':18C,191C,878B,950A ''OddiKH'' ''pKVpZ'' ''tennFTPfL''':::TSVECTOR, '-54 years -4 mons -678 days -08:52:50.115988':::INTERVAL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('1991-02-12 11:56:04.000833+00:00':::TIMESTAMPTZ, '00000000-0000-0000-0000-000000000000':::UUID, '''ErjAeo'':96A,134A,534B,938A ''P'':136B,314C,484C ''TT'' ''Un'' ''ZFrIMPyF'':424B,937B,956B ''cfje'' ''jTdNEDtwyN'':72A,124B,474C ''zC''':::TSVECTOR, ARRAY[(-1):::INT8,(-29467):::INT8]) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''VzRlacLSS'':22A,121B,590B,809B ''syobvom''':::TSVECTOR, '0102000020E610000002000000000000000000F03F000000000000F03F00000000000000400000000000000040':::GEOGRAPHY) ;
        
        Stack trace:
    rsg_test.go:822: 3m20s of 5m0s: 742889 executions, 458237 successful
    rsg_test.go:822: 3m25s of 5m0s: 760730 executions, 469176 successful
    rsg_test.go:822: 3m30s of 5m0s: 779459 executions, 480739 successful
    rsg_test.go:822: 3m35s of 5m0s: 798112 executions, 492173 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object(12098:::INT8, 8.80143213962092615E+28:::DECIMAL, '''ATxW'' ''JcioB'' ''OJcPDtBvn'' ''eBAeOsLIq'':726A,975B ''j'':357C ''mhTzdxnb'':131B,179C,705A ''mmuXw'' ''y''':::TSVECTOR, (-0.007571320501982198):::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 816868 executions, 503730 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''OaaHCdIt''':::TSQUERY, ('1994-09-27 09:56:12.000871':::TIMESTAMP, false)) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('2014-12-17 11:09:41.000943':::TIMESTAMP, '48.210.136.103':::INET, '''U''':::TSQUERY, 2310202323:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m45s of 5m0s: 835639 executions, 515303 successful
    rsg_test.go:822: 3m50s of 5m0s: 853822 executions, 526463 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''ATUE'':319C ''LpnfGF'' ''QSUe'' ''caXPGkPO'' ''sWdyteYu''':::TSVECTOR, NULL, '07:47:49.757587+02:35:00':::TIMETZ, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 872505 executions, 537972 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''HNJjvtNBR''':::TSQUERY, 0.5285051897159379:::FLOAT8, '2033-02-26 23:17:03.000297+00:00':::TIMESTAMPTZ, ARRAY[1013437132:::OID,2708116468:::OID]) ;
        
        Stack trace:
    rsg_test.go:822: 4m0s of 5m0s: 891175 executions, 549464 successful
    rsg_test.go:822: 4m5s of 5m0s: 909901 executions, 561040 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7867058?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7867058?buildTab=artifacts#/) on master @ [8165e3974c10e88b6ae11c6255872ea16f3a67e3](https://github.com/cockroachdb/cockroach/commits/8165e3974c10e88b6ae11c6255872ea16f3a67e3):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7456117 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 3m5s of 5m0s: 661686 executions, 408587 successful
    rsg_test.go:822: 3m10s of 5m0s: 680123 executions, 419997 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('BOX(-1.2803643570671004 -1.458919952366451,-1.1591169696954045 0.906920579604277)':::BOX2D, 'BOX(-1.1951052428976376 0.03594932075860663,0.2833664661410177 1.8461415799445493)':::BOX2D, '''EZWFWPNCR'' ''jN''':::TSVECTOR, '03:32:12.241528':::TIME) ;
        
        Stack trace:
    rsg_test.go:822: 3m15s of 5m0s: 698604 executions, 431331 successful
    rsg_test.go:822: 3m20s of 5m0s: 717014 executions, 442773 successful
    rsg_test.go:822: 3m25s of 5m0s: 735709 executions, 454282 successful
    rsg_test.go:822: 3m30s of 5m0s: 753701 executions, 465313 successful
    rsg_test.go:822: 3m35s of 5m0s: 770876 executions, 475959 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''I''':::TSQUERY, 0:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 788657 executions, 486832 successful
    rsg_test.go:822: 3m45s of 5m0s: 806627 executions, 497992 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''EltIYc'':363C,625C,671A ''FCE'' ''QNzuqVM'' ''arWxY'' ''dKltQ'':73A,609C ''rbCnCoRsAx'':591B,682C,718A':::TSVECTOR, 496634166:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 824918 executions, 509318 successful
    rsg_test.go:822: 3m55s of 5m0s: 842983 executions, 520433 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object(e'\'3Ijsuw\'':::STRING, ':':::STRING, '''QADqYOo''':::TSQUERY, '''FKYkQ'':746A ''LhvFcnVTB'' ''LopVaV'':590A,677B,934C ''WOdvW'' ''Xyrb'':252B,370A,668A ''b'' ''sCT''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 4m0s of 5m0s: 860121 executions, 531082 successful
    rsg_test.go:822: 4m5s of 5m0s: 878431 executions, 542378 successful
    rsg_test.go:822: 4m10s of 5m0s: 896857 executions, 553652 successful
    rsg_test.go:822: 4m15s of 5m0s: 915060 executions, 564944 successful
    rsg_test.go:822: 4m20s of 5m0s: 933397 executions, 576294 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''C'' ''EMWpyMrHl'' ''FTorJnAQA'' ''IIqb'':347A,489B,859A,946C ''TlxxAzC'' ''XNTPOrG'' ''Z'':9A,81C,317C,446A ''liO'' ''rZUOpOarQ'':112A,214B,763B ''tlprG'':125C,345C,586C,685A,914A':::TSVECTOR, e'\x7f\x02K\\DPmO-':::STRING:::NAME) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 952089 executions, 587757 successful
    rsg_test.go:822: 4m30s of 5m0s: 969694 executions, 598608 successful
    rsg_test.go:822: 4m35s of 5m0s: 988012 executions, 609965 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''IDPEWml'' ''UTfobn'':509C ''hEJ'' ''hHEkesbfk'' ''qXL'':552C,667A':::TSVECTOR, B'1111011001101000010101100100000001110000100') ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1006476 executions, 621358 successful
    rsg_test.go:822: 4m45s of 5m0s: 1024673 executions, 632474 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''WiQPwuwCDC''':::TSQUERY, e'\'':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1042919 executions, 643706 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7879910?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7879910?buildTab=artifacts#/) on master @ [ec095bc2fdbe4e518b076db20e4920fab67222bf](https://github.com/cockroachdb/cockroach/commits/ec095bc2fdbe4e518b076db20e4920fab67222bf):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7385077 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 2m45s of 5m0s: 599911 executions, 370325 successful
    rsg_test.go:822: 2m50s of 5m0s: 618402 executions, 381724 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''M'':888C ''TdxTZqjBkc'':99C,266B,418C ''UEUkLH'':108A,282B ''US'' ''d'':40C,381A,800C,941B ''hjhOPmIgxH'':315A,447C ''idpqON'':310B,734B ''jrlMGK'':765B ''wPbtKxCI'':363B,378A ''xH'':274C,455C,917C':::TSVECTOR, (SELECT NULL)::anyelement, e'\x1fOLj\x1c\x11Jd5':::STRING, 4141253182:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 2m55s of 5m0s: 636016 executions, 392643 successful
    rsg_test.go:822: 3m0s of 5m0s: 654682 executions, 404077 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''AklCeg''':::TSQUERY, 3694427711:::OID, NULL, '''C''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:822: 3m5s of 5m0s: 672779 executions, 415254 successful
    rsg_test.go:822: 3m10s of 5m0s: 690658 executions, 426318 successful
    rsg_test.go:822: 3m15s of 5m0s: 709228 executions, 437787 successful
    rsg_test.go:822: 3m20s of 5m0s: 727276 executions, 448957 successful
    rsg_test.go:822: 3m25s of 5m0s: 745761 executions, 460349 successful
    rsg_test.go:822: 3m30s of 5m0s: 764284 executions, 471717 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''KBNvGhuSGe'':396A,439A ''WmmQbLH'' ''dIECDAiSfU'':15A,691C ''gQ'':322B,792B ''r'':531B,619A,707A':::TSVECTOR, e'+\x0bO':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 3m35s of 5m0s: 782975 executions, 483181 successful
    rsg_test.go:822: 3m40s of 5m0s: 801677 executions, 494731 successful
    rsg_test.go:822: 3m45s of 5m0s: 820477 executions, 506372 successful
    rsg_test.go:822: 3m50s of 5m0s: 839138 executions, 517871 successful
    rsg_test.go:822: 3m55s of 5m0s: 856949 executions, 528974 successful
    rsg_test.go:822: 4m0s of 5m0s: 875609 executions, 540476 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('2027-08-07 21:57:58.00097+00:00':::TIMESTAMPTZ, NULL::anyelement, '''CbxqcSsLC''':::TSQUERY, '2015-03-11 08:21:26.00053':::TIMESTAMP) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''bAZear''':::TSQUERY, '30cc:9260:d96f:c6bf:eaa2:ac46:b1e6:1c38/52':::INET) ;
        
        Stack trace:
    rsg_test.go:822: 4m5s of 5m0s: 894429 executions, 552074 successful
    rsg_test.go:822: 4m10s of 5m0s: 913081 executions, 563595 successful
    rsg_test.go:822: 4m15s of 5m0s: 931655 executions, 575088 successful
    rsg_test.go:822: 4m20s of 5m0s: 950161 executions, 586542 successful
    rsg_test.go:822: 4m25s of 5m0s: 968953 executions, 598152 successful
    rsg_test.go:822: 4m30s of 5m0s: 986400 executions, 608841 successful
    rsg_test.go:822: 4m35s of 5m0s: 1005162 executions, 620448 successful
    rsg_test.go:822: 4m40s of 5m0s: 1023936 executions, 632075 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('':::VOID, '0105000060E61000000900000001020000400500000098CC6E9CD64242C0C9F4C4D8D46C52C08000F1FE0F539541043809C92BC35240682AAFA3F4973340387175BFB339D54124EF7CFA999C4740981D817B16C82B40108CFB878811DB4178E584EFCB6355404ECB19E385B5544068CF290850AAD641504A7907508033C098C3A23CF5383C40BAA2957DD3F2F4410102000040030000009C589C53E67E634088A2A3D36C8136C08003456AD67CA9C14BFB8A45AD1B54C0063ABBFE088640407890912A1CD9E9C11EC607EF440D65C0DCFADFEB621B31C008DAC30EF7F3D9410102000040020000001276ABDBECBA56C06059FA7253F440C0F464196CBDE5FB4196C7C1F0747A6340DC793D54827446C0CE2BDFDDA76CFAC10102000040040000008013CB58623D44C06C7EB19EFC1B44C01A14EE16709DFAC1D4650A97021043C008CFD10C4A1D54C0C43CAC8103D7E2C1C29BC0516D79604078CA87FAA23030C0EAE919560B6EF541D0C0003DE8056340147CBBB6D1B4324096273E4D84EAECC10102000040040000006C3026C8CCB160C0403CCD7C279D22C070162DF32EADDAC10E520814ECD960C0A6188F37BFE842C0F4444DC736CFF8413D39E23433D856C06411C84627E446C0703DEBDF524EC4413825B108E52F54C0D487FE5DD0854E40F47DCEBB28D9F9C1010200004005000000F8D8B97AD0473F40E0752CCBDBE320409AC7394F0434FA41957AE457F1FD62C05C11547034784740A04BB8BD5208BA41463ADC7691D94EC018F2A13BEA7729C036C70F1D8BD7FBC1FBF955BB1F3D64C0009B3CE8DD90E23F00809445980B13C11CA1F37217D858C0A0177951E5E12BC030A018C0C98FD1410102000040040000004213F89DEABC62C0A29EDB4E6CF145C0E67CCE5A6851F1414F61AC160EF950C0A4477182ED7546C084D5A158D32EFDC10C9A20BD903D634068DCD6B042E3394028EF68156159F44180D959993D5854C0924E30F4C08755404C002B6A1251DAC1010200004003000000E8533E8E49773040C08CFBD9B4263A40E8DA8EC5862CF2414077FE944F7031C098DC8C249D212DC0E36CDBD44AB3F1C1382C3D1155A062C010B5DB8F61D824C00008F57673D8CFC10102000040020000002875DB546F0342406273BB7B3B774DC0E03E28C94538B9419E7668C82D156040289E393ED4EA2E404AEAAEB13CEDF8C1':::GEOGRAPHY, '''B'':71C,107A,636B,706B ''NuRMOzpBB'':547A,629A,676A ''SXCbZpqLJ'' ''ikWOxYa''':::TSVECTOR, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1042212 executions, 643436 successful
    rsg_test.go:822: 4m50s of 5m0s: 1060343 executions, 654550 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7895032?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7895032?buildTab=artifacts#/) on master @ [24854994805cede37e6845ee2a94e10272b5506b](https://github.com/cockroachdb/cockroach/commits/24854994805cede37e6845ee2a94e10272b5506b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7508842 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 3m45s of 5m0s: 842138 executions, 519972 successful
    rsg_test.go:822: 3m50s of 5m0s: 861369 executions, 531793 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('BOX(-10 -10,10 10)':::BOX2D, (SELECT NULL)::anyelement, '''AFCEkc'' ''Ef'' ''I'':28A,93B ''MwtTGlfaMD'' ''qWPDYjl'':976C ''uKkc'':745C,766C':::TSVECTOR, B'00110000001001000000011000110000') ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 880000 executions, 543256 successful
    rsg_test.go:822: 4m0s of 5m0s: 898663 executions, 554779 successful
    rsg_test.go:822: 4m5s of 5m0s: 917507 executions, 566390 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''GgrGvLlPgI'' ''H'' ''X'':19C,478A,481C ''XEjINa'' ''Yb'':512B,817C ''ivIZ'':455C,821A ''lKt'':468B,969A ''otK'':377A,579C,592B,796A':::TSVECTOR, e'\x07)':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''Af'':172B,298C ''BRTBlWmArM'':770B ''HevqCbE'' ''Qo'':429B ''TkyQXVmhw'':59A,194C ''ercd'':848C,972C ''vTQbfwhpUA'':219C,363C,572B ''wZbEoOdBd'':130B ''yqgCRFwq''':::TSVECTOR, '06:58:46.44106':::TIME, (-2.103381596491894):::FLOAT8, 'J':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 935624 executions, 577656 successful
    rsg_test.go:822: 4m15s of 5m0s: 954793 executions, 589430 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''h'' ''hUYSJeSyn'' ''j'' ''kdJzWdE''':::TSVECTOR, '''GsqaADw'':82B,501C ''QpW'' ''ShyYEKjm'':133B,541B,709A ''TU'':718C ''TUf'' ''Tt'' ''dkXnz'':166B ''lzzzBCYbDQ'':108C,885B ''px'':839C,867B ''xUnMxedw''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 973914 executions, 601237 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''Ff'' ''PHDcVXfT'' ''UEOsQU'' ''WdIykBO'' ''sT'' ''uXL'' ''vOs'':100C,336C,653B':::TSVECTOR, 3388066740:::OID, e'X1\x0bF\x1b7D':::STRING, '{"4LL0tjWg": "baz", "nhBPSUO": null, "uxqXSWX": [false]}':::JSONB) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 992880 executions, 612858 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object(B'1011011000011011111101100000010010010001000', e'\t.*%EAY\x14':::STRING, '''Sl''':::TSQUERY, ARRAY[]:::BOOL[]) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 1011811 executions, 624575 successful
    rsg_test.go:822: 4m35s of 5m0s: 1029979 executions, 635768 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''lBaiXS''':::TSQUERY, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1048947 executions, 647485 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''AsbF''':::TSQUERY, e'\x16':::STRING, 3716941797:::OID, 0.15809073186388:::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1067868 executions, 659190 successful
    rsg_test.go:822: 4m50s of 5m0s: 1086687 executions, 670790 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''LoY''':::TSQUERY, 0:::OID, NULL, e'\x11\x03?t\\\x1aT\x1bV':::STRING) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7909428?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7909428?buildTab=artifacts#/) on master @ [f2b00e8039af6ea8887ec124dad8daf19da6fbf1](https://github.com/cockroachdb/cockroach/commits/f2b00e8039af6ea8887ec124dad8daf19da6fbf1):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7625247 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 3m25s of 5m0s: 766106 executions, 472602 successful
    rsg_test.go:822: 3m30s of 5m0s: 784616 executions, 484009 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''zsCcY''':::TSQUERY, 0.5013381664034218:::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:822: 3m35s of 5m0s: 803748 executions, 495895 successful
    rsg_test.go:822: 3m40s of 5m0s: 823011 executions, 507805 successful
    rsg_test.go:822: 3m45s of 5m0s: 842246 executions, 519629 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''K''':::TSQUERY, '12:04:41.080764':::TIME) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 861533 executions, 531456 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''CYWPcjL'' ''LoDt'':439B,460B,652C,677A,776B ''hvZnAa'' ''rEHD'' ''vy'' ''xrQ'' ''yhF'':689B,949B,951B':::TSVECTOR, (SELECT NULL)::anyelement, NULL, ARRAY['1980-09-07':::DATE,'1981-01-10':::DATE,'1983-10-16':::DATE]) ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 880537 executions, 543124 successful
    rsg_test.go:822: 4m0s of 5m0s: 899429 executions, 554758 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object(2754930165:::OID, (-1.0):::FLOAT8, '''BoujaySMX'' ''En'':207C,763B,839B ''FYCr'':21C,618B ''MulHsX'' ''Ndg'':378B,886B ''ciFJp'' ''dhLTm'' ''oPfTUKzM'' ''uCUfUcuyc'' ''wjeuKJVh'':181B,451A,738C':::TSVECTOR, '1970-08-09':::DATE) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object((-0.8838133215904236):::FLOAT8, false, '''FQRtvvet'' ''H'' ''LNbd'':534C,789B ''MmBNxqx'':175C,921A ''dpywH'' ''lxrwWv'':58C,263A ''zCYt''':::TSVECTOR, '':::VOID) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('cf93e9c3-34db-4f0a-a5c6-b6c11b755309':::UUID, NULL, '''uItncfJF''':::TSQUERY, 'ad1d4863-25d4-4d7e-beb1-d944f3d08e21':::UUID) ;
        
        Stack trace:
    rsg_test.go:822: 4m5s of 5m0s: 917694 executions, 566052 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object(']Bo>Z$g':::STRING, 'R':::STRING, '''l''':::TSQUERY, ARRAY[(-370707066558523.9367):::DECIMAL,76.94715122094832893:::DECIMAL]) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 936669 executions, 577745 successful
    rsg_test.go:822: 4m15s of 5m0s: 955853 executions, 589697 successful
    rsg_test.go:822: 4m20s of 5m0s: 975351 executions, 601669 successful
    rsg_test.go:822: 4m25s of 5m0s: 994400 executions, 613427 successful
    rsg_test.go:822: 4m30s of 5m0s: 1013323 executions, 625093 successful
    rsg_test.go:822: 4m35s of 5m0s: 1032995 executions, 637334 successful
    rsg_test.go:822: 4m40s of 5m0s: 1051691 executions, 648848 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''IomiERMg'' ''Lr'':198C,515A,768A ''NCzPouaI'':390A,587A,597C ''OOUykheFTG'':709A,948C ''QCjHn'':151A,604C,943B ''UbWTF'':882B ''hGj'' ''q'' ''uTHFLTNEZg''':::TSVECTOR, '3':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1070972 executions, 660671 successful
    rsg_test.go:822: 4m50s of 5m0s: 1090321 executions, 672703 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7916367?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7916367?buildTab=artifacts#/) on master @ [c050c9b4b57ecf2ceb5d449c31c617fe12c920e0](https://github.com/cockroachdb/cockroach/commits/c050c9b4b57ecf2ceb5d449c31c617fe12c920e0):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7273136 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 901798 executions, 555777 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''ALFRRWmX'' ''pZqHKFDu'':243B':::TSVECTOR, 777318834:::OID) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''VNe''':::TSQUERY, '00:00:00+15:59:00':::TIMETZ, e'G\x168a\x05\x06\x0e(':::STRING:::NAME, '248.25.85.84/27':::INET) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 919974 executions, 566929 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('0107000060E6100000080000000105000060E6100000090000000102000040030000002C753F39EB1265C0DE5CCF7B953F55C04C1811F59C69EE41E079AF574E5C3640808612AA7A554C40B8611D9F4D380142BA89CEAAD73962C078C19CB2C9A43F40F7C77133B5C700C201020000400700000078ABDE20E17660C0268801136E9F55C0405D4D875B6AF141A44C2886649F6340B975AB58F54F50C09C9746614B5DE1411AEA8341C6296640586C5C23983524C0B0E341D998F9FE413CA900A531F85340E0988530096A164046DDF4D7421DF74174F9489DAC65534000CD50BBF3B746405A58187496FF014290B8FB20C45D57C03219E0F3A2A5554044AEC3CC56B3FE41ECDFAB006A2F47C018A394F2F3A42F4030C96416843CFDC1010200004003000000D007A0962FFB65C058CE0B61A1E52540009FB0749C7DB1C111546BDE785F61C022489735BC5E45C0206EF424EEDABB41B8CBBA7BBB703AC0BCA2819DBA754740D14794268B5900C201020000400500000034D956AD38474C40706C512FD1AB3FC078A2D6038AEE01C28876D26BD3255D40881AC26009F837C090022119BDB1D34144944E3235226640F02029EBFD6A1FC0B044892D2ADDEA410CF2DD72DCF14A4018364CBED75E324025287775818FFBC148E7D7226C3742C06A5FBB157E1B53403E425F6BC4A200420102000040030000007BC95B512F1964C03474C492B27C4DC01EADB292924BF34116C8C058632F59C0B221A549BF5E55C0B9DB3B67208DFDC1C0A74FDDB8D806C07AEA77CE5BE152C0171F772C1F93FBC1010200004003000000B0897F0162385140D84FD0A0FE794BC000483BF502AEFA414AE7CFC54EDC604062C43B5D77F04BC0EDE165706D6EF0C150A966BB25FE404090A51B0352A81AC01B9FDDABA27B00C20102000040040000007C89247E0D435F40AC46DD7B8B8A31C0C3EB9C714C6602C2F492AE00845A53406C2698F6E9684540E0F47CA7DEA3E6C1C4554695606E45C09E33C18DEA5D544050208528D49D00C23DCC08CA2BD159C060D51E7721F04340608ED337B41FE3C1010200004004000000BA6E19F4368D6440F0823E245D8415C059182421AFF8F5C1D6A8294BE1EB6040149CF63353F14540A4CA6715C877E541ACCF93C24DBC6240E6C15FAD37805040606F0D9D139EBC4188D5E3D959A83C40EE340BE1BE4C5640100FE1CF1F68C7410102000040020000000880059DD0974840A43CE508D62B514060E300FCE262DCC10078C4C3A722D9BF5ACC81A4D49D54406195A3FDAB4FF1C10102000060E61000000700000054531F2C8D5365C070855B196B8A52C014CDADFD0A35DFC184CDCD2C758A46C035A6D616FA8045C0007B238081A49C41F22D45097E5B6040A62EEC5FD3BF4DC080C0D03783259D41707CC04406B0574050A1EE1C283039C094510B304934EDC18A009F59D81D65401C6FDDBB1CF655405271BA32B542F5C1B0339D55690629C0B812F146E5013E408006F5A4E3B9E441B06A1E33314B4DC0BA5D2B5AAD1F52402AACC559765A01420105000060E610000009000000010200004009000000A0F56634D0CC54C0F12F2C408C9C55C0C5D338CDE9B301C2945E81331C645040094ADC5E134B4AC0880B49DAAD7AFE411C299C80D82A65409C1ADCB7277B54C0BC26F12927F0E74108A493A93BF15E40AAE0FEBB00225240F41A79D1F6D1FA411CFB3F26D99E5440D8269BFD72335040101669C207B1E7C1B8349D8CEED134404E719A7AE6F45140801DF453C0559441E0D40217721B2540380A4D69554A4D4002AB0FB155DAEEC1402EABA584C101C02CC817C2E01D5340838C3833D8BAF6C12D76275D50AE61C06678B3BFA62B524068436549A111F3410102000040080000002B20A927FE5352C0B96CC8A1CE7B55C038037A663269D1412809A3CD9B403840DBA29876C7D645C0AEE300804E22F5C1FA36D0DB320B654004A318D8A1F754C058333AFF3124E24104FDB6DB087B6340AB3E257496CD4FC0F0BCDF5B5A12D041043C66C8C6FA5340788EA35124274040C61F852B8980FA41F48BD024D99E4D4072757721D6AF5040FCE938376E1EEF4124BB35229CAA44C0002D55F107E608406F867840B5AAF8C181EB587E64C059C034184810B7D53240283A991F7D60F7C101020000400500000020B2BE9703C727C0F701211E111256C0B6156CEEF344E1C1F4C98F96A2E9464054C8E9153F0A51C0673571391E2801C2FCF6109295795A40141F7DD38F454640624EE5183D76E1C1CAD27B66AF834AC0903B1E4E1DB721C0F297494A4F63F1C170B51D33CCC335C004D07D4D362834C032A67A38A3B4F14101020000400B0000007F10DD58443C65C07506882FF7E550C0F43247EC5E8600422FFFF69377F259C09645B7E78BA14FC098BA61DFE565E641C404B8E443FA45C088A0BD737E673BC080A950FE518CD441EA305B2B38214CC028756BDE9B9946C05B1044C3529601C2D8C12D8C27CD45400E6AAC7D60B04EC0100968199F62FF4114EBF6DE8F105B40B8099E06F7863BC0AE66FB689B8AF14148557885F926554040126DA48FEA30C0AC3B09B34DC2F24150CDDBDF9D062EC080A8B8E4A5AB2BC07EC731C4FBB7E5C170EB7AD44D235240400E1D4EEF0406401856AAA2FE07D2C135D05C0F4BA35CC0B2FBB50E9D185040EE1E5FDA33EFEBC1C9EA2C777CB963C0E63411BBA23455405877FF56A81401C20102000040080000007E378F4213AE5BC08910D6C43F1D55C060F8FD17D11FB341B4B711C49E8847C0D8A4F4A0B18855C075FA6BC152FDF4C190F78E0478AB27C0E8BA9C15D4313DC0D8191A55535BDA413CB9885BA91846403D722EF3021D4AC0C8FC3501077EF241BEC13FD8589C6440708F6AB5BD474940E7177947203FFCC1A0C97D6A2C772C402071BF2FF4912A40C20D89A125FC014238FCEEFFFCEB3540A8EAC2D5716952409237E8B66ADDF4C1C5A8DD0BB8B858C0D83A2F038E6956400AEEAF5EDECCE3C101020000400900000071EC43B632A357C0ED844DD1F9114AC0E040C2FC7AB1D541E4DDB9A27E995140A28317D3017254C06AC20279E92AF441581C239156E34E4070C98281AE6537C0C50E1E422943F1C1E8F92966820F5F40A0772921837715407C691952683DF6416CCD5F30EB695340B431436E56333140903C87F86A3BD5418C887737AEEE58404CB793021E8745400D03FBDB5FE700C2546C80509C7D6040CC42873D27D24B401097B8DEDC43FE41B0D29DDCFDE74A40540A46FD968753407C79E43A2DBFF8C140999AC615FA2F40F64E6D44F8465140E4A877ADB641E4410102000040050000004F6230E678B05AC0B3975E10E05A52C04C2F619227A6F3C1E2D048B65D864BC01A63FF5088A550C07E0DDDF2FC88FA41A4A5C8A73A94544081765DBAB73041C0558607F4320002C2384335D4801C4E40E810092B471E27C0962EF5142B7BF2417A516E75E09E62401269EC9EE91140401CE0EBAF763BF5C10102000040030000002B2D6B650EAC61C0683E6E450A1439C0A40C8162C9E2F6C15B15D4BDBDA35CC0FCDCAD8E3CC6434056EADB6A589C01C2A9B5697359B762C00CAC25E827F74D4078408DD25A7AF541010200004003000000AC5064EE4CA644C0145B481733EC3A404250D2F27CFEF741CAF832C64433624032A8CADD71CE514076465437247D0242B199698F3B3265C0B8DDB52574584A40BA18D72DE281F0410101000060E6100000780FD9D9DD624440706B03062018484041715959DC58FCC10103000060E6100000010000000C00000014D039F3694A66C0F5CD8F28FD614AC0F0EBA2A7D810F641254A3DBE0EAB5BC0B20CDE89D6F942C0E82AA245C485EF416015B13B5F5927C0392E8EFAA82651C002285EB320A1FA413CA3E822ABDA5240B4CFFD2FCED654C00085F55891CFB1C12C81627CDDF36240EC752656FC9D55C090F1B544E3AEE641182E45EBB2306540FCA5567BF63630C012ABB75A6A08E7C10097EECB1928EA3F3845D1B1E97C5240A876741862D6E2C148EE6A17D65454C06037A59276215640A2C9528B2D3FF6412B0104FAB05D5FC00AC5DC51D2DB52400CDFAAA8E011EBC181C799BE06E062C04203F4F07BF45540D68A3C6A40D10142AB05D97A72215EC0807F26CC7C903840009829015F92DD4114D039F3694A66C0F5CD8F28FD614AC0F0EBA2A7D810F6410103000060E610000001000000060000005CAD076EBAC541C09888F451B1702A40485817826922D0413C10AD308E7F4B401A884FD6658749C038CEA6A8658102425CE95D5136055F4020CF3893F35D384008B886CDCB86EEC1A5D5B7E2ACCB64C024B2B6D7E2065240D0FDC77839C50042855C4F00687C51C034FBCDE2640F394080E6CB62C1E2ED415CAD076EBAC541C09888F451B1702A40485817826922D0410105000060E6100000050000000102000040020000002EE618D79D2E6240E00427546CBF21C0C0AE60BF22D9F0C18005C0FF0DFB224028DC1FB6468B53405EB61B81AE3DFCC10102000040050000001E3654057968664090A2AB2F158129C0F8F0EF7941E6EA4118777CC74E884B40583CC996283E32407CDF4E31A24700C2FC96C91349594040BCE7E70E30E54B4014E82BF25EEDF6C1684F16F8150F33407074BACD16CF1F40DC07FB6212C6F3C19D6C2FD4F3735EC0E8056459EACC3340D866440022C8F2C1010200004009000000D0E89D49739534408255032F4B5042C0E0A65A503484C641B068ECAAD9913E40831AE6AFDC3F43C0E1F1052F8757F8C10828EBD82F545740C2A368AE09673FC08472CA649F07E3C120AD1833C9DB5D400A73AC9360153EC0EA7F63A67518F7419082D06D58315D407AE8A7D51FA83BC0801995C202A69F41DCE75E24654C554070A25FF7C1E51D406018DFC359FEFC41CC6875712E515E400446513CF3444A407C45C4AF4476E8C14082EB233A4C4A40E81245F876764B4040AC09E9B870AAC10085B2BDFEA6F4BF26618353CC76554094EA0DE145FDFD4101020000400900000058321941DF0041407060CB0681782AC0223E055DE1490142445B1A840AC4434000436E21E65F37C0A8449FECCF980142AC9B3570652D464000E26562B5DF30C054EE3DD46D71F9419C69687DB38E584030F60C1F756440C0BE54888C339901C20613FD52494564400DCE665F962643C08CA218229878DDC164A5A352D63D6440E02EC94BCCE941400050C419F62357C10CE0D9731E2F5A4020FCC95C35363B40F04B745B3010D94100748E194E43EB3F7C2EC239D3A74F40D8D7D7AC7DE0D9C1C342ABF6554E58C028343CE641CC5340607D2E217A20F541010200004004000000ED97DBD5D72156C01EA92941C8183CC0E0A6A7F262E2C741085B9C85BAA6594052B1B87BC46A4DC08A42C53A14A500422420F181116B56408048F86ACFD8ECBFAE5182792898E2C1C89AF68BE63131C094EC363B07CC44403AF29B3CC4C7F7410103000060E61000000100000007000000AA4FF92A953C58C0820BFEC9B04040C03D8ACCDB89BC00C2884D7E41130444407CA30DAB922E36C0680F38A22497F141D8A1A56C143E634088DDB188917B36400441A27B4FC0EEC1688559FD88E1594034571A94B8AC5340B04702E7585ECEC1083F2496100739402AA4904DC9ED554008138992AA84FC419C8EA827C0C763C0A27046A5E1D1524008B726A3DC5CDA41AA4FF92A953C58C0820BFEC9B04040C03D8ACCDB89BC00C2':::GEOGRAPHY, 32767:::INT8, '''DFadAb''':::TSQUERY, '/':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''zGpWbMnaN''':::TSQUERY, '1985-12-22':::DATE) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 938755 executions, 578604 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object(0.11645062267780304:::FLOAT8, 'ffffffff-ffff-ffff-ffff-ffffffffffff':::UUID, '''tZ''':::TSQUERY, 1328845244:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 957212 executions, 589928 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''zCNVFeYPCG''':::TSQUERY, NULL::anyelement, ARRAY[2446:::INT8,(-1):::INT8,11868:::INT8,17813:::INT8,(-21115):::INT8], 1.161521077156067:::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 974840 executions, 600871 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('15 years 5 mons 117 days 07:32:31.319683':::INTERVAL, NULL, '''D'' ''UBHtQgB'':760A ''ZHaPAod'':616B ''eIatJHOG'':103A,982A ''kd'':102A,183A ''lmlBix'':469B,538B,600B,945C ''pjLSh'':188C,386B ''vYEdpq'' ''wWgDr'':52B,504A':::TSVECTOR, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object(2871206401:::OID, (NULL,), '''JipTthxAe'':127B,632A ''U'' ''Wnv'' ''gMfHSK'' ''uIvgaKzSJI''':::TSVECTOR, 'e':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object(512343792:::OID, '''MwqaUIzvv''':::TSQUERY, '''lIUOj''':::TSQUERY, ARRAY[772449577:::OID,181525775:::OID]) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 992966 executions, 611965 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''BZTadirdxD''':::TSQUERY, (SELECT NULL)::anyelement, 1712004258:::INT8, 29661:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1011355 executions, 623307 successful
    rsg_test.go:822: 4m45s of 5m0s: 1029729 executions, 634694 successful
    rsg_test.go:822: 4m50s of 5m0s: 1048361 executions, 646207 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7923316?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7923316?buildTab=artifacts#/) on master @ [942a4d468e9c8ad0ef45a7be33f0a326dfb19fef](https://github.com/cockroachdb/cockroach/commits/942a4d468e9c8ad0ef45a7be33f0a326dfb19fef):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7582286 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 3m25s of 5m0s: 766550 executions, 472832 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''VsKs''':::TSQUERY, 'BOX(-0.866777336239279 0.46529727674449184,-0.5269713275158943 0.7496842617700966)':::BOX2D, 950757021:::OID, 1138678392919200472:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 3m30s of 5m0s: 785073 executions, 484257 successful
    rsg_test.go:822: 3m35s of 5m0s: 802718 executions, 495202 successful
    rsg_test.go:822: 3m40s of 5m0s: 821550 executions, 506876 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''AsA'' ''IjAaJm'' ''b'' ''dcfEcUX'':156A,256B,451C,805C ''dfDsxurWR'':622C ''oBb'':499B,820B ''rlvWi'':87B,413C,441A,965A ''shzY'':592C':::TSVECTOR, '7fd8:2584:1b85:10b4:c7ae:3aed:f895:b4cd/26':::INET) ;
        
        Stack trace:
    rsg_test.go:822: 3m45s of 5m0s: 840453 executions, 518550 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''Dl'':92C,100A,636A ''OadSQwZRmE'' ''YQKMfLG'' ''e'' ''jzxZcaeP'':553C,799C':::TSVECTOR, ('07:03:05.456543':::TIME, '\x9339402f1cd4391bb4':::BYTES, true, '2006-08-09 23:55:05.000473+00:00':::TIMESTAMPTZ), 0.22629323656980582:::FLOAT8, '1979-03-30 06:37:11.000266+00:00':::TIMESTAMPTZ) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 859131 executions, 530057 successful
    rsg_test.go:822: 3m55s of 5m0s: 878086 executions, 541668 successful
    rsg_test.go:822: 4m0s of 5m0s: 896836 executions, 553307 successful
    rsg_test.go:822: 4m5s of 5m0s: 915167 executions, 564707 successful
    rsg_test.go:822: 4m10s of 5m0s: 934513 executions, 576613 successful
    rsg_test.go:822: 4m15s of 5m0s: 953514 executions, 588285 successful
    rsg_test.go:822: 4m20s of 5m0s: 972496 executions, 600037 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''NQutOjUEv'':119C,120B,835C ''UVxWsfRMqB'':39C,355A,528C ''h'':408B,760B,896C,954C ''nD'':20C,578C,754B ''sMslse'':772C':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 991638 executions, 611925 successful
    rsg_test.go:822: 4m30s of 5m0s: 1010268 executions, 623413 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''befxCRXd''':::TSQUERY, e'\x117\tD_h ':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''YvQL''':::TSQUERY, 147704843:::OID, (-1197348906):::INT8, e'O\txvVDI':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 1028470 executions, 634593 successful
    rsg_test.go:822: 4m40s of 5m0s: 1047715 executions, 646506 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object(e'Tc:O\x0f|':::STRING, '':::VOID, '''BROeoR'':388B ''Fnr'':135A,195A,378C ''KehIfhBzpW'' ''PasSITlyK'':9A,101B,169C,939A ''PhUM'' ''QmaeNoXlc'' ''VurqF'':183C,981A ''zKibrH''':::TSVECTOR, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1066527 executions, 658085 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''sokO''':::TSQUERY, '1977-09-05':::DATE) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1085497 executions, 669715 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience deprecated-branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7937047?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7937047?buildTab=artifacts#/) on master @ [a80652b2e4691ea76ea49e797b1b9e0998e1d61f](https://github.com/cockroachdb/cockroach/commits/a80652b2e4691ea76ea49e797b1b9e0998e1d61f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7237623 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 3m25s of 5m0s: 838849 executions, 517641 successful
    rsg_test.go:822: 3m30s of 5m0s: 858549 executions, 529870 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''IipOFAwAH''':::TSVECTOR, true) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''CEjEUaa'':75C,533C,749C,954C ''M'':35A,361B,701C,874B ''MriHhAmZfm'':80B,302A ''Nm'':216B,957B ''NuDDXOVKc'' ''Q'' ''VdilbIJ'' ''wHfSfal''':::TSVECTOR, 2097145397:::OID) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''asmbiUMNY''':::TSQUERY, true) ;
        
        Stack trace:
    rsg_test.go:822: 3m35s of 5m0s: 879433 executions, 542830 successful
    rsg_test.go:822: 3m40s of 5m0s: 900551 executions, 555731 successful
    rsg_test.go:822: 3m45s of 5m0s: 921291 executions, 568562 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object((-0.7387379110552323):::FLOAT8, 0.6332998302573430945:::DECIMAL, '''IcJnI'':109C,462A,473B,663C ''JV'' ''WWkgkJJE'' ''h'':295B,620C,891A ''l'' ''pIshJ'':205B,446B,534C,827C ''pYXfXyVq'':12C,329A ''zpD'':935A':::TSVECTOR, ARRAY[(-12604):::INT8,(-859):::INT8]) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 941833 executions, 581162 successful
    rsg_test.go:822: 3m55s of 5m0s: 962433 executions, 593856 successful
    rsg_test.go:822: 4m0s of 5m0s: 982348 executions, 606208 successful
    rsg_test.go:822: 4m5s of 5m0s: 1002919 executions, 618951 successful
    rsg_test.go:822: 4m10s of 5m0s: 1023965 executions, 631901 successful
    rsg_test.go:822: 4m15s of 5m0s: 1045021 executions, 644892 successful
    rsg_test.go:822: 4m20s of 5m0s: 1066122 executions, 657921 successful
    rsg_test.go:822: 4m25s of 5m0s: 1087043 executions, 670903 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object(B'111101111011110011110011', ARRAY['19:59:56.049109-05:45:00':::TIMETZ,'15:29:48.401369+05:32:00':::TIMETZ,'06:14:40.076297-08:28:00':::TIMETZ,'02:15:14.569202+05:19:00':::TIMETZ], '''lDhgOpl''':::TSQUERY, ']>_':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''DemRIebXOx'' ''FjAeYLFTq'' ''GV'':674A,783A,906A ''IMJZjJpvf'':39C,251B,828B ''MPYqOl'':133B,299A,686B,955A ''PZYhz'' ''bkGP'':745A ''dXFQofSE'' ''yevLxu'':13A,315A,416B,502A':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''vfmzzG''':::TSQUERY, '2015-01-09 01:10:47.000327+00:00':::TIMESTAMPTZ, e'\x16':::STRING, ARRAY[2513611917:::OID,665971293:::OID,3580896058:::OID,0:::OID]) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 1107343 executions, 683461 successful
    rsg_test.go:822: 4m35s of 5m0s: 1128250 executions, 696442 successful
    rsg_test.go:822: 4m40s of 5m0s: 1149210 executions, 709241 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''kLbYisUbyO''':::TSQUERY, (1.8981852531433105:::FLOAT8, 3685627252:::OID, NULL, '125.6.76.177/4':::INET)) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1169781 executions, 721874 successful
    rsg_test.go:822: 4m50s of 5m0s: 1190571 executions, 734728 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-experience branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7952899?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7952899?buildTab=artifacts#/) on master @ [052acc88ad9d7296ce6b8b441627fb469cc74d95](https://github.com/cockroachdb/cockroach/commits/052acc88ad9d7296ce6b8b441627fb469cc74d95):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 8088859 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT json_build_object('''BoFEXcRg'':421C ''NM'':337B,634C ''QZpDfxuk'' ''VwKx'' ''dLG'':235C,288A,495A,558B ''echTmtX'':525C,695B ''fSFCAa'':218B ''qdnrJuai'':195A,663C ''waPV'' ''yNqPbFBiWA'':147A,582A,677C,821A':::TSVECTOR, 1228004408:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 2m40s of 5m0s: 616471 executions, 380122 successful
    rsg_test.go:822: 2m45s of 5m0s: 636377 executions, 392535 successful
    rsg_test.go:822: 2m50s of 5m0s: 656056 executions, 404657 successful
    rsg_test.go:822: 2m55s of 5m0s: 675538 executions, 416744 successful
    rsg_test.go:822: 3m0s of 5m0s: 694934 executions, 428649 successful
    rsg_test.go:822: 3m5s of 5m0s: 713596 executions, 440123 successful
    rsg_test.go:822: 3m10s of 5m0s: 733527 executions, 452559 successful
    rsg_test.go:822: 3m15s of 5m0s: 753185 executions, 464652 successful
    rsg_test.go:822: 3m20s of 5m0s: 773015 executions, 476890 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object(1693551961:::OID, '':::STRING, '''CMqxdvBb'':319B,598A,646C ''SEV'':502A,551A,859C ''URyPvl'' ''WLiHZ'':135C ''lRd'' ''xWbIvteMpU'':120A,636B ''zCu'':350B,485B,486A,558A':::TSVECTOR, 756082783:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m25s of 5m0s: 792887 executions, 489104 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''GvcASsa'' ''SxRYT'' ''UGt'':332C,343C,488A ''bcAPTjY'' ''beW'':969B ''cLT'' ''lFjA'' ''r'':169A ''tMCZpKcsOO'':39C,357A,512B,999C':::TSVECTOR, e'\x14]o\x13\x1b.':::STRING:::NAME, '''jexkwZmlO''':::TSQUERY, 942876283:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m30s of 5m0s: 812899 executions, 501479 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''oGOz''':::TSQUERY, (-1.204746961593628):::FLOAT8, '''ZFqLizTw'':208B,548A ''egkcdjjSdt'' ''hDZlqPQ'':521C ''hXn'' ''hdvTcVrvPg'' ''mkggv'':250B,298A,330C,895B ''n'':579C ''naA'' ''rJCoLaHI'':491A,648C,695C,912A ''y''':::TSVECTOR, B'0011000011111111101011000101001') ;
        
        Stack trace:
    rsg_test.go:822: 3m35s of 5m0s: 832808 executions, 513802 successful
    rsg_test.go:822: 3m40s of 5m0s: 851641 executions, 525444 successful
    rsg_test.go:822: 3m45s of 5m0s: 871027 executions, 537405 successful
    rsg_test.go:822: 3m50s of 5m0s: 890558 executions, 549516 successful
    rsg_test.go:822: 3m55s of 5m0s: 909915 executions, 561358 successful
    rsg_test.go:822: 4m0s of 5m0s: 928633 executions, 572903 successful
    rsg_test.go:822: 4m5s of 5m0s: 947820 executions, 584775 successful
    rsg_test.go:822: 4m10s of 5m0s: 966244 executions, 596147 successful
    rsg_test.go:822: 4m15s of 5m0s: 985869 executions, 608320 successful
    rsg_test.go:822: 4m20s of 5m0s: 1005276 executions, 620244 successful
    rsg_test.go:822: 4m25s of 5m0s: 1024181 executions, 631923 successful
    rsg_test.go:822: 4m30s of 5m0s: 1043337 executions, 643780 successful
    rsg_test.go:822: 4m35s of 5m0s: 1062739 executions, 655863 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object(1604527663:::OID, NULL::anyelement, '''o'':164C,359B,642C,690B':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1081629 executions, 667570 successful
    rsg_test.go:822: 4m45s of 5m0s: 1100875 executions, 679439 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''MtuMrqnlfG''':::TSQUERY, NULL::anyelement, NULL, '''TUigjUny''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1120249 executions, 691422 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7969269?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7969269?buildTab=artifacts#/) on master @ [d6f98e90684894fd36f53596e6aac355676d232e](https://github.com/cockroachdb/cockroach/commits/d6f98e90684894fd36f53596e6aac355676d232e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7723160 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        
        Stack trace:
    rsg_test.go:822: 2m55s of 5m0s: 688984 executions, 424573 successful
    rsg_test.go:822: 3m0s of 5m0s: 708338 executions, 436613 successful
    rsg_test.go:822: 3m5s of 5m0s: 728245 executions, 448878 successful
    rsg_test.go:822: 3m10s of 5m0s: 748439 executions, 461382 successful
    rsg_test.go:822: 3m15s of 5m0s: 768315 executions, 473649 successful
    rsg_test.go:822: 3m20s of 5m0s: 788399 executions, 486058 successful
    rsg_test.go:822: 3m25s of 5m0s: 808492 executions, 498354 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''AiQ'' ''HWATNofh'' ''KIOJ'':170B,607B,771B ''MikAc'' ''YnJPlWK'' ''Z'':49B,173A,517C,884C ''lPEaspUzxk''':::TSVECTOR, '4c1a:1e67:98b6:7555:9da7:872f:a9f1:b779/114':::INET) ;
        
        Stack trace:
    rsg_test.go:822: 3m30s of 5m0s: 828087 executions, 510477 successful
    rsg_test.go:822: 3m35s of 5m0s: 848036 executions, 522774 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''KuoVrgCf''':::TSQUERY, ARRAY[1502332903:::OID,1623651145:::OID,3171769727:::OID,0:::OID], 3117242032765873546:::INT8, 1127166018:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 868136 executions, 535193 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object(2754442055:::OID, 0:::OID, '''Gpay'':165C,740C':::TSVECTOR, B'1011101001000000011000111011111000110100100001') ;
        
        Stack trace:
    rsg_test.go:822: 3m45s of 5m0s: 888417 executions, 547671 successful
    rsg_test.go:822: 3m50s of 5m0s: 908525 executions, 560173 successful
    rsg_test.go:822: 3m55s of 5m0s: 929003 executions, 572842 successful
    rsg_test.go:822: 4m0s of 5m0s: 948246 executions, 584636 successful
    rsg_test.go:822: 4m5s of 5m0s: 967910 executions, 596736 successful
    rsg_test.go:822: 4m10s of 5m0s: 988028 executions, 609128 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''xQt''':::TSQUERY, 'BOX(-0.9037264309027099 -1.4146060077626412,-0.3710533736625198 1.7806230459613637)':::BOX2D, 0:::OID, ()) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 1007792 executions, 621349 successful
    rsg_test.go:822: 4m20s of 5m0s: 1027989 executions, 633769 successful
    rsg_test.go:822: 4m25s of 5m0s: 1047904 executions, 645968 successful
    rsg_test.go:822: 4m30s of 5m0s: 1067658 executions, 658247 successful
    rsg_test.go:822: 4m35s of 5m0s: 1086956 executions, 670124 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''ZuQFV''':::TSQUERY, 1478419853:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1107081 executions, 682577 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''Kw'' ''xkEY'':21A,458A':::TSVECTOR, (-5699164848927712957):::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1127214 executions, 695041 successful
    rsg_test.go:822: 4m50s of 5m0s: 1146992 executions, 707179 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7985126?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/7985126?buildTab=artifacts#/) on master @ [dcdf599e9eaee8010137167f419509ccb7627406](https://github.com/cockroachdb/cockroach/commits/dcdf599e9eaee8010137167f419509ccb7627406):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7668084 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT jsonb_build_object('''wkltyDxCX''':::TSQUERY, NULL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''WUM''':::TSQUERY, ARRAY[1717731478:::OID,2049323366:::OID], (SELECT NULL)::anyelement, '':::VOID) ;
        
        Stack trace:
    rsg_test.go:822: 3m0s of 5m0s: 703724 executions, 434116 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''ANZvR'' ''Swag'' ''bZIYIV'':993C ''csXgL'' ''ePsFsJP'':251C,575C,890C ''fDUrmqIK'':72B,615C ''xvtSUfShk'':198A,522C,624A':::TSVECTOR, '1974-04-06 19:20:40.000394':::TIMESTAMP, '8465:43ff:7c:1078:c2b5:597f:a759:6e15/22':::INET, '0103000000010000000A000000502C2EE72F3500C22059E622D058FAC16D60CDEFE243F2C11B8D1AE176C300C2A89A15FC7AFBD241465527CEC51700C2F442FCD81D96FE41642B8100EA67FAC1383FFF3D6142F041E09A0D9FAF1CE74172539366ED76F74142C5CA30B9D3F5415EBACE29AF03EEC158CF89CE2FC4D94174A4DA24825CF9C1AC5C92525D4FE1410E6BC0A22D5D00C2B8CDD002DB21D541502C2EE72F3500C22059E622D058FAC1':::GEOMETRY) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''QaaOKjUPOW'' ''ffXZJZ'':143B,458A,664A ''lFB''':::TSVECTOR, '0101000040626E5F2B2E6EEFC19056531445E3FB41A2D5AE9FC6FBF541':::GEOMETRY) ;
        
        Stack trace:
    rsg_test.go:822: 3m5s of 5m0s: 723635 executions, 446526 successful
    rsg_test.go:822: 3m10s of 5m0s: 743386 executions, 458663 successful
    rsg_test.go:822: 3m15s of 5m0s: 763526 executions, 471098 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''fVbO''':::TSQUERY, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 3m20s of 5m0s: 783475 executions, 483423 successful
    rsg_test.go:822: 3m25s of 5m0s: 803779 executions, 495978 successful
    rsg_test.go:822: 3m30s of 5m0s: 823068 executions, 507874 successful
    rsg_test.go:822: 3m35s of 5m0s: 843255 executions, 520459 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('1988-05-11 07:39:07.000297':::TIMESTAMP, '':::VOID, '''FYtZASW''':::TSQUERY, '02:13:25.780413':::TIME) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 862915 executions, 532487 successful
    rsg_test.go:822: 3m45s of 5m0s: 883265 executions, 545074 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object((-20281):::INT8, 3241728238:::OID, '''VMHUvYQT'' ''etWunVH'' ''kRd'' ''qbq'':173C,862C,981B ''rG'':825A,912C,978A ''sa'':41B,386B,603A,640C ''xOljAusA'':705A':::TSVECTOR, '4a1d:fc8f:119a:d0a0:62dc:5a96:a1a3:7cd9/43':::INET) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 903958 executions, 557862 successful
    rsg_test.go:822: 3m55s of 5m0s: 924645 executions, 570539 successful
    rsg_test.go:822: 4m0s of 5m0s: 943905 executions, 582441 successful
    rsg_test.go:822: 4m5s of 5m0s: 964242 executions, 594992 successful
    rsg_test.go:822: 4m10s of 5m0s: 984549 executions, 607527 successful
    rsg_test.go:822: 4m15s of 5m0s: 1005031 executions, 620051 successful
    rsg_test.go:822: 4m20s of 5m0s: 1025129 executions, 632484 successful
    rsg_test.go:822: 4m25s of 5m0s: 1045601 executions, 645165 successful
    rsg_test.go:822: 4m30s of 5m0s: 1065143 executions, 657146 successful
    rsg_test.go:822: 4m35s of 5m0s: 1085090 executions, 669480 successful
    rsg_test.go:822: 4m40s of 5m0s: 1105095 executions, 681671 successful
    rsg_test.go:822: 4m45s of 5m0s: 1124438 executions, 693648 successful
    rsg_test.go:822: 4m50s of 5m0s: 1143154 executions, 705141 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8000031?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8000031?buildTab=artifacts#/) on master @ [7c8e7012b73a598f3aa1da9bbf422e833706f5d3](https://github.com/cockroachdb/cockroach/commits/7c8e7012b73a598f3aa1da9bbf422e833706f5d3):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7079231 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 4m5s of 5m0s: 850596 executions, 524774 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x30ba61':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 868238 executions, 535739 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 884310 executions, 545578 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 901842 executions, 556469 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''Rzyduk''':::TSVECTOR, '''OS''':::TSQUERY) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 918964 executions, 566948 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''QfGJTUQ''':::TSQUERY, (SELECT NULL)::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 936700 executions, 577807 successful
    rsg_test.go:822: 4m35s of 5m0s: 953584 executions, 588188 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''AhdkP'':90A,344A,563C,837B ''BSwbXOIgbP'':215A,611B,725C,987C ''OiDleZh'':163B,291C,759B ''QOBuDNLf'':130B,159C,459B ''WYbfuj'' ''YRrdwCye'' ''pLc'':16A,318C ''vUdt'' ''wucQWS''':::TSVECTOR, 1598542284:::OID, '''JddQs''':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 970309 executions, 598492 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''xF''':::TSQUERY, 5461055302110899209:::INT8, '23:36:13.53376+10:59:00':::TIMETZ, (-1842517457):::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 987393 executions, 609061 successful
    rsg_test.go:822: 4m50s of 5m0s: 1005507 executions, 620108 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8014306?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8014306?buildTab=artifacts#/) on master @ [94e03c016955dfd64d4e15358ea92226a8f362aa](https://github.com/cockroachdb/cockroach/commits/94e03c016955dfd64d4e15358ea92226a8f362aa):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6758023 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT json_build_object('''AxdiY'':513A,634B,900A ''NNgwmXj'':112B,862C':::TSVECTOR, 2920351879:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 944392 executions, 582266 successful
    rsg_test.go:822: 4m25s of 5m0s: 962671 executions, 593496 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''v''':::TSQUERY, 2452608845:::OID, 1801505119:::OID, NULL) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 981508 executions, 605066 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 999381 executions, 616052 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''prBecDAUwK'':326B,957C':::TSVECTOR, B'00110010000000', true, e'\x1fp1$C2q\x1c':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1018052 executions, 627607 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''JLYTyB''':::TSQUERY, NULL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1036597 executions, 639078 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1054733 executions, 650198 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''FJgynZi'' ''OQMd'' ''SbX'' ''WBBcQanB'':88B,172A,174B,736C ''u''':::TSVECTOR, '1984-08-06':::DATE, 3236475575:::OID, '':::VOID) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8030993?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8030993?buildTab=artifacts#/) on master @ [61f7c816574848f804f1adf7f42f7315bfcaaeba](https://github.com/cockroachdb/cockroach/commits/61f7c816574848f804f1adf7f42f7315bfcaaeba):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7103478 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 909311 executions, 560495 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''Krs''':::TSQUERY, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''Gf'' ''HVYp'' ''TaLY'':382B,500B,601C ''hdpvHQfxY'':435B,438A,725C,976A ''pNCqezxHnD'':499A,606A':::TSVECTOR, e'r+FP\t\x1b9g\x04':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''AkwRbpO'':539B ''IrjFgKdchL'' ''K'':588C ''PRwF'' ''ZCOav'' ''dSLnzZUqp'':644C ''fivHLHo'' ''ttdDFqNzaX'':151B,267C ''xIjRAF''':::TSVECTOR, e'\x00':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 927209 executions, 571472 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 944964 executions, 582486 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('':::STRING, '''uj''':::TSQUERY, '''Jj'' ''K'':375C,534C,819A ''OVufydo'':617A,792B ''XnGVYiWfX'' ''cAMfscLAJX'':501B,902A ''cAPf'' ''n'' ''nvCrgx'':336B,672C,691C ''sxhMdW'' ''zo''':::TSVECTOR, 's':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 962316 executions, 593097 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''JVVXB''':::TSQUERY, (SELECT NULL)::anyelement, 3462006614:::OID, false) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 979932 executions, 603992 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 997762 executions, 615036 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8046103?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8046103?buildTab=artifacts#/) on master @ [10266a323f94c3cf397d5de590e512d987e63e22](https://github.com/cockroachdb/cockroach/commits/10266a323f94c3cf397d5de590e512d987e63e22):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7365061 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''Ep'':87B,815C ''HNIc'':442B ''RZvNLEHQ'' ''cO'' ''eRrRJC'' ''f'':187A,622B,682A,780B ''hjnZgdnBKM'' ''rZZjdL'':417A,628B ''xDiYeGgfyJ'':301A,537B,953A,980C':::TSVECTOR, '00:05:29.538948':::TIME) ;
        
        Stack trace:
    rsg_test.go:822: 4m5s of 5m0s: 866876 executions, 534729 successful
    rsg_test.go:822: 4m10s of 5m0s: 884086 executions, 545344 successful
    rsg_test.go:822: 4m15s of 5m0s: 901903 executions, 556338 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''DNLRwIGTJ''':::TSQUERY, 3262067322:::OID, ('':::VOID, '@?P':::STRING, (-808510523):::INT8, '''lqgsbiMKm''':::TSQUERY), '':::VOID) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 919867 executions, 567313 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x3006':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 937457 executions, 578222 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 955285 executions, 589159 successful
    rsg_test.go:822: 4m35s of 5m0s: 973177 executions, 600165 successful
    rsg_test.go:822: 4m40s of 5m0s: 991211 executions, 611267 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''Ei''':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''uaHQxfK''':::TSQUERY, '\xbb7326ac945ecd':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1008130 executions, 621713 successful
    rsg_test.go:822: 4m50s of 5m0s: 1025733 executions, 632589 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8061583?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8061583?buildTab=artifacts#/) on master @ [15bc0c4780681d95e705c22dbe2e7d8d190054ef](https://github.com/cockroachdb/cockroach/commits/15bc0c4780681d95e705c22dbe2e7d8d190054ef):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6755656 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\xbd09dc69da34':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 929398 executions, 573295 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 947233 executions, 584257 successful
    rsg_test.go:822: 4m35s of 5m0s: 965309 executions, 595490 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''EX'':290B ''JMh'' ''MHt'':629B ''sJuVtheDn'' ''sSjChhVP''':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''VBueuQHO''':::TSQUERY, (SELECT NULL)::anyelement, 1650604902:::OID, (SELECT NULL)::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 983455 executions, 606640 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''AdlynC'':253A,268C,883B ''NfUA'':77A,357C,751B,820A,907C':::TSVECTOR, (-0.5118666291236877):::FLOAT8, (B'1101010', ARRAY[]:::INT2[]), (-0.04468802532591043927):::DECIMAL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1001398 executions, 617684 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1018448 executions, 628164 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\xe0d401c70a':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8078821?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8078821?buildTab=artifacts#/) on master @ [4232883add85a151c423c45904ac4096d04656c5](https://github.com/cockroachdb/cockroach/commits/4232883add85a151c423c45904ac4096d04656c5):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6950317 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 4m5s of 5m0s: 845514 executions, 521747 successful
    rsg_test.go:822: 4m10s of 5m0s: 862996 executions, 532392 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 880290 executions, 543102 successful
    rsg_test.go:822: 4m20s of 5m0s: 898089 executions, 554186 successful
    rsg_test.go:822: 4m25s of 5m0s: 915714 executions, 564916 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''ncAQgQkxaO''':::TSQUERY, 486026915:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 933520 executions, 575968 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''LOF'':104C,716A,832C,983C ''VZnqDDBy'':652A,962C ''kkzvRJfv'':10C,571A,995B ''rKvRvqWE'':222A,616C,841B':::TSVECTOR, ARRAY[]:::VARCHAR[]) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 951516 executions, 587021 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''A'':437C,456B,860A ''ArpOy'' ''Fdz'' ''KgVyoOegE'':445B,479B,785A,992B ''OmQt'' ''RFls'':357C ''UOeCjw'':112B,965A ''UcKq'':409A ''hExHmGG''':::TSVECTOR, B'10101011001000010100010010101') ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''ZYrWRLML''':::TSQUERY, NULL::anyelement, '1979-03-03':::DATE, NULL) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 968916 executions, 597690 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 986842 executions, 608669 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1004904 executions, 619894 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8087939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8087939?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7276797 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''LXhPbNfBpD'':461B,760B ''LhynRKmZ'' ''O'' ''RFv'' ''gsjDtcNjZE'' ''jQgXSjxbgw'' ''lAyORKdZ''':::TSVECTOR, (2237202305:::OID,), 'sno%/':::STRING, 'BOX(-2.95971333263753 -0.6052194549405248,0.8673645399337768 1.1383697682208302)':::BOX2D) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 914518 executions, 563679 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 932002 executions, 574518 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''EwHrFgjmu'':932A ''S'' ''XEfVmVjK'':130C,635B':::TSVECTOR, '\x0c26':::BYTES, e'r\x1azJC\x01y$c':::STRING:::NAME, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x3057':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 949646 executions, 585316 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 967331 executions, 596370 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object(1.710302075336425473E+27:::DECIMAL, (3340952653:::OID, ARRAY['0.0.0.0/0':::INET]), '''wfMPyyFKn''':::TSVECTOR, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 984696 executions, 607002 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1002089 executions, 617680 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8094307?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8094307?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6995363 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 4m25s of 5m0s: 922456 executions, 568705 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''EcYKD'':470C ''UtDMbke'' ''axQaZ'':367A,556A ''fVDwHjH'':161C,845A ''hRqBu'':395A,744B,954A':::TSVECTOR, '01020000A0E610000004000000B4EF2235A3D154404C9CD960C5BD3BC07C2B0404DE3DE04168D4209948036440B02692EF7D024640F8C2F1D23B9DEF416816590D045F6140300253E7A862414070525AE10674D64198DADF6710454B40E8D4B6DBDFB02640102B737BD3EBDC41':::GEOGRAPHY, (0.6486717139141555:::FLOAT8, '2023-02-02 19:06:48.000093':::TIMESTAMP), NULL) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 939863 executions, 579338 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 957623 executions, 590322 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''NcHt''':::TSQUERY, NULL::anyelement, '1975-02-08':::DATE, 'BOX(-1.4559647265382956 -0.9800932796833278,0.31757297995036293 1.38009421177106)':::BOX2D) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 975881 executions, 601662 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''Xl''':::TSQUERY, 864910997:::OID) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''ckklOR''':::TSQUERY, '-12 years -7 mons -16 days -13:06:17.820123':::INTERVAL) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 993690 executions, 612713 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\xcd59ee30f425':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1011555 executions, 623659 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8100531?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8100531?buildTab=artifacts#/) on master @ [c4bde8b72cdd4016845ae70ef5162b3f11fab1fb](https://github.com/cockroachdb/cockroach/commits/c4bde8b72cdd4016845ae70ef5162b3f11fab1fb):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7063471 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 4m25s of 5m0s: 922308 executions, 569251 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''HDk''':::TSQUERY, '18:25:27.21766+04:38:00':::TIMETZ) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 939967 executions, 580075 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 957378 executions, 590775 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''FujDlDm'':410B,854C,887B ''KJxnJkZw'':439A,626B ''XgaCNWQRW'':352A,583A,610C':::TSVECTOR, '1988-07-19':::DATE, '':::VOID, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 975103 executions, 601750 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 992810 executions, 612611 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''CPDUx''':::TSQUERY, (SELECT NULL)::anyelement, B'111010110010001110100110100011001000011', 425704774:::OID) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''gnOq''':::TSQUERY, (SELECT NULL)::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1010185 executions, 623292 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8107009?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8107009?buildTab=artifacts#/) on master @ [9c5375f6a7375724cdbcbaa0029ed97a230d7abe](https://github.com/cockroachdb/cockroach/commits/9c5375f6a7375724cdbcbaa0029ed97a230d7abe):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7304330 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m0s of 5m0s: 855998 executions, 528106 successful
    rsg_test.go:822: 4m5s of 5m0s: 873120 executions, 538641 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 891178 executions, 549709 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''VHY''':::TSQUERY, '1984-04-08 16:45:56.000755':::TIMESTAMP) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 908900 executions, 560573 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''AVsyDd'' ''PuOy'':781A ''Pwep'':132B,220C,524C ''X'':385A,614A ''ZnRPqvp'' ''cnzQryc'':9A,23C,374A,541B,660C ''gxGojjwHgD'' ''puHxlHCy'':662B,798C ''tDoEyuB'':898B ''uORVYZj'':435B,508C':::TSVECTOR, 5.165386099595408398E+35:::DECIMAL, 1.2815570831298828:::FLOAT8, ARRAY['w':::STRING,':':::STRING,')':::STRING,e'\t':::STRING]) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 926790 executions, 571503 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 944821 executions, 582689 successful
    rsg_test.go:822: 4m30s of 5m0s: 962510 executions, 593487 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 979716 executions, 604111 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\xcdc31aa55d7c87':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 997592 executions, 615211 successful
    rsg_test.go:822: 4m45s of 5m0s: 1015364 executions, 626141 successful
    rsg_test.go:822: 4m50s of 5m0s: 1033235 executions, 637226 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8117089?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8117089?buildTab=artifacts#/) on master @ [6723e00a46aaa3ea575093bd82a02b7d6f6b131b](https://github.com/cockroachdb/cockroach/commits/6723e00a46aaa3ea575093bd82a02b7d6f6b131b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7001209 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''oD''':::TSQUERY, 'o':::STRING, 3587936738:::OID, '>':::STRING:::NAME) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''iM''':::TSQUERY, 3736851102:::OID, e'\x0b':::STRING, 28037:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 966367 executions, 595886 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 984292 executions, 607021 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1001248 executions, 617427 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1019039 executions, 628343 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1037457 executions, 639646 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8127829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8127829?buildTab=artifacts#/) on master @ [64e4fc9faa4e0ab19fe5ba78f053bc2b1390cb5e](https://github.com/cockroachdb/cockroach/commits/64e4fc9faa4e0ab19fe5ba78f053bc2b1390cb5e):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7292367 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 4m5s of 5m0s: 941815 executions, 580560 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 961652 executions, 592899 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''WiXE''':::TSQUERY, B'100001111110011011101001001011001100011000101') ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 981538 executions, 605072 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 1000281 executions, 616602 successful
    rsg_test.go:822: 4m25s of 5m0s: 1019788 executions, 628691 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''GJyUNv''':::TSQUERY, NULL, (-0.8238477357459884):::FLOAT8, e'\\':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''JhMqbVp'':123B,261A ''ctyocToIOl'':131A,495A,540C,915B ''sdgdklTWZ'' ''wliLEZTR'':110A,123C,339B,850A':::TSVECTOR, B'01111111111111111111111111111111') ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 1039620 executions, 640842 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''zEaCIF''':::TSQUERY, 3731402832:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 1058939 executions, 652807 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1079098 executions, 665231 successful
    rsg_test.go:822: 4m45s of 5m0s: 1099353 executions, 677752 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1118236 executions, 689446 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8138870?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8138870?buildTab=artifacts#/) on master @ [250238cd29102391dddbc8cc71380090c49ce509](https://github.com/cockroachdb/cockroach/commits/250238cd29102391dddbc8cc71380090c49ce509):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7938389 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('11:22:15.725828+12:06:00':::TIMETZ, ARRAY[2556698296:::OID], '''M'' ''acRRyNEZKH'':213A,263A,508C,796B ''diH'' ''etdl'' ''l'':351C,599C,621C,762B ''sHBhYTYvnI''':::TSVECTOR, '1970-01-01':::DATE) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 1032227 executions, 636507 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''mQtVEEphV''':::TSQUERY, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 1052369 executions, 648919 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x3d77a02314e00d1c':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''O'':348C,894A ''ZVzKZDxRj''':::TSVECTOR, NULL, 0.25089672207832336:::FLOAT8, '\xbd13b4f47a89':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 1071932 executions, 660943 successful
    rsg_test.go:822: 4m35s of 5m0s: 1091817 executions, 673285 successful
    rsg_test.go:822: 4m40s of 5m0s: 1111891 executions, 685678 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''pn''':::TSQUERY, (-1.1798017024993896):::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x889d5bf8acb436':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1130768 executions, 697276 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''dXMfkSjexX''':::TSQUERY, 'BOX(-0.7823079530151896 -1.548989373385941,1.0784166905588028 1.4004935922211361)':::BOX2D) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1150806 executions, 709636 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''LflEMa'':30C,254C,815B,868C ''RBxnfXCIo'':130B,423C ''fjqX'' ''gx'' ''nsRFMTxnKi'':150C,661A,964C ''tgYXHmFy''':::TSVECTOR, NULL::anyelement, '\xda':::BYTES, '{"test": "json"}':::JSONB) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8147266?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8147266?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7800556 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 4m25s of 5m0s: 1009188 executions, 622057 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 1028314 executions, 633905 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''XJr'' ''mWccnuVlS'':119C':::TSVECTOR, 'd4ec:7265:1b29:1f4c:2266:638f:6159:b56e/41':::INET) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object(B'01010000101100011100101100', (-7772284719586917390):::INT8, '''AcAO'' ''CzDRbES'':532B ''JbE'':56C,135C,681B,959B ''MfVBg'' ''UC'':225A,380A ''gKIfiHPVnQ'' ''gc'':42A,82C,135A,928B ''sNtlh'':529C,914B ''yWbIt''':::TSVECTOR, '2015-01-07 14:50:24.000758':::TIMESTAMP) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 1047453 executions, 645705 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x28e6ffdaf5d1dbff25':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 1065860 executions, 657066 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''jV''':::TSQUERY, '':::VOID, 6.787637315526741086E+30:::DECIMAL, 0.7042511105537415:::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1084773 executions, 668725 successful
    rsg_test.go:822: 4m50s of 5m0s: 1104001 executions, 680657 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8153415?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8153415?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7152526 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 895076 executions, 551555 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 912568 executions, 562370 successful
    rsg_test.go:822: 4m25s of 5m0s: 929720 executions, 572959 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x299a0690ed1f2aeea0':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 947537 executions, 583879 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 965263 executions, 594883 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 983082 executions, 605869 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 1000447 executions, 616622 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1017804 executions, 627421 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('2006-10-11 01:03:45.000615':::TIMESTAMP, 954746893:::OID, '''oai''':::TSQUERY, '':::STRING:::NAME) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8159338?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8159338?buildTab=artifacts#/) on master @ [0725273ac7f789ba8ed78aacaf73cc953ca47fe8](https://github.com/cockroachdb/cockroach/commits/0725273ac7f789ba8ed78aacaf73cc953ca47fe8):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7209545 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''HdBXyVyri'' ''ViVetjUGvB''':::TSVECTOR, '''QZozvXi'' ''SMcQF'' ''fzNUfRgfDS''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 921740 executions, 568494 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''Azhy''':::TSQUERY, 17085:::INT8, 1868627725:::OID, 0.4506057464900973:::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''zC'':279B,348A,866A':::TSVECTOR, 0.4794994592666626:::FLOAT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 940013 executions, 579756 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 958077 executions, 590988 successful
    rsg_test.go:822: 4m35s of 5m0s: 975989 executions, 601961 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m40s of 5m0s: 993418 executions, 612749 successful
    rsg_test.go:822: 4m45s of 5m0s: 1011177 executions, 623611 successful
    rsg_test.go:822: 4m50s of 5m0s: 1028973 executions, 634530 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8166115?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8166115?buildTab=artifacts#/) on master @ [1d7bd69205c2197ccac33df9e2e6d4ff8c0fdbcf](https://github.com/cockroachdb/cockroach/commits/1d7bd69205c2197ccac33df9e2e6d4ff8c0fdbcf):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 7101493 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 864555 executions, 533234 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x95b42ad7f944d8':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''OVyC''':::TSQUERY, 2796818152144886650:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 881297 executions, 543578 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 899185 executions, 554577 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''UceVMNOD''':::TSQUERY, '0106000000030000000103000000010000000900000074115A7E6CA4E3C13456F8E0E34500C2ECF5CB9325F7E141B4B05A04CE9AFFC1CA667220BFBEF241B00FD1249238FDC1DCA954D8D133FF4166EA2ED1C62BEDC1545BD4109E68F141B0C4224DDEFAD6C16AE033A8455AF74148F9ACDC4E3DE941EEF6482282E4F54172A34CCF9F00F8417C1403F7BB85EC410070253BD6AE6C4174115A7E6CA4E3C13456F8E0E34500C20103000000010000000C0000003B2D87D493BBF9C1F05D54495108D2C1728F8049D236E8C1884E9B652A91D0C19C456656F3E9D9C13586280EA571F2C1406C1AC2F955A141C8752C0293CAD4C10C0F1F790B44F4412049D9833C03C5C1A05D69EEB762E241C8024EC9DEDDE5414C28BBC780D1F1412C32319E5AF7004244CC5E283EB0ED41588C75C2B785FE41E4A3C1E31A99E4410887C5BEF66302426077627420B8C541E8442D4959ADFE4106AA98BA575DE7C16E551116058700423B2D87D493BBF9C1F05D54495108D2C10103000000010000000D0000009E1432365B6200C2C48B36DAF82B01C2AA8FDE66D35BF441321F6268EE45F4C1FE2AC622808100420D189DC6156A00C276B857260417F841BC0126079834E9C1D464637F49C6FC41A0AC99CBA392B3415C5C86DCFC0F004214FADEB83D83E04140D6994E9B0FEB41802AE3447E8EA7C152B6FD97A71202420807449EEC5EEC41C0F2466A314EA8C190B5B7999B36E54125EA42C2915AF8C1A89C84B498E6F641142A31FD5562F6C1A006BAA50D7FB9C16FA3529AEB13F1C110352EA4D052D0C19E1432365B6200C2C48B36DAF82B01C2':::GEOMETRY) ;
        
        Stack trace:
    rsg_test.go:822: 4m25s of 5m0s: 917033 executions, 565626 successful
    rsg_test.go:822: 4m30s of 5m0s: 934732 executions, 576549 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''Bww'' ''HIXEw'':428B,469C ''aWDsLl''':::TSVECTOR, '-19 years -10 mons -194 days -04:46:15.507402':::INTERVAL) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\x':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 4m35s of 5m0s: 952387 executions, 587522 successful
    rsg_test.go:822: 4m40s of 5m0s: 969948 executions, 598257 successful
    rsg_test.go:822: 4m45s of 5m0s: 987757 executions, 609300 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: Payload.Type called on a payload with an unknown details type: <nil>
        SELECT crdb_internal.job_payload_type('\xe5b4cf33ab1b9e52':::BYTES) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('\x0062e2c242':::BYTES, '00:00:00':::TIME, '''B'' ''KXuVj'' ''cRVHNLsowv'':749A,897B,944C ''vjsECf'':600A,693A,909C':::TSVECTOR, '''IyKq'' ''NlXnVR'':129A,533C ''ZFxSng''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 4m50s of 5m0s: 1004789 executions, 619783 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8230919?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8230919?buildTab=artifacts#/) on master @ [9b567e3439e49c109e5204c9d0afec49c1a3169b](https://github.com/cockroachdb/cockroach/commits/9b567e3439e49c109e5204c9d0afec49c1a3169b):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5528259 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
*   | 	github.com/cockroachdb/cockroach/pkg/sql/values.go:80
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec.func2
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:523
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:112
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:297
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visit
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:79
*   | github.com/cockroachdb/cockroach/pkg/sql.walkPlan
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:43
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:526
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planNodeToRowSource).Start
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan_node_to_row_source.go:173
*   | github.com/cockroachdb/cockroach/pkg/sql/colflow.(*FlowCoordinator).Start.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/colflow/flow_coordinator.go:122
* Wraps: (4) runtime error: makeslice: cap out of range
* Error types: (1) *withstack.withStack (2) *safedetails.withSafeDetails (3) *withstack.withStack (4) runtime.errorString
*
    rsg_test.go:849: Crash detected: server panic: driver: bad connection
        SELECT crdb_internal.generate_test_objects(e'\rpN':::STRING, 4079819717277530702:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 2m45s of 5m0s: 308814 executions, 190331 successful
    rsg_test.go:822: 2m50s of 5m0s: 308840 executions, 190348 successful
    rsg_test.go:822: 2m55s of 5m0s: 309253 executions, 190524 successful
    rsg_test.go:822: 3m0s of 5m0s: 309267 executions, 190537 successful
    rsg_test.go:822: 3m5s of 5m0s: 310397 executions, 191324 successful
    rsg_test.go:822: 3m10s of 5m0s: 310448 executions, 191356 successful
    rsg_test.go:822: 3m15s of 5m0s: 310448 executions, 191356 successful
    rsg_test.go:822: 3m20s of 5m0s: 310448 executions, 191356 successful
    rsg_test.go:822: 3m25s of 5m0s: 310448 executions, 191356 successful
    rsg_test.go:822: 3m30s of 5m0s: 310449 executions, 191356 successful
    rsg_test.go:822: 3m35s of 5m0s: 310449 executions, 191356 successful
    rsg_test.go:822: 3m40s of 5m0s: 310449 executions, 191356 successful
    rsg_test.go:822: 3m45s of 5m0s: 310449 executions, 191356 successful
    rsg_test.go:822: 3m50s of 5m0s: 311118 executions, 191659 successful
    rsg_test.go:822: 3m55s of 5m0s: 311125 executions, 191666 successful
    rsg_test.go:822: 4m0s of 5m0s: 311145 executions, 191682 successful
    rsg_test.go:822: 4m5s of 5m0s: 311146 executions, 191682 successful
    rsg_test.go:822: 4m10s of 5m0s: 312327 executions, 192503 successful
    rsg_test.go:822: 4m15s of 5m0s: 312374 executions, 192531 successful
    rsg_test.go:822: 4m20s of 5m0s: 313005 executions, 192808 successful
    rsg_test.go:822: 4m25s of 5m0s: 313203 executions, 192949 successful
    rsg_test.go:822: 4m30s of 5m0s: 314155 executions, 193618 successful
    rsg_test.go:822: 4m35s of 5m0s: 314279 executions, 193687 successful
    rsg_test.go:822: 4m40s of 5m0s: 314451 executions, 193777 successful
    rsg_test.go:822: 4m45s of 5m0s: 314451 executions, 193777 successful
    rsg_test.go:822: 4m50s of 5m0s: 314876 executions, 193961 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8223831?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8223831?buildTab=artifacts#/) on master @ [7594fb2d1438dd5ed8f80a887b8d60b6cdd296ed](https://github.com/cockroachdb/cockroach/commits/7594fb2d1438dd5ed8f80a887b8d60b6cdd296ed):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5507342 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.(*evaluator).EvalFuncExpr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:479
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FuncExpr).Eval
*   | 	github.com/cockroachdb/cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sem/tree/eval_expr_generated.go:276
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.Expr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:26
*   | github.com/cockroachdb/cockroach/pkg/sql.(*valuesNode).startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/values.go:80
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec.func2
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:523
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:112
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:297
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visit
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:79
*   | github.com/cockroachdb/cockroach/pkg/sql.walkPlan
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:43
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:526
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planNodeToRowSource).Start
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan_node_to_row_source.go:173
*   | github.com/cockroachdb/cockroach/pkg/sql/colflow.(*FlowCoordinator).Start.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/colflow/flow_coordinator.go:122
* Wraps: (4) runtime error: makeslice: cap out of range
* Error types: (1) *withstack.withStack (2) *safedetails.withSafeDetails (3) *withstack.withStack (4) runtime.errorString
*
    rsg_test.go:849: Crash detected: server panic: driver: bad connection
        SELECT crdb_internal.generate_test_objects('E':::STRING, 3945960761224017332:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 3m21s of 5m0s: 230349 executions, 142114 successful
    rsg_test.go:822: 3m26s of 5m0s: 232145 executions, 143217 successful
    rsg_test.go:822: 3m31s of 5m0s: 232227 executions, 143269 successful
    rsg_test.go:822: 3m36s of 5m0s: 233408 executions, 144091 successful
    rsg_test.go:822: 3m41s of 5m0s: 234087 executions, 144400 successful
    rsg_test.go:822: 3m46s of 5m0s: 234792 executions, 144863 successful
    rsg_test.go:822: 3m51s of 5m0s: 235313 executions, 145259 successful
    rsg_test.go:822: 3m56s of 5m0s: 235958 executions, 145554 successful
    rsg_test.go:822: 4m1s of 5m0s: 236164 executions, 145708 successful
    rsg_test.go:822: 4m6s of 5m0s: 237219 executions, 146438 successful
    rsg_test.go:822: 4m11s of 5m0s: 237844 executions, 146707 successful
    rsg_test.go:822: 4m16s of 5m0s: 239045 executions, 147548 successful
    rsg_test.go:822: 4m21s of 5m0s: 239716 executions, 147861 successful
    rsg_test.go:822: 4m26s of 5m0s: 239743 executions, 147883 successful
    rsg_test.go:822: 4m31s of 5m0s: 240924 executions, 148720 successful
    rsg_test.go:822: 4m36s of 5m0s: 241196 executions, 148869 successful
    rsg_test.go:822: 4m41s of 5m0s: 241623 executions, 149056 successful
    rsg_test.go:822: 4m46s of 5m0s: 242805 executions, 149891 successful
    rsg_test.go:822: 4m51s of 5m0s: 243419 executions, 150180 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8255177?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8255177?buildTab=artifacts#/) on master @ [b5cffc06bf36ce42b601d3307a1e38f933be40c4](https://github.com/cockroachdb/cockroach/commits/b5cffc06bf36ce42b601d3307a1e38f933be40c4):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6331103 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:148
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:884
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).genMultipleTables.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/tables.go:42
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:884
*   | runtime.panicmakeslicecap
*   | 	GOROOT/src/runtime/slice.go:33
*   | runtime.makeslice
*   | 	GOROOT/src/runtime/slice.go:100
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).selectNamesAndTemplates
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/tables.go:98
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).genMultipleTables
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/tables.go:58
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).maybeGenSchemas
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/recurse.go:49
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).maybeGenDatabases
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/recurse.go:32
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).Generate.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:189
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).Generate
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:204
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planner).GenerateTestObjects
*   | 	github.com/cockroachdb/cockroach/pkg/sql/generate_objects.go:65
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/builtins.glob..func291
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/builtins.go:6383
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.(*evaluator).EvalFuncExpr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:479
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FuncExpr).Eval
*   | 	github.com/cockroachdb/cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sem/tree/eval_expr_generated.go:276
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.Expr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:26
*   | github.com/cockroachdb/cockroach/pkg/sql.(*valuesNode).startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/values.go:80
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec.func2
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:523
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:112
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:297
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visit
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:79
*   | github.com/cockroachdb/cockroach/pkg/sql.walkPlan
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:43
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:526
* Wraps: (4) runtime error: makeslice: cap out of range
* Error types: (1) *withstack.withStack (2) *safedetails.withSafeDetails (3) *withstack.withStack (4) runtime.errorString
*
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8240120?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8240120?buildTab=artifacts#/) on master @ [27ebfa830ac7122fc3b286136158fef615b20b8f](https://github.com/cockroachdb/cockroach/commits/27ebfa830ac7122fc3b286136158fef615b20b8f):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6477786 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
*   | runtime.gopanic
*   | 	GOROOT/src/runtime/panic.go:884
*   | runtime.panicmakeslicecap
*   | 	GOROOT/src/runtime/slice.go:33
*   | runtime.makeslice
*   | 	GOROOT/src/runtime/slice.go:100
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).selectNamesAndTemplates
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/tables.go:98
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).genMultipleTables
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/tables.go:58
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).maybeGenSchemas
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/recurse.go:49
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).maybeGenDatabases
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/recurse.go:32
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).Generate.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:189
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).Generate
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:204
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planner).GenerateTestObjects
*   | 	github.com/cockroachdb/cockroach/pkg/sql/generate_objects.go:65
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/builtins.glob..func291
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/builtins.go:6383
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.(*evaluator).EvalFuncExpr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:479
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FuncExpr).Eval
*   | 	github.com/cockroachdb/cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sem/tree/eval_expr_generated.go:276
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.Expr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:26
*   | github.com/cockroachdb/cockroach/pkg/sql.(*valuesNode).startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/values.go:80
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec.func2
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:523
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:112
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:297
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visit
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:79
*   | github.com/cockroachdb/cockroach/pkg/sql.walkPlan
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:43
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:526
* Wraps: (4) runtime error: makeslice: cap out of range
* Error types: (1) *withstack.withStack (2) *safedetails.withSafeDetails (3) *withstack.withStack (4) runtime.errorString
*
    rsg_test.go:822: 4m50s of 5m0s: 704418 executions, 433911 successful
    rsg_test.go:849: Crash detected: server panic: driver: bad connection
        SELECT crdb_internal.generate_test_objects('X':::STRING, 1996584401953223823:::INT8) ;
        
        Stack trace:
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8270431?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8270431?buildTab=artifacts#/) on master @ [cec8af84bf01918d28b1e06a29b18ac9bee7cbb8](https://github.com/cockroachdb/cockroach/commits/cec8af84bf01918d28b1e06a29b18ac9bee7cbb8):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6954286 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/tables.go:58
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).maybeGenSchemas
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/recurse.go:49
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).maybeGenDatabases
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/recurse.go:32
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).Generate.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:189
*   | github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen.(*testSchemaGenerator).Generate
*   | 	github.com/cockroachdb/cockroach/pkg/sql/catalog/randgen/randgen.go:204
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planner).GenerateTestObjects
*   | 	github.com/cockroachdb/cockroach/pkg/sql/generate_objects.go:65
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/builtins.glob..func291
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/builtins/builtins.go:6383
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.(*evaluator).EvalFuncExpr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:479
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/tree.(*FuncExpr).Eval
*   | 	github.com/cockroachdb/cockroach/bazel-out/k8-fastbuild/bin/pkg/sql/sem/tree/eval_expr_generated.go:276
*   | github.com/cockroachdb/cockroach/pkg/sql/sem/eval.Expr
*   | 	github.com/cockroachdb/cockroach/pkg/sql/sem/eval/expr.go:26
*   | github.com/cockroachdb/cockroach/pkg/sql.(*valuesNode).startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/values.go:80
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec.func2
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:523
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal.func1
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:112
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visitInternal
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:297
*   | github.com/cockroachdb/cockroach/pkg/sql.(*planVisitor).visit
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:79
*   | github.com/cockroachdb/cockroach/pkg/sql.walkPlan
*   | 	github.com/cockroachdb/cockroach/pkg/sql/walk.go:43
*   | github.com/cockroachdb/cockroach/pkg/sql.startExec
*   | 	github.com/cockroachdb/cockroach/pkg/sql/plan.go:526
* Wraps: (4) runtime error: makeslice: cap out of range
* Error types: (1) *withstack.withStack (2) *safedetails.withSafeDetails (3) *withstack.withStack (4) runtime.errorString
*
    rsg_test.go:822: 4m25s of 5m0s: 678929 executions, 418198 successful
    rsg_test.go:849: Crash detected: server panic: driver: bad connection
        SELECT crdb_internal.generate_test_objects('r':::STRING, 243171546204082085:::INT8) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('19:51:55.701433':::TIME, '06eeae48-5a73-4ce8-b38e-991792d3c022':::UUID, '''JkOV'':477C ''RPmRHwL'' ''krZuxuVn'' ''uEB''':::TSVECTOR, (SELECT NULL)::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 690724 executions, 425398 successful
    rsg_test.go:822: 4m35s of 5m0s: 700706 executions, 431573 successful
    rsg_test.go:822: 4m40s of 5m0s: 712869 executions, 439089 successful
    rsg_test.go:822: 4m45s of 5m0s: 726116 executions, 447223 successful
    rsg_test.go:822: 4m50s of 5m0s: 739574 executions, 455550 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8318160?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8318160?buildTab=artifacts#/) on master @ [8866ecadd99e5f6dc8db9fed5d09bc57cd90b03c](https://github.com/cockroachdb/cockroach/commits/8866ecadd99e5f6dc8db9fed5d09bc57cd90b03c):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6935689 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        SELECT json_build_object('''DtiYBGwyr''':::TSQUERY, 1643487600:::OID, 1.053224452112162:::FLOAT8, 'k':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 2m40s of 5m0s: 418226 executions, 257468 successful
    rsg_test.go:822: 2m45s of 5m0s: 431435 executions, 265670 successful
    rsg_test.go:822: 2m50s of 5m0s: 444499 executions, 273603 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''Wj''':::TSQUERY, 'j':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 2m55s of 5m0s: 458390 executions, 282218 successful
    rsg_test.go:822: 3m0s of 5m0s: 472101 executions, 290787 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''lvjDWqE''':::TSQUERY, 1583114540:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m5s of 5m0s: 485354 executions, 298915 successful
    rsg_test.go:822: 3m10s of 5m0s: 499031 executions, 307252 successful
    rsg_test.go:822: 3m15s of 5m0s: 512078 executions, 315287 successful
    rsg_test.go:822: 3m20s of 5m0s: 521614 executions, 321144 successful
    rsg_test.go:822: 3m25s of 5m0s: 532185 executions, 327745 successful
    rsg_test.go:822: 3m30s of 5m0s: 545029 executions, 335628 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object(0:::OID, '20:44:07.745593+06:46:00':::TIMETZ, '''ZpJ''':::TSQUERY, true) ;
        
        Stack trace:
    rsg_test.go:822: 3m35s of 5m0s: 558387 executions, 343879 successful
    rsg_test.go:822: 3m40s of 5m0s: 568076 executions, 349835 successful
    rsg_test.go:822: 3m45s of 5m0s: 581443 executions, 358028 successful
    rsg_test.go:822: 3m50s of 5m0s: 591744 executions, 364415 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('':::VOID, 'BOX(-0.1363696890518662 -0.41041594321930247,0.22339275780208234 0.8196420121186915)':::BOX2D, '''cwHcvpTp''':::TSQUERY, 552878966:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 601047 executions, 370130 successful
    rsg_test.go:822: 4m0s of 5m0s: 614315 executions, 378306 successful
    rsg_test.go:822: 4m5s of 5m0s: 626156 executions, 385680 successful
    rsg_test.go:822: 4m10s of 5m0s: 637681 executions, 392689 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('4':::STRING, 1734656895:::OID, '''E''':::TSQUERY, true) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 649920 executions, 400154 successful
    rsg_test.go:822: 4m20s of 5m0s: 661679 executions, 407445 successful
    rsg_test.go:822: 4m25s of 5m0s: 675658 executions, 416023 successful
    rsg_test.go:822: 4m30s of 5m0s: 689026 executions, 424326 successful
    rsg_test.go:822: 4m35s of 5m0s: 703311 executions, 433170 successful
    rsg_test.go:822: 4m40s of 5m0s: 714718 executions, 440193 successful
    rsg_test.go:822: 4m45s of 5m0s: 724887 executions, 446390 successful
    rsg_test.go:822: 4m50s of 5m0s: 738330 executions, 454686 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8286072?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8286072?buildTab=artifacts#/) on master @ [73485e56768d61b14a5aa9b3f880613d7820edb9](https://github.com/cockroachdb/cockroach/commits/73485e56768d61b14a5aa9b3f880613d7820edb9):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6833985 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 2m25s of 5m0s: 394882 executions, 243628 successful
    rsg_test.go:822: 2m30s of 5m0s: 407122 executions, 251058 successful
    rsg_test.go:822: 2m35s of 5m0s: 421562 executions, 260005 successful
    rsg_test.go:822: 2m40s of 5m0s: 432850 executions, 266965 successful
    rsg_test.go:822: 2m45s of 5m0s: 447386 executions, 275899 successful
    rsg_test.go:822: 2m50s of 5m0s: 458514 executions, 282673 successful
    rsg_test.go:822: 2m55s of 5m0s: 470215 executions, 289902 successful
    rsg_test.go:822: 3m0s of 5m0s: 481852 executions, 296996 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('"':::STRING:::NAME, 'aVO1Y':::STRING, '''VZnGaSIxVO''':::TSQUERY, e'\x03':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 3m5s of 5m0s: 495313 executions, 305286 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('01030000E0E61000000100000006000000BC5470E3FD7F43C0E4579FAB73F151C098F05C62A5E8F8C1E6C14132089BFCC120D895D1EF97544010975E614A8836C0A68D2AD17EB5E4C1C03D47920ADBF14160B88DEA2D6E64405040285031E51BC060846305DAC4014240B27C4FBF6401C2E06F95982E0B5C4040500F6AB007414074EA1AB958B8E3C1A4C2BEC25525E2C1E741B432E07B66C080AFAA81813C4A40F7FDD7468F5CF4C1B816DDE8C13FF741BC5470E3FD7F43C0E4579FAB73F151C098F05C62A5E8F8C1E6C14132089BFCC1':::GEOGRAPHY, NULL, '''JLbPg'' ''VdKhLqiT'':144C,458C,634C,962B ''dj'' ''dq'':310A,423C,590C ''ex'' ''iFR'':141B,600B,976A ''r'' ''uIOSWG''':::TSVECTOR, 674873793:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 3m10s of 5m0s: 509509 executions, 314106 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''BcA'':218B,898B ''GikCXnj'' ''KI'':176B,517A,784C,803C ''OzJbmECj'':625A,915B ''ndeqMcT'':20C,463B,907B ''tCqvElHfVu'':595B ''wHrHvcdNHv'' ''yAmbUEO'':462A,717B ''ypQ'':434B,909B':::TSVECTOR, '''ZF'':746C,817A,939A ''gtWJ'' ''iedqRt'':324C,349B ''tvifZxmkj'':167C,500A,687C ''uiyVcuB''':::TSVECTOR, 2688183337:::OID, 2482640444:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 3m15s of 5m0s: 521968 executions, 321762 successful
    rsg_test.go:822: 3m20s of 5m0s: 534738 executions, 329608 successful
    rsg_test.go:822: 3m25s of 5m0s: 547652 executions, 337620 successful
    rsg_test.go:822: 3m30s of 5m0s: 561758 executions, 346279 successful
    rsg_test.go:822: 3m35s of 5m0s: 576233 executions, 355167 successful
    rsg_test.go:822: 3m40s of 5m0s: 587735 executions, 362266 successful
    rsg_test.go:822: 3m45s of 5m0s: 600812 executions, 370271 successful
    rsg_test.go:822: 3m50s of 5m0s: 613650 executions, 378167 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: crdb_internal.generate_test_objects(): generating 32767 tables with name pattern "☃" under schema defaultdb.public: generating table defaultdb.public.☃23023: adding uncommitted relation "☃22196" (22607) version 1: memory usage exceeds limit for uncommittedDescriptors: root: memory budget exceeded: 10240 bytes requested, 134215680 currently allocated, 134217728 bytes in budget
        SELECT crdb_internal.generate_test_objects(e'\U00002603':::STRING, 32767:::INT8) ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 617482 executions, 380551 successful
    rsg_test.go:822: 4m0s of 5m0s: 629058 executions, 387704 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('248.139.110.82/6':::INET, 1506962134:::OID, '''MOWyUBwf'' ''MQjcoMljN'' ''Oh'' ''RDG'' ''TSSWFEnh'' ''cTiqu'':208A,636C,860C,902C ''sE'' ''uWSNs'' ''zndA''':::TSVECTOR, '1992-09-08 07:09:19.000714':::TIMESTAMP) ;
        
        Stack trace:
    rsg_test.go:822: 4m5s of 5m0s: 638511 executions, 393471 successful
    rsg_test.go:822: 4m10s of 5m0s: 650607 executions, 400988 successful
    rsg_test.go:822: 4m15s of 5m0s: 660639 executions, 407072 successful
    rsg_test.go:822: 4m20s of 5m0s: 670533 executions, 413241 successful
    rsg_test.go:822: 4m25s of 5m0s: 681849 executions, 420193 successful
    rsg_test.go:822: 4m30s of 5m0s: 693280 executions, 427234 successful
    rsg_test.go:822: 4m35s of 5m0s: 706507 executions, 435373 successful
    rsg_test.go:822: 4m40s of 5m0s: 716373 executions, 441561 successful
    rsg_test.go:822: 4m45s of 5m0s: 727437 executions, 448345 successful
    rsg_test.go:822: 4m50s of 5m0s: 739046 executions, 455481 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8335592?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8335592?buildTab=artifacts#/) on master @ [43cd78ecd87591d9b1d3b142f29ddfd76013b276](https://github.com/cockroachdb/cockroach/commits/43cd78ecd87591d9b1d3b142f29ddfd76013b276):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 5718841 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 1m5s of 5m0s: 11391 executions, 7022 successful
    rsg_test.go:822: 1m10s of 5m0s: 11688 executions, 7250 successful
    rsg_test.go:822: 1m15s of 5m0s: 12302 executions, 7520 successful
    rsg_test.go:822: 1m20s of 5m0s: 12356 executions, 7548 successful
    rsg_test.go:822: 1m25s of 5m0s: 12365 executions, 7556 successful
    rsg_test.go:822: 1m30s of 5m0s: 12384 executions, 7570 successful
    rsg_test.go:822: 1m35s of 5m0s: 13613 executions, 8418 successful
    rsg_test.go:822: 1m40s of 5m0s: 14235 executions, 8704 successful
    rsg_test.go:822: 1m45s of 5m0s: 15408 executions, 9532 successful
    rsg_test.go:822: 1m50s of 5m0s: 15730 executions, 9706 successful
    rsg_test.go:822: 1m55s of 5m0s: 16143 executions, 9890 successful
    rsg_test.go:822: 2m0s of 5m0s: 17371 executions, 10753 successful
    rsg_test.go:822: 2m5s of 5m0s: 18007 executions, 11048 successful
    rsg_test.go:822: 2m10s of 5m0s: 19228 executions, 11878 successful
    rsg_test.go:822: 2m15s of 5m0s: 19900 executions, 12213 successful
    rsg_test.go:822: 2m20s of 5m0s: 21083 executions, 13037 successful
    rsg_test.go:822: 2m25s of 5m0s: 21697 executions, 13314 successful
    rsg_test.go:822: 2m30s of 5m0s: 21752 executions, 13345 successful
    rsg_test.go:822: 2m35s of 5m0s: 21956 executions, 13491 successful
    rsg_test.go:822: 2m40s of 5m0s: 22961 executions, 14170 successful
    rsg_test.go:822: 2m45s of 5m0s: 23232 executions, 14308 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''YixqWDLkb'':63A,527C,785C,844B,894A ''o'':355A,555C':::TSVECTOR, ('1970-06-21 12:57:42.000937+00:00':::TIMESTAMPTZ, 0.0868157297372818:::FLOAT8), 4231689449:::OID, true) ;
        
        Stack trace:
    rsg_test.go:822: 2m50s of 5m0s: 23577 executions, 14446 successful
    rsg_test.go:822: 2m55s of 5m0s: 23837 executions, 14626 successful
    rsg_test.go:822: 3m0s of 5m0s: 24887 executions, 15356 successful
    rsg_test.go:822: 3m5s of 5m0s: 25517 executions, 15637 successful
    rsg_test.go:822: 3m10s of 5m0s: 25886 executions, 15912 successful
    rsg_test.go:822: 3m15s of 5m0s: 26990 executions, 16626 successful
    rsg_test.go:822: 3m20s of 5m0s: 27396 executions, 16795 successful
    rsg_test.go:822: 3m25s of 5m0s: 27607 executions, 16945 successful
    rsg_test.go:822: 3m30s of 5m0s: 28603 executions, 17643 successful
    rsg_test.go:822: 3m35s of 5m0s: 29178 executions, 17910 successful
    rsg_test.go:822: 3m40s of 5m0s: 29268 executions, 17949 successful
    rsg_test.go:822: 3m45s of 5m0s: 29295 executions, 17972 successful
    rsg_test.go:822: 3m50s of 5m0s: 30658 executions, 18879 successful
    rsg_test.go:822: 3m55s of 5m0s: 31160 executions, 19112 successful
    rsg_test.go:822: 4m0s of 5m0s: 32355 executions, 19946 successful
    rsg_test.go:822: 4m5s of 5m0s: 32971 executions, 20225 successful
    rsg_test.go:822: 4m10s of 5m0s: 33053 executions, 20270 successful
    rsg_test.go:822: 4m15s of 5m0s: 34446 executions, 21187 successful
    rsg_test.go:822: 4m20s of 5m0s: 34912 executions, 21392 successful
    rsg_test.go:822: 4m25s of 5m0s: 35618 executions, 21857 successful
    rsg_test.go:822: 4m30s of 5m0s: 36642 executions, 22453 successful
    rsg_test.go:822: 4m35s of 5m0s: 36797 executions, 22548 successful
    rsg_test.go:822: 4m40s of 5m0s: 37956 executions, 23368 successful
    rsg_test.go:822: 4m45s of 5m0s: 38609 executions, 23678 successful
    rsg_test.go:822: 4m50s of 5m0s: 38869 executions, 23857 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8351911?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8351911?buildTab=artifacts#/) on master @ [4ec5a5f43980583eff8b8f8dff1b23090c7e2304](https://github.com/cockroachdb/cockroach/commits/4ec5a5f43980583eff8b8f8dff1b23090c7e2304):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6531747 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''PbFgiB'' ''z''':::TSVECTOR, B'00000000') ;
        
        Stack trace:
    rsg_test.go:822: 3m25s of 5m0s: 521651 executions, 321653 successful
    rsg_test.go:822: 3m30s of 5m0s: 534736 executions, 329751 successful
    rsg_test.go:822: 3m35s of 5m0s: 546568 executions, 337063 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''uyehTpSx'':404A,785C':::TSVECTOR, '\xfc0387148e':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 560086 executions, 345398 successful
    rsg_test.go:822: 3m45s of 5m0s: 571635 executions, 352607 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object(481632232:::OID, (-0.07835777848958969):::FLOAT8, '''HyaoP'' ''WxCcgjmtYH'':433A,517A,935A ''uaFFjADG'':240C,686A,769C':::TSVECTOR, e'\r':::STRING) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''i'':184B,329A,602B ''vMjSRvEmZ'':173A,641C':::TSVECTOR, 'BOX(-0.6768287022128451 -2.2839721059313023,1.2360300302815763 -1.124182575415215)':::BOX2D, (-0.39064204692840576):::FLOAT8, '''JC'':101C,631B,840B ''wbngSlVvxS''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 3m50s of 5m0s: 583588 executions, 359816 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''aFOQljMU''':::TSQUERY, ARRAY['19:45:55.006094+02:34:00':::TIMETZ,'14:42:39.815284+05:35:00':::TIMETZ,'08:06:01.581939+02:08:00':::TIMETZ], 0:::OID, '010700000000000000':::GEOMETRY) ;
        
        Stack trace:
    rsg_test.go:822: 3m55s of 5m0s: 593352 executions, 365892 successful
    rsg_test.go:822: 4m0s of 5m0s: 603543 executions, 372193 successful
    rsg_test.go:822: 4m5s of 5m0s: 616914 executions, 380453 successful
    rsg_test.go:822: 4m10s of 5m0s: 629074 executions, 387908 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''FWH''':::TSQUERY, '2007-01-05 06:23:57.000317':::TIMESTAMP, '03:35:41.3465+09:57:00':::TIMETZ, '"':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 640987 executions, 395330 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''FLGZUhIY''':::TSQUERY, ARRAY[3584552807:::OID,679132327:::OID,2093691007:::OID,697370812:::OID]) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''GTuviDnw'' ''Js'':83A,606A,689B ''RNulhlSLe'' ''dPyiHZrqxp'':278A,980C ''yqZCnXX'':241A,945C':::TSVECTOR, 0:::OID, '05:30:07.277422':::TIME, e'\x1b':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m20s of 5m0s: 653068 executions, 402655 successful
    rsg_test.go:822: 4m25s of 5m0s: 664911 executions, 409973 successful
    rsg_test.go:822: 4m30s of 5m0s: 675382 executions, 416476 successful
    rsg_test.go:822: 4m35s of 5m0s: 687262 executions, 423790 successful
    rsg_test.go:822: 4m40s of 5m0s: 699935 executions, 431595 successful
    rsg_test.go:822: 4m45s of 5m0s: 709487 executions, 437488 successful
    rsg_test.go:822: 4m50s of 5m0s: 720306 executions, 444152 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8311522?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8311522?buildTab=artifacts#/) on master @ [8866ecadd99e5f6dc8db9fed5d09bc57cd90b03c](https://github.com/cockroachdb/cockroach/commits/8866ecadd99e5f6dc8db9fed5d09bc57cd90b03c):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6405584 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        Stack trace:
    rsg_test.go:822: 3m10s of 5m0s: 498372 executions, 307186 successful
    rsg_test.go:822: 3m15s of 5m0s: 513655 executions, 316539 successful
    rsg_test.go:822: 3m20s of 5m0s: 525804 executions, 323949 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''xEfSyBQGyf''':::TSQUERY, '''KC'' ''OOQ'' ''VHlzCVJwE'':944C ''aDVSsxRZW'':168A,422B,654C,864B ''pKFfuEu'' ''u''':::TSVECTOR, '01020000A0E610000007000000BED3D1C1AF9A6340C453DC924DA73440225FE8213D82F4415444726C5FA7594084421624A88D3440D4178270BD22F1414A61710296F06540506C26AF716D4640F03FA06354FDFE4166A0AEC113F36340A8B03C865D784640CCB6D9F25D150042880550DC8AEE52400843E8E3D170344090BE8646D064C8C194942E3DD43B4F400801BFD9BB234340F855FB20FB5EF741B47945838D0B4640F0B3AB3C6C4033401CE959C0F3B7FC41':::GEOGRAPHY, '1997-01-21 14:47:19.000048+00:00':::TIMESTAMPTZ) ;
        
        Stack trace:
    rsg_test.go:822: 3m25s of 5m0s: 537766 executions, 331390 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('66 years 10 mons 761 days 02:08:46.475521':::INTERVAL, 'BOX(1.0664250617880509 -0.5657868996608407,1.162258797128537 2.0312934972462675)':::BOX2D, '''WnSNAbYZ''':::TSQUERY, e'\x00':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 3m30s of 5m0s: 551252 executions, 339704 successful
    rsg_test.go:822: 3m35s of 5m0s: 564317 executions, 347769 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('1996-02-04':::DATE, 662169828:::OID, '''dndHaakeDx'' ''jAYta'':116A,159A,925B ''pgXuECnvHK'':119B,699A ''rnCBky'':285B ''vdB'':213C,348C ''ywGA'':283C,345B,639B,715C':::TSVECTOR, NULL) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 576535 executions, 355280 successful
    rsg_test.go:822: 3m45s of 5m0s: 587930 executions, 362309 successful
    rsg_test.go:822: 3m50s of 5m0s: 599291 executions, 369271 successful
    rsg_test.go:822: 3m55s of 5m0s: 609940 executions, 375795 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''DmLsLjku'':51A,271B,537A,648B ''FffXM'' ''VlgeY'':6B,90B,156B,904C ''XIN'' ''bRqi'':64B,251C ''gNc'':43A,51A ''nZkdDWbf'':858B ''obKbjCvxy'':164C':::TSVECTOR, (SELECT NULL)::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 4m0s of 5m0s: 621181 executions, 382664 successful
    rsg_test.go:822: 4m5s of 5m0s: 633441 executions, 390314 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''AHQHGIK'':375A,378A ''I'' ''UVWqSO'':211A,778C ''Zn'' ''de'':102A,150B,470B,971A ''lUG'' ''nYPSmxd'':555B':::TSVECTOR, '''vu'' ''wCHPSbkz''':::TSVECTOR) ;
        
        Stack trace:
    rsg_test.go:822: 4m10s of 5m0s: 643873 executions, 396694 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''ClrctEEoX''':::TSVECTOR, 'P[ge':::STRING:::NAME, ARRAY[2317083410:::OID,2811733197:::OID], '225.224.166.119/16':::INET) ;
        
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''mCfJaKy''':::TSQUERY, '00:00:00':::INTERVAL) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 656882 executions, 404710 successful
    rsg_test.go:822: 4m20s of 5m0s: 670330 executions, 413040 successful
    rsg_test.go:822: 4m25s of 5m0s: 684798 executions, 421967 successful
    rsg_test.go:822: 4m30s of 5m0s: 697149 executions, 429620 successful
    rsg_test.go:822: 4m35s of 5m0s: 709143 executions, 437005 successful
    rsg_test.go:822: 4m40s of 5m0s: 723882 executions, 446106 successful
    rsg_test.go:822: 4m45s of 5m0s: 736131 executions, 453583 successful
    rsg_test.go:822: 4m50s of 5m0s: 750233 executions, 462311 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8299499?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8299499?buildTab=artifacts#/) on master @ [a3d53668ff52c2306f37fcad8e9212ff665ca115](https://github.com/cockroachdb/cockroach/commits/a3d53668ff52c2306f37fcad8e9212ff665ca115):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6249974 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
    rsg_test.go:822: 2m25s of 5m0s: 368696 executions, 227378 successful
    rsg_test.go:822: 2m30s of 5m0s: 380598 executions, 234598 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT json_build_object('''ATVrW''':::TSQUERY, 'e694dd8a-0aec-47eb-9b33-a832fdce6e2f':::UUID, '-31 years -847 days -16:46:41.945572':::INTERVAL, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 2m35s of 5m0s: 392333 executions, 241920 successful
    rsg_test.go:822: 2m40s of 5m0s: 404188 executions, 249254 successful
    rsg_test.go:822: 2m45s of 5m0s: 416147 executions, 256539 successful
    rsg_test.go:822: 2m50s of 5m0s: 428200 executions, 263995 successful
    rsg_test.go:822: 2m55s of 5m0s: 439812 executions, 271208 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''F'':17C,494B,545C,725B':::TSVECTOR, '{"foo": "a", "zFp9": 0.3148849311961495}':::JSONB, 'c9E':::STRING, NULL::anyelement) ;
        
        Stack trace:
    rsg_test.go:822: 3m0s of 5m0s: 453378 executions, 279551 successful
    rsg_test.go:822: 3m5s of 5m0s: 466383 executions, 287623 successful
    rsg_test.go:822: 3m10s of 5m0s: 475718 executions, 293345 successful
    rsg_test.go:822: 3m15s of 5m0s: 486389 executions, 299839 successful
    rsg_test.go:822: 3m20s of 5m0s: 499843 executions, 308082 successful
    rsg_test.go:822: 3m25s of 5m0s: 515282 executions, 317646 successful
    rsg_test.go:822: 3m30s of 5m0s: 529730 executions, 326464 successful
    rsg_test.go:822: 3m35s of 5m0s: 543479 executions, 335029 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('''SB''':::TSQUERY, 613686031:::OID, '24.46.171.229/16':::INET, '1991-02-09 01:55:22.000451':::TIMESTAMP) ;
        
        Stack trace:
    rsg_test.go:822: 3m40s of 5m0s: 554627 executions, 341899 successful
    rsg_test.go:822: 3m45s of 5m0s: 568540 executions, 350400 successful
    rsg_test.go:822: 3m50s of 5m0s: 581356 executions, 358380 successful
    rsg_test.go:822: 3m55s of 5m0s: 592614 executions, 365296 successful
    rsg_test.go:822: 4m0s of 5m0s: 603791 executions, 372227 successful
    rsg_test.go:822: 4m5s of 5m0s: 615492 executions, 379418 successful
    rsg_test.go:822: 4m10s of 5m0s: 626992 executions, 386559 successful
    rsg_test.go:822: 4m15s of 5m0s: 639479 executions, 394328 successful
    rsg_test.go:822: 4m20s of 5m0s: 649780 executions, 400658 successful
    rsg_test.go:822: 4m25s of 5m0s: 662439 executions, 408421 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('1975-11-28 04:32:33.000111':::TIMESTAMP, 'X':::STRING, '''SzsJzcA''':::TSQUERY, 2831348092:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m30s of 5m0s: 674015 executions, 415555 successful
    rsg_test.go:822: 4m35s of 5m0s: 685315 executions, 422487 successful
    rsg_test.go:822: 4m40s of 5m0s: 696627 executions, 429499 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('c8be:444e:569b:2ed5:5c9:bf8f:a47b:82f1/35':::INET, ((-0.5223566293716431):::FLOAT8, 2682600831:::OID, B'11100101111010000011'), '''QVsEjnuQ'' ''S'' ''Tg'':795B,870B ''kFB'':93A,278A ''n'' ''pldiavku'' ''uwnSYRkA'' ''wHGXE'':73A,137A,574B,724A':::TSVECTOR, '-24 years -7 mons -34 days -05:13:34.862048':::INTERVAL) ;
        
        Stack trace:
    rsg_test.go:822: 4m45s of 5m0s: 709039 executions, 437203 successful
    rsg_test.go:822: 4m50s of 5m0s: 722134 executions, 445230 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

sql/tests.TestRandomSyntaxFunctions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8381775?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RandomSyntaxTestsBazel/8381775?buildTab=artifacts#/) on master @ [d698e5eaf36bee19901beeff7db3425315402907](https://github.com/cockroachdb/cockroach/commits/d698e5eaf36bee19901beeff7db3425315402907):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 6419345 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
        Stack trace:
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object('BOX(0.013581829542992696 -0.07198833643469316,1.8615272200122317 0.5725200781498724)':::BOX2D, B'0101111100111011001110001', '''YVFvocXcC''':::TSQUERY, 2240568098:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 2m50s of 5m0s: 431740 executions, 266185 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object((-1.541676640510559):::FLOAT8, (SELECT NULL)::anyelement, '''kgVHsmZ''':::TSVECTOR, '1982-11-26':::DATE) ;
        
        Stack trace:
    rsg_test.go:822: 2m55s of 5m0s: 445024 executions, 274355 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object(897978014:::OID, (SELECT NULL)::anyelement, '''BJpbucl'':522C,652C ''IrmMZtOYI'' ''PFQmjGHw'' ''aYnd'' ''jS'' ''nnNa'':186A,422A ''oV'' ''oZYOVAjiL'':406C,814C ''tqSf'' ''yoAuuNP''':::TSVECTOR, '\x3aec1950dfa7c4c39a':::BYTES) ;
        
        Stack trace:
    rsg_test.go:822: 3m0s of 5m0s: 458426 executions, 282610 successful
    rsg_test.go:822: 3m5s of 5m0s: 470421 executions, 290042 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSQuery for key value
        SELECT jsonb_build_object(0:::OID, NULL::anyelement, '''kIq''':::TSQUERY, '51$':::STRING:::NAME) ;
        
        Stack trace:
    rsg_test.go:822: 3m10s of 5m0s: 483574 executions, 298140 successful
    rsg_test.go:822: 3m15s of 5m0s: 496948 executions, 306343 successful
    rsg_test.go:822: 3m20s of 5m0s: 508422 executions, 313390 successful
    rsg_test.go:822: 3m25s of 5m0s: 520324 executions, 320788 successful
    rsg_test.go:822: 3m30s of 5m0s: 533192 executions, 328783 successful
    rsg_test.go:822: 3m35s of 5m0s: 545405 executions, 336344 successful
    rsg_test.go:822: 3m40s of 5m0s: 557983 executions, 344011 successful
    rsg_test.go:822: 3m45s of 5m0s: 569801 executions, 351394 successful
    rsg_test.go:822: 3m50s of 5m0s: 581074 executions, 358379 successful
    rsg_test.go:822: 3m55s of 5m0s: 595581 executions, 367272 successful
    rsg_test.go:822: 4m0s of 5m0s: 607713 executions, 374776 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: json_build_object(): unexpected type *tree.DTSVector for key value
        SELECT json_build_object('''AFu'' ''EKxrLk'' ''KwwMj'' ''LKtcdLdhB'' ''OdkI'' ''YNLVDz'' ''ljuZH'':135A,787B,822B,934C ''qD'':662C,733B ''zWOIuNxPWD'':314B,365C,783A':::TSVECTOR, 1026266063:::OID, (-1):::DECIMAL, 1172724250:::OID) ;
        
        Stack trace:
    rsg_test.go:822: 4m5s of 5m0s: 618629 executions, 381545 successful
    rsg_test.go:822: 4m10s of 5m0s: 630125 executions, 388645 successful
    rsg_test.go:849: Crash detected: server panic: pq: internal error: jsonb_build_object(): unexpected type *tree.DTSVector for key value
        SELECT jsonb_build_object('''FXwAvsMbou'' ''VT'':218B,235B,404A,483C,935A ''cT'':110A,156A,918A ''gQGjsl'':289B,937A ''js'' ''ndZWrIBicD'':104A,238A,245B,778C,808C ''pp'':6C,661A ''vyZr'':581C,758C':::TSVECTOR, 'P':::STRING) ;
        
        Stack trace:
    rsg_test.go:822: 4m15s of 5m0s: 643656 executions, 396930 successful
    rsg_test.go:822: 4m20s of 5m0s: 654928 executions, 403918 successful
    rsg_test.go:822: 4m25s of 5m0s: 665366 executions, 410424 successful
    rsg_test.go:822: 4m30s of 5m0s: 679653 executions, 419221 successful
    rsg_test.go:822: 4m35s of 5m0s: 693321 executions, 427659 successful
    rsg_test.go:822: 4m40s of 5m0s: 709376 executions, 437599 successful
    rsg_test.go:822: 4m45s of 5m0s: 711372 executions, 438827 successful
    rsg_test.go:822: 4m50s of 5m0s: 711372 executions, 438827 successful
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89470 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot T-sql-sessions branch-release-22.2.0]
- #87571 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-22.2]
- #66514 sql/tests: TestRandomSyntaxFunctions failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRandomSyntaxFunctions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/sql/tests/rsg_test.go
+++ b/pkg/sql/tests/rsg_test.go
@@ -781,6 +781,12 @@ func testRandomSyntax(
 	err := db.exec(t, ctx, "SET CLUSTER SETTING schemachanger.job.max_retry_backoff='1s'")
 	require.NoError(t, err)
 
+	// Disable the test object generator. This merely causes the built-in function to report
+	// an error when called. This is OK -- we are testing syntax, so this will still ensure
+	// the function syntax is exercised.
+	err = db.exec(t, ctx, "SET CLUSTER SETTING sql.schema.test_object_generator.enabled = false")
+	require.NoError(t, err)
+
 	yBytes, err := os.ReadFile(datapathutils.TestDataPath(t, "rsg", "sql.y"))
 	if err != nil {
 		t.Fatal(err)

